{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## This notebook takes fwd and rev reads from each sample (i.e., already demultiplexed) and combines them into a single sequence file with quality scores"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "With thanks to Dan Buckley's lab's protocols:  https://github.com/buckleylab/Buckley_Lab_SIP_project_protocols/blob/master/sequence_analysys_walkthrough/detailed_walkthroughs/qc.md"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "I did most of this with the demultiplexed files, where we might have actually just wanted them to be all one file, as we go for at the end. Anyway."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Get ready for some magics!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%load_ext rpy2.ipython"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%load_ext mothurmagic"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \r\n",
      "|  _ \\| ____|  / \\  |  _ \\\r\n",
      "| |_) |  _|   / _ \\ | |_) |\r\n",
      "|  __/| |___ / ___ \\|  _ <\r\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\r\n",
      "PEAR v0.9.6 [January 15, 2015]  - [+bzlib +zlib]\r\n",
      "\r\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\r\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\r\n",
      "\r\n",
      "License: Creative Commons Licence\r\n",
      "Bug-reports and requests to: Tomas.Flouri@h-its.org and Jiajie.Zhang@h-its.org\r\n",
      "\r\n",
      "\r\n",
      "Usage: pear <options>\r\n",
      "Standard (mandatory):\r\n",
      "  -f, --forward-fastq         <str>     Forward paired-end FASTQ file.\r\n",
      "  -r, --reverse-fastq         <str>     Reverse paired-end FASTQ file.\r\n",
      "  -o, --output                <str>     Output filename.\r\n",
      "Optional:\r\n",
      "  -p, --p-value               <float>   Specify  a p-value for the statistical test. If the computed\r\n",
      "                                        p-value of a possible assembly exceeds the specified p-value\r\n",
      "                                        then  paired-end  read  will not be assembled. Valid options\r\n",
      "                                        are: 0.0001, 0.001, 0.01, 0.05 and 1.0. Setting 1.0 disables\r\n",
      "                                        the test. (default: 0.01)\r\n",
      "  -v, --min-overlap           <int>     Specify the minimum overlap size. The minimum overlap may be\r\n",
      "                                        set to 1 when the statistical test is used. However, further\r\n",
      "                                        restricting  the  minimum overlap size to a proper value may\r\n",
      "                                        reduce false-positive assembles. (default: 10)\r\n",
      "  -m, --max-assembly-length   <int>     Specify   the  maximum  possible  length  of  the  assembled\r\n",
      "                                        sequences.  Setting this value to 0 disables the restriction\r\n",
      "                                        and assembled sequences may be arbitrary long. (default: 0)\r\n",
      "  -n, --min-assembly-length   <int>     Specify   the  minimum  possible  length  of  the  assembled\r\n",
      "                                        sequences.  Setting this value to 0 disables the restriction\r\n",
      "                                        and  assembled  sequences  may be arbitrary short. (default:\r\n",
      "                                        50)\r\n",
      "  -t, --min-trim-length       <int>     Specify  the  minimum length of reads after trimming the low\r\n",
      "                                        quality part (see option -q). (default: 1)\r\n",
      "  -q, --quality-threshold     <int>     Specify  the  quality  score  threshold for trimming the low\r\n",
      "                                        quality  part  of  a  read.  If  the  quality  scores of two\r\n",
      "                                        consecutive  bases  are  strictly  less  than  the specified\r\n",
      "                                        threshold,  the  rest of the read will be trimmed. (default:\r\n",
      "                                        0)\r\n",
      "  -u, --max-uncalled-base     <float>   Specify  the maximal proportion of uncalled bases in a read.\r\n",
      "                                        Setting this value to 0 will cause PEAR to discard all reads\r\n",
      "                                        containing  uncalled  bases.  The other extreme setting is 1\r\n",
      "                                        which  causes  PEAR  to process all reads independent on the\r\n",
      "                                        number of uncalled bases. (default: 1)\r\n",
      "  -g, --test-method           <int>     Specify  the  type  of  statistical  test.  Two  options are\r\n",
      "                                        available. (default: 1)\r\n",
      "                                        1: Given the minimum allowed overlap, test using the highest\r\n",
      "                                        OES. Note that due to its discrete nature, this test usually\r\n",
      "                                        yields  a lower p-value for the assembled read than the cut-\r\n",
      "                                        off  (specified  by -p). For example, setting the cut-off to\r\n",
      "                                        0.05  using  this  test,  the  assembled reads might have an\r\n",
      "                                        actual p-value of 0.02.\r\n",
      "\r\n",
      "                                        2. Use the acceptance probability (m.a.p). This test methods\r\n",
      "                                        computes  the same probability as test method 1. However, it\r\n",
      "                                        assumes  that  the  minimal  overlap is the observed overlap\r\n",
      "                                        with  the  highest  OES, instead of the one specified by -v.\r\n",
      "                                        Therefore,  this  is  not  a  valid statistical test and the\r\n",
      "                                        'p-value'  is  in fact the maximal probability for accepting\r\n",
      "                                        the assembly. Nevertheless, we observed in practice that for\r\n",
      "                                        the case the actual overlap sizes are relatively small, test\r\n",
      "                                        2  can  correctly  assemble  more  reads  with only slightly\r\n",
      "                                        higher false-positive rate.\r\n",
      "  -e, --empirical-freqs                 Disable  empirical base frequencies. (default: use empirical\r\n",
      "                                        base frequencies)\r\n",
      "  -s, --score-method          <int>     Specify the scoring method. (default: 2)\r\n",
      "                                        1. OES with +1 for match and -1 for mismatch.\r\n",
      "                                        2: Assembly score (AS). Use +1 for match and -1 for mismatch\r\n",
      "                                        multiplied by base quality scores.\r\n",
      "                                        3: Ignore quality scores and use +1 for a match and -1 for a\r\n",
      "                                        mismatch.\r\n",
      "  -b, --phred-base            <int>     Base PHRED quality score. (default: 33)\r\n",
      "  -y, --memory                <str>     Specify  the  amount of memory to be used. The number may be\r\n",
      "                                        followed  by  one  of  the  letters  K,  M,  or  G  denoting\r\n",
      "                                        Kilobytes,  Megabytes and Gigabytes, respectively. Bytes are\r\n",
      "                                        assumed in case no letter is specified.\r\n",
      "  -c, --cap                   <int>     Specify  the upper bound for the resulting quality score. If\r\n",
      "                                        set to zero, capping is disabled. (default: 40)\r\n",
      "  -j, --threads               <int>     Number of threads to use\r\n",
      "  -z, --nbase                           When  merging  a  base-pair  that  consists of two non-equal\r\n",
      "                                        bases  out  of which none is degenerate, set the merged base\r\n",
      "                                        to N and use the highest quality score of the two bases\r\n",
      "  -h, --help                            This help screen.\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!pear -h\n",
    "# Check out the Pear merge help files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir ../data/Seq_data/q_data/MergedSeqs\n",
    "# Create the directory the product merged sequences will go"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-19O-1_TGCTCGTA-GATCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-19O-1_TGCTCGTA-GATCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214818\n",
      "  C: 0.289926\n",
      "  G: 0.279739\n",
      "  T: 0.215517\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 30,583 / 30,789 (99.331%)\n",
      "Discarded reads ...................: 0 / 30,789 (0.000%)\n",
      "Not assembled reads ...............: 206 / 30,789 (0.669%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-19O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-19O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-19O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-19O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-19O-2_TAGTCTCC-TGCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-19O-2_TAGTCTCC-TGCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214853\n",
      "  C: 0.289368\n",
      "  G: 0.279913\n",
      "  T: 0.215866\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 78,961 / 79,841 (98.898%)\n",
      "Discarded reads ...................: 0 / 79,841 (0.000%)\n",
      "Not assembled reads ...............: 880 / 79,841 (1.102%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-19O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-19O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-19O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-19O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-20O-A-1_TAGCAGCT-GACACCGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-20O-A-1_TAGCAGCT-GACACCGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214361\n",
      "  C: 0.290155\n",
      "  G: 0.280452\n",
      "  T: 0.215032\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 10,230 / 10,307 (99.253%)\n",
      "Discarded reads ...................: 0 / 10,307 (0.000%)\n",
      "Not assembled reads ...............: 77 / 10,307 (0.747%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-20O-A-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-20O-A-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-20O-A-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-20O-A-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-20O-A-2_AACGCTGA-GTCAGATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-20O-A-2_AACGCTGA-GTCAGATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214000\n",
      "  C: 0.290503\n",
      "  G: 0.280842\n",
      "  T: 0.214654\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,527 / 22,659 (99.417%)\n",
      "Discarded reads ...................: 0 / 22,659 (0.000%)\n",
      "Not assembled reads ...............: 132 / 22,659 (0.583%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-20O-A-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-20O-A-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-20O-A-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-20O-A-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-20O-B-1_CGAGCGAC-TAGTGTAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-20O-B-1_CGAGCGAC-TAGTGTAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212929\n",
      "  C: 0.292248\n",
      "  G: 0.281041\n",
      "  T: 0.213782\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 31,175 / 31,393 (99.306%)\n",
      "Discarded reads ...................: 0 / 31,393 (0.000%)\n",
      "Not assembled reads ...............: 218 / 31,393 (0.694%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-20O-B-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-20O-B-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-20O-B-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-20O-B-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-20O-B-2_CGAGCGAC-ATCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-20O-B-2_CGAGCGAC-ATCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214255\n",
      "  C: 0.290068\n",
      "  G: 0.281087\n",
      "  T: 0.214590\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 27,241 / 27,387 (99.467%)\n",
      "Discarded reads ...................: 0 / 27,387 (0.000%)\n",
      "Not assembled reads ...............: 146 / 27,387 (0.533%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-20O-B-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-20O-B-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-20O-B-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-20O-B-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-21M-1_TCTCTATG-CTACTATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-21M-1_TCTCTATG-CTACTATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.211579\n",
      "  C: 0.292740\n",
      "  G: 0.283448\n",
      "  T: 0.212233\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 29,062 / 29,326 (99.100%)\n",
      "Discarded reads ...................: 0 / 29,326 (0.000%)\n",
      "Not assembled reads ...............: 264 / 29,326 (0.900%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-21M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-21M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-21M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-21M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-21M-2_CGTAGCGA-ACGACGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-21M-2_CGTAGCGA-ACGACGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.210990\n",
      "  C: 0.293564\n",
      "  G: 0.283730\n",
      "  T: 0.211715\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 15,281 / 15,458 (98.855%)\n",
      "Discarded reads ...................: 0 / 15,458 (0.000%)\n",
      "Not assembled reads ...............: 177 / 15,458 (1.145%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-21M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-21M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-21M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-21M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-21O-1_ACTACGAC-CGTTACTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-21O-1_ACTACGAC-CGTTACTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213097\n",
      "  C: 0.290943\n",
      "  G: 0.282375\n",
      "  T: 0.213585\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 14,035 / 14,143 (99.236%)\n",
      "Discarded reads ...................: 0 / 14,143 (0.000%)\n",
      "Not assembled reads ...............: 108 / 14,143 (0.764%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-21O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-21O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-21O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-21O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-21O-2_GATCTACG-ATATACAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-21O-2_GATCTACG-ATATACAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213118\n",
      "  C: 0.291065\n",
      "  G: 0.282184\n",
      "  T: 0.213633\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 20,106 / 20,247 (99.304%)\n",
      "Discarded reads ...................: 0 / 20,247 (0.000%)\n",
      "Not assembled reads ...............: 141 / 20,247 (0.696%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-21O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-21O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-21O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-21O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-22M-1_GACATAGT-TAGCGAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-22M-1_GACATAGT-TAGCGAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212392\n",
      "  C: 0.292173\n",
      "  G: 0.282523\n",
      "  T: 0.212912\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 17,432 / 17,548 (99.339%)\n",
      "Discarded reads ...................: 0 / 17,548 (0.000%)\n",
      "Not assembled reads ...............: 116 / 17,548 (0.661%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-22M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-22M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-22M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-22M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-22M-2_GTCTGCTA-AGAGTCAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-22M-2_GTCTGCTA-AGAGTCAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212200\n",
      "  C: 0.292271\n",
      "  G: 0.282811\n",
      "  T: 0.212718\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 9,666 / 9,747 (99.169%)\n",
      "Discarded reads ...................: 0 / 9,747 (0.000%)\n",
      "Not assembled reads ...............: 81 / 9,747 (0.831%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-22M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-22M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-22M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-22M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-22O-1_GTAACGAG-CGTCGCTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-22O-1_GTAACGAG-CGTCGCTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217379\n",
      "  C: 0.286900\n",
      "  G: 0.277819\n",
      "  T: 0.217902\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 26,087 / 26,235 (99.436%)\n",
      "Discarded reads ...................: 0 / 26,235 (0.000%)\n",
      "Not assembled reads ...............: 148 / 26,235 (0.564%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-22O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-22O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-22O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-22O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-22O-2_ACGCTACT-CTGCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-22O-2_ACGCTACT-CTGCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216619\n",
      "  C: 0.287939\n",
      "  G: 0.278034\n",
      "  T: 0.217408\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 18,823 / 18,966 (99.246%)\n",
      "Discarded reads ...................: 0 / 18,966 (0.000%)\n",
      "Not assembled reads ...............: 143 / 18,966 (0.754%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-22O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-22O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-22O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-22O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-23M-1_GTCTATGA-TACGAGAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-23M-1_GTCTATGA-TACGAGAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217591\n",
      "  C: 0.286747\n",
      "  G: 0.277292\n",
      "  T: 0.218370\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 16,841 / 16,987 (99.141%)\n",
      "Discarded reads ...................: 0 / 16,987 (0.000%)\n",
      "Not assembled reads ...............: 146 / 16,987 (0.859%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-23M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-23M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-23M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-23M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-23M-2_ACGTGCGC-CTAGAGCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-23M-2_ACGTGCGC-CTAGAGCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217327\n",
      "  C: 0.287304\n",
      "  G: 0.277659\n",
      "  T: 0.217711\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 17,988 / 18,129 (99.222%)\n",
      "Discarded reads ...................: 0 / 18,129 (0.000%)\n",
      "Not assembled reads ...............: 141 / 18,129 (0.778%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-23M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-23M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-23M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-23M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-23O-1_ACTCACTG-TCATCGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-23O-1_ACTCACTG-TCATCGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.220502\n",
      "  C: 0.284346\n",
      "  G: 0.273544\n",
      "  T: 0.221607\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 46,370 / 47,007 (98.645%)\n",
      "Discarded reads ...................: 0 / 47,007 (0.000%)\n",
      "Not assembled reads ...............: 637 / 47,007 (1.355%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-23O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-23O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-23O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-23O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-23O-2_TATAGCGA-ACGTCTCG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-23O-2_TATAGCGA-ACGTCTCG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.219731\n",
      "  C: 0.285349\n",
      "  G: 0.273705\n",
      "  T: 0.221215\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 34,495 / 34,944 (98.715%)\n",
      "Discarded reads ...................: 0 / 34,944 (0.000%)\n",
      "Not assembled reads ...............: 449 / 34,944 (1.285%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-23O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-23O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-23O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-23O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-24M-1_ATAGTACC-GCTCTAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-24M-1_ATAGTACC-GCTCTAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215973\n",
      "  C: 0.288365\n",
      "  G: 0.279142\n",
      "  T: 0.216519\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 31,060 / 31,295 (99.249%)\n",
      "Discarded reads ...................: 0 / 31,295 (0.000%)\n",
      "Not assembled reads ...............: 235 / 31,295 (0.751%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-24M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-24M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-24M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-24M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-24M-2_TGAGTACG-CGTGAGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-24M-2_TGAGTACG-CGTGAGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216986\n",
      "  C: 0.287846\n",
      "  G: 0.277546\n",
      "  T: 0.217622\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 28,481 / 28,734 (99.120%)\n",
      "Discarded reads ...................: 0 / 28,734 (0.000%)\n",
      "Not assembled reads ...............: 253 / 28,734 (0.880%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-24M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-24M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-24M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-24M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-24O-1_CTCGACTT-TCGACGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-24O-1_CTCGACTT-TCGACGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216899\n",
      "  C: 0.287842\n",
      "  G: 0.277289\n",
      "  T: 0.217971\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 13,175 / 13,328 (98.852%)\n",
      "Discarded reads ...................: 0 / 13,328 (0.000%)\n",
      "Not assembled reads ...............: 153 / 13,328 (1.148%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-24O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-24O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-24O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-24O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-24O-2_GCGTATAC-GACACTGA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-24O-2_GCGTATAC-GACACTGA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215864\n",
      "  C: 0.288857\n",
      "  G: 0.278370\n",
      "  T: 0.216908\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 35,130 / 35,485 (99.000%)\n",
      "Discarded reads ...................: 0 / 35,485 (0.000%)\n",
      "Not assembled reads ...............: 355 / 35,485 (1.000%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-24O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-24O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-24O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-24O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-25O-1_CTGCGTAG-GGATATCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-25O-1_CTGCGTAG-GGATATCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218711\n",
      "  C: 0.286760\n",
      "  G: 0.274892\n",
      "  T: 0.219637\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 56,269 / 56,815 (99.039%)\n",
      "Discarded reads ...................: 0 / 56,815 (0.000%)\n",
      "Not assembled reads ...............: 546 / 56,815 (0.961%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-25O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-25O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-25O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-25O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-25O-2_CGAAGTAT-GATCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-25O-2_CGAAGTAT-GATCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.219072\n",
      "  C: 0.285935\n",
      "  G: 0.275348\n",
      "  T: 0.219645\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 29,715 / 29,970 (99.149%)\n",
      "Discarded reads ...................: 0 / 29,970 (0.000%)\n",
      "Not assembled reads ...............: 255 / 29,970 (0.851%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-25O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-25O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-25O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-25O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-26O-1_TGCTCGTA-TGCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-26O-1_TGCTCGTA-TGCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217813\n",
      "  C: 0.286271\n",
      "  G: 0.277564\n",
      "  T: 0.218352\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 24,518 / 24,654 (99.448%)\n",
      "Discarded reads ...................: 0 / 24,654 (0.000%)\n",
      "Not assembled reads ...............: 136 / 24,654 (0.552%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-26O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-26O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-26O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-26O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-26O-2_TAGTCTCC-GACACCGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-26O-2_TAGTCTCC-GACACCGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216983\n",
      "  C: 0.287365\n",
      "  G: 0.278195\n",
      "  T: 0.217457\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 15,737 / 15,839 (99.356%)\n",
      "Discarded reads ...................: 0 / 15,839 (0.000%)\n",
      "Not assembled reads ...............: 102 / 15,839 (0.644%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-26O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-26O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-26O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-26O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-27M-1_AACGCTGA-ATCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-27M-1_AACGCTGA-ATCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214492\n",
      "  C: 0.289956\n",
      "  G: 0.280241\n",
      "  T: 0.215310\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,090 / 22,306 (99.032%)\n",
      "Discarded reads ...................: 0 / 22,306 (0.000%)\n",
      "Not assembled reads ...............: 216 / 22,306 (0.968%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-27M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-27M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-27M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-27M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-27M-2_CGAGCGAC-CTACTATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-27M-2_CGAGCGAC-CTACTATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214195\n",
      "  C: 0.290173\n",
      "  G: 0.280861\n",
      "  T: 0.214771\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 36,503 / 36,824 (99.128%)\n",
      "Discarded reads ...................: 0 / 36,824 (0.000%)\n",
      "Not assembled reads ...............: 321 / 36,824 (0.872%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-27M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-27M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-27M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-27M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-27O-1_TCTCTATG-ACGACGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-27O-1_TCTCTATG-ACGACGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215936\n",
      "  C: 0.288399\n",
      "  G: 0.279294\n",
      "  T: 0.216371\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 24,400 / 24,601 (99.183%)\n",
      "Discarded reads ...................: 0 / 24,601 (0.000%)\n",
      "Not assembled reads ...............: 201 / 24,601 (0.817%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-27O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-27O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-27O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-27O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-27O-2_CGTAGCGA-ACTATCTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-27O-2_CGTAGCGA-ACTATCTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215925\n",
      "  C: 0.288654\n",
      "  G: 0.278845\n",
      "  T: 0.216576\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 33,823 / 34,152 (99.037%)\n",
      "Discarded reads ...................: 0 / 34,152 (0.000%)\n",
      "Not assembled reads ...............: 329 / 34,152 (0.963%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-27O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-27O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-27O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-27O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-28M-1_CGAGAGTT-CGTTACTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-28M-1_CGAGAGTT-CGTTACTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214238\n",
      "  C: 0.290915\n",
      "  G: 0.279203\n",
      "  T: 0.215645\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 36,967 / 37,658 (98.165%)\n",
      "Discarded reads ...................: 0 / 37,658 (0.000%)\n",
      "Not assembled reads ...............: 691 / 37,658 (1.835%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-28M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-28M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-28M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-28M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-28M-2_ACTACGAC-ATATACAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-28M-2_ACTACGAC-ATATACAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214342\n",
      "  C: 0.290369\n",
      "  G: 0.279534\n",
      "  T: 0.215755\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 10,977 / 11,127 (98.652%)\n",
      "Discarded reads ...................: 0 / 11,127 (0.000%)\n",
      "Not assembled reads ...............: 150 / 11,127 (1.348%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-28M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-28M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-28M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-28M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-28O-1_GACATAGT-AGAGTCAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-28O-1_GACATAGT-AGAGTCAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216962\n",
      "  C: 0.289286\n",
      "  G: 0.274344\n",
      "  T: 0.219408\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 25,941 / 26,626 (97.427%)\n",
      "Discarded reads ...................: 0 / 26,626 (0.000%)\n",
      "Not assembled reads ...............: 685 / 26,626 (2.573%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-28O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-28O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-28O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-28O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-28O-2_GTCTGCTA-CGTCGCTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-28O-2_GTCTGCTA-CGTCGCTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217482\n",
      "  C: 0.288462\n",
      "  G: 0.274818\n",
      "  T: 0.219238\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 17,568 / 17,960 (97.817%)\n",
      "Discarded reads ...................: 0 / 17,960 (0.000%)\n",
      "Not assembled reads ...............: 392 / 17,960 (2.183%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-28O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-28O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-28O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-28O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-29O-1_GTAACGAG-CTGCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-29O-1_GTAACGAG-CTGCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.210899\n",
      "  C: 0.293658\n",
      "  G: 0.283766\n",
      "  T: 0.211677\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 20,525 / 20,716 (99.078%)\n",
      "Discarded reads ...................: 0 / 20,716 (0.000%)\n",
      "Not assembled reads ...............: 191 / 20,716 (0.922%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-29O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-29O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-29O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-29O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-29O-2_ACGCTACT-TACGAGAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-29O-2_ACGCTACT-TACGAGAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.209241\n",
      "  C: 0.294963\n",
      "  G: 0.285737\n",
      "  T: 0.210059\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 15,393 / 15,560 (98.927%)\n",
      "Discarded reads ...................: 0 / 15,560 (0.000%)\n",
      "Not assembled reads ...............: 167 / 15,560 (1.073%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-29O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-29O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-29O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-29O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-30M-1_GTCTATGA-CTAGAGCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-30M-1_GTCTATGA-CTAGAGCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215520\n",
      "  C: 0.289386\n",
      "  G: 0.278905\n",
      "  T: 0.216188\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 34,276 / 34,556 (99.190%)\n",
      "Discarded reads ...................: 0 / 34,556 (0.000%)\n",
      "Not assembled reads ...............: 280 / 34,556 (0.810%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-30M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-30M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-30M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-30M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-30M-2_ACGTGCGC-TCATCGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-30M-2_ACGTGCGC-TCATCGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216102\n",
      "  C: 0.288281\n",
      "  G: 0.279250\n",
      "  T: 0.216366\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 39,780 / 40,065 (99.289%)\n",
      "Discarded reads ...................: 0 / 40,065 (0.000%)\n",
      "Not assembled reads ...............: 285 / 40,065 (0.711%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-30M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-30M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-30M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-30M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-30O-1_ACTCACTG-ACGTCTCG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-30O-1_ACTCACTG-ACGTCTCG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.224986\n",
      "  C: 0.279684\n",
      "  G: 0.269255\n",
      "  T: 0.226076\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 13,974 / 14,095 (99.142%)\n",
      "Discarded reads ...................: 0 / 14,095 (0.000%)\n",
      "Not assembled reads ...............: 121 / 14,095 (0.858%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-30O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-30O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-30O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-30O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-30O-2_TATAGCGA-GCTCTAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-30O-2_TATAGCGA-GCTCTAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.226235\n",
      "  C: 0.278074\n",
      "  G: 0.268545\n",
      "  T: 0.227146\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,551 / 22,691 (99.383%)\n",
      "Discarded reads ...................: 0 / 22,691 (0.000%)\n",
      "Not assembled reads ...............: 140 / 22,691 (0.617%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-30O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-30O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-30O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-30O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-31M-1_ATAGTACC-CGTGAGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-31M-1_ATAGTACC-CGTGAGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213148\n",
      "  C: 0.291200\n",
      "  G: 0.281841\n",
      "  T: 0.213812\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 17,232 / 17,397 (99.052%)\n",
      "Discarded reads ...................: 0 / 17,397 (0.000%)\n",
      "Not assembled reads ...............: 165 / 17,397 (0.948%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-31M-2_TGAGTACG-TCGACGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-31M-2_TGAGTACG-TCGACGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.211988\n",
      "  C: 0.292794\n",
      "  G: 0.282857\n",
      "  T: 0.212361\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 12,998 / 13,128 (99.010%)\n",
      "Discarded reads ...................: 0 / 13,128 (0.000%)\n",
      "Not assembled reads ...............: 130 / 13,128 (0.990%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-31O-A-1_CTCGACTT-GACACTGA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-31O-A-1_CTCGACTT-GACACTGA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.222970\n",
      "  C: 0.281206\n",
      "  G: 0.271865\n",
      "  T: 0.223959\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 21,345 / 21,511 (99.228%)\n",
      "Discarded reads ...................: 0 / 21,511 (0.000%)\n",
      "Not assembled reads ...............: 166 / 21,511 (0.772%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31O-A-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31O-A-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31O-A-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31O-A-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-31O-A-2_GCGTATAC-GGATATCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-31O-A-2_GCGTATAC-GGATATCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.221687\n",
      "  C: 0.283486\n",
      "  G: 0.271377\n",
      "  T: 0.223450\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 26,143 / 26,460 (98.802%)\n",
      "Discarded reads ...................: 0 / 26,460 (0.000%)\n",
      "Not assembled reads ...............: 317 / 26,460 (1.198%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31O-A-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31O-A-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31O-A-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31O-A-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-31O-B-1_CTGCGTAG-GATCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-31O-B-1_CTGCGTAG-GATCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214071\n",
      "  C: 0.291069\n",
      "  G: 0.280226\n",
      "  T: 0.214634\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 12,565 / 12,667 (99.195%)\n",
      "Discarded reads ...................: 0 / 12,667 (0.000%)\n",
      "Not assembled reads ...............: 102 / 12,667 (0.805%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31O-B-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31O-B-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31O-B-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31O-B-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-31O-B-2_CGAAGTAT-TGCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-31O-B-2_CGAAGTAT-TGCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213745\n",
      "  C: 0.290666\n",
      "  G: 0.281029\n",
      "  T: 0.214561\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 13,381 / 13,510 (99.045%)\n",
      "Discarded reads ...................: 0 / 13,510 (0.000%)\n",
      "Not assembled reads ...............: 129 / 13,510 (0.955%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31O-B-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-31O-B-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31O-B-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-31O-B-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-32O-1_TGCTCGTA-GACACCGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-32O-1_TGCTCGTA-GACACCGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216186\n",
      "  C: 0.288178\n",
      "  G: 0.278976\n",
      "  T: 0.216660\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 10,721 / 10,786 (99.397%)\n",
      "Discarded reads ...................: 0 / 10,786 (0.000%)\n",
      "Not assembled reads ...............: 65 / 10,786 (0.603%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-32O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-32O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-32O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-32O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-32O-2_TAGTCTCC-GTCAGATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-32O-2_TAGTCTCC-GTCAGATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215691\n",
      "  C: 0.288538\n",
      "  G: 0.279762\n",
      "  T: 0.216010\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 11,884 / 11,976 (99.232%)\n",
      "Discarded reads ...................: 0 / 11,976 (0.000%)\n",
      "Not assembled reads ...............: 92 / 11,976 (0.768%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-32O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-32O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-32O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-32O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-33M-1_TAGCAGCT-TAGTGTAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-33M-1_TAGCAGCT-TAGTGTAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.211981\n",
      "  C: 0.292958\n",
      "  G: 0.282370\n",
      "  T: 0.212691\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 94%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 83,877 / 84,625 (99.116%)\n",
      "Discarded reads ...................: 0 / 84,625 (0.000%)\n",
      "Not assembled reads ...............: 748 / 84,625 (0.884%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-33M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-33M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-33M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-33M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-33M-2_TAGCAGCT-ATCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-33M-2_TAGCAGCT-ATCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212461\n",
      "  C: 0.291632\n",
      "  G: 0.282820\n",
      "  T: 0.213087\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 15,092 / 15,193 (99.335%)\n",
      "Discarded reads ...................: 0 / 15,193 (0.000%)\n",
      "Not assembled reads ...............: 101 / 15,193 (0.665%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-33M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-33M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-33M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-33M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-33O-1_AACGCTGA-CTACTATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-33O-1_AACGCTGA-CTACTATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216756\n",
      "  C: 0.287530\n",
      "  G: 0.278190\n",
      "  T: 0.217524\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 13,267 / 13,375 (99.193%)\n",
      "Discarded reads ...................: 0 / 13,375 (0.000%)\n",
      "Not assembled reads ...............: 108 / 13,375 (0.807%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-33O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-33O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-33O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-33O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-33O-2_CGAGCGAC-ACGACGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-33O-2_CGAGCGAC-ACGACGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216926\n",
      "  C: 0.287461\n",
      "  G: 0.278146\n",
      "  T: 0.217466\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 16,830 / 16,964 (99.210%)\n",
      "Discarded reads ...................: 0 / 16,964 (0.000%)\n",
      "Not assembled reads ...............: 134 / 16,964 (0.790%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-33O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-33O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-33O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-33O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-34M-1_TCTCTATG-ACTATCTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-34M-1_TCTCTATG-ACTATCTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.211974\n",
      "  C: 0.292631\n",
      "  G: 0.282927\n",
      "  T: 0.212468\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 8,731 / 8,803 (99.182%)\n",
      "Discarded reads ...................: 0 / 8,803 (0.000%)\n",
      "Not assembled reads ...............: 72 / 8,803 (0.818%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-34M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-34M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-34M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-34M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-34M-2_CGTAGCGA-CGTTACTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-34M-2_CGTAGCGA-CGTTACTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.211184\n",
      "  C: 0.293455\n",
      "  G: 0.283300\n",
      "  T: 0.212061\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 14,190 / 14,336 (98.982%)\n",
      "Discarded reads ...................: 0 / 14,336 (0.000%)\n",
      "Not assembled reads ...............: 146 / 14,336 (1.018%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-34M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-34M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-34M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-34M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-34O-1_CGAGAGTT-ATATACAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-34O-1_CGAGAGTT-ATATACAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215163\n",
      "  C: 0.289913\n",
      "  G: 0.278429\n",
      "  T: 0.216496\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 35,915 / 36,457 (98.513%)\n",
      "Discarded reads ...................: 0 / 36,457 (0.000%)\n",
      "Not assembled reads ...............: 542 / 36,457 (1.487%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-34O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-34O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-34O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-34O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-34O-2_GACATAGT-CGTCGCTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-34O-2_GACATAGT-CGTCGCTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214252\n",
      "  C: 0.290769\n",
      "  G: 0.279257\n",
      "  T: 0.215721\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 72,495 / 73,551 (98.564%)\n",
      "Discarded reads ...................: 0 / 73,551 (0.000%)\n",
      "Not assembled reads ...............: 1,056 / 73,551 (1.436%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-34O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-34O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-34O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-34O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-35O-1_GTAACGAG-TACGAGAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-35O-1_GTAACGAG-TACGAGAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213694\n",
      "  C: 0.290506\n",
      "  G: 0.281477\n",
      "  T: 0.214323\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 15,351 / 15,467 (99.250%)\n",
      "Discarded reads ...................: 0 / 15,467 (0.000%)\n",
      "Not assembled reads ...............: 116 / 15,467 (0.750%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-35O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-35O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-35O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-35O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-35O-2_ACGCTACT-CTAGAGCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-35O-2_ACGCTACT-CTAGAGCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213492\n",
      "  C: 0.291055\n",
      "  G: 0.281375\n",
      "  T: 0.214078\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 16,901 / 17,040 (99.184%)\n",
      "Discarded reads ...................: 0 / 17,040 (0.000%)\n",
      "Not assembled reads ...............: 139 / 17,040 (0.816%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-35O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-35O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-35O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-35O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-36O-1_GTCTATGA-TCATCGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-36O-1_GTCTATGA-TCATCGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216990\n",
      "  C: 0.287806\n",
      "  G: 0.277932\n",
      "  T: 0.217272\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 13,329 / 13,413 (99.374%)\n",
      "Discarded reads ...................: 0 / 13,413 (0.000%)\n",
      "Not assembled reads ...............: 84 / 13,413 (0.626%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-36O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-36O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-36O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-36O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-36O-2_ACGTGCGC-ACGTCTCG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-36O-2_ACGTGCGC-ACGTCTCG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216725\n",
      "  C: 0.287943\n",
      "  G: 0.278302\n",
      "  T: 0.217030\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 42,896 / 43,213 (99.266%)\n",
      "Discarded reads ...................: 0 / 43,213 (0.000%)\n",
      "Not assembled reads ...............: 317 / 43,213 (0.734%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-36O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-36O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-36O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-36O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-38M-1_ATAGTACC-TCGACGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-38M-1_ATAGTACC-TCGACGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216268\n",
      "  C: 0.288153\n",
      "  G: 0.278829\n",
      "  T: 0.216751\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 19,067 / 19,196 (99.328%)\n",
      "Discarded reads ...................: 0 / 19,196 (0.000%)\n",
      "Not assembled reads ...............: 129 / 19,196 (0.672%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-38M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-38M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-38M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-38M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-38M-2_TGAGTACG-GACACTGA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-38M-2_TGAGTACG-GACACTGA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215944\n",
      "  C: 0.288772\n",
      "  G: 0.278888\n",
      "  T: 0.216396\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 21,366 / 21,541 (99.188%)\n",
      "Discarded reads ...................: 0 / 21,541 (0.000%)\n",
      "Not assembled reads ...............: 175 / 21,541 (0.812%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-38M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-38M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-38M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-38M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-38O-1_ACTCACTG-GCTCTAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-38O-1_ACTCACTG-GCTCTAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217128\n",
      "  C: 0.288338\n",
      "  G: 0.275948\n",
      "  T: 0.218586\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 51,612 / 52,681 (97.971%)\n",
      "Discarded reads ...................: 0 / 52,681 (0.000%)\n",
      "Not assembled reads ...............: 1,069 / 52,681 (2.029%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-38O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-38O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-38O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-38O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-38O-2_TATAGCGA-CGTGAGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-38O-2_TATAGCGA-CGTGAGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217162\n",
      "  C: 0.288327\n",
      "  G: 0.275850\n",
      "  T: 0.218660\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 61%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 127,263 / 129,452 (98.309%)\n",
      "Discarded reads ...................: 0 / 129,452 (0.000%)\n",
      "Not assembled reads ...............: 2,189 / 129,452 (1.691%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-38O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-38O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-38O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-38O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-39M-1_CTCGACTT-GGATATCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-39M-1_CTCGACTT-GGATATCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214497\n",
      "  C: 0.290797\n",
      "  G: 0.279062\n",
      "  T: 0.215643\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,091 / 22,354 (98.823%)\n",
      "Discarded reads ...................: 0 / 22,354 (0.000%)\n",
      "Not assembled reads ...............: 263 / 22,354 (1.177%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-39M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-39M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-39M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-39M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-39M-2_GCGTATAC-GATCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-39M-2_GCGTATAC-GATCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214618\n",
      "  C: 0.290551\n",
      "  G: 0.279547\n",
      "  T: 0.215283\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 21,311 / 21,498 (99.130%)\n",
      "Discarded reads ...................: 0 / 21,498 (0.000%)\n",
      "Not assembled reads ...............: 187 / 21,498 (0.870%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-39M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-39M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-39M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-39M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-39O-1_CTGCGTAG-TGCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-39O-1_CTGCGTAG-TGCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218453\n",
      "  C: 0.286087\n",
      "  G: 0.276015\n",
      "  T: 0.219445\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 7,203 / 7,256 (99.270%)\n",
      "Discarded reads ...................: 0 / 7,256 (0.000%)\n",
      "Not assembled reads ...............: 53 / 7,256 (0.730%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-39O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-39O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-39O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-39O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-39O-2_CGAAGTAT-GACACCGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-39O-2_CGAAGTAT-GACACCGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218264\n",
      "  C: 0.286165\n",
      "  G: 0.276457\n",
      "  T: 0.219113\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 20,636 / 20,814 (99.145%)\n",
      "Discarded reads ...................: 0 / 20,814 (0.000%)\n",
      "Not assembled reads ...............: 178 / 20,814 (0.855%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-39O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-39O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-39O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-39O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-40M-1_TGCTCGTA-GTCAGATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-40M-1_TGCTCGTA-GTCAGATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217091\n",
      "  C: 0.287496\n",
      "  G: 0.277796\n",
      "  T: 0.217617\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 14,122 / 14,236 (99.199%)\n",
      "Discarded reads ...................: 0 / 14,236 (0.000%)\n",
      "Not assembled reads ...............: 114 / 14,236 (0.801%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-40M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-40M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-40M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-40M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-40M-2_TAGTCTCC-TAGTGTAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-40M-2_TAGTCTCC-TAGTGTAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214567\n",
      "  C: 0.290472\n",
      "  G: 0.279454\n",
      "  T: 0.215507\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 25,160 / 25,409 (99.020%)\n",
      "Discarded reads ...................: 0 / 25,409 (0.000%)\n",
      "Not assembled reads ...............: 249 / 25,409 (0.980%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-40M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-40M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-40M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-40M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-40O-1_TAGTCTCC-ATCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-40O-1_TAGTCTCC-ATCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.224271\n",
      "  C: 0.279681\n",
      "  G: 0.270832\n",
      "  T: 0.225215\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 25,427 / 25,624 (99.231%)\n",
      "Discarded reads ...................: 0 / 25,624 (0.000%)\n",
      "Not assembled reads ...............: 197 / 25,624 (0.769%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-40O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-40O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-40O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-40O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-40O-2_TAGCAGCT-CTACTATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-40O-2_TAGCAGCT-CTACTATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.225047\n",
      "  C: 0.279016\n",
      "  G: 0.270073\n",
      "  T: 0.225864\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 60%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 130,273 / 131,023 (99.428%)\n",
      "Discarded reads ...................: 0 / 131,023 (0.000%)\n",
      "Not assembled reads ...............: 750 / 131,023 (0.572%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-40O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-40O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-40O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-40O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-41M-1_AACGCTGA-ACGACGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-41M-1_AACGCTGA-ACGACGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214535\n",
      "  C: 0.290139\n",
      "  G: 0.280182\n",
      "  T: 0.215144\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 56%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 139,687 / 141,164 (98.954%)\n",
      "Discarded reads ...................: 0 / 141,164 (0.000%)\n",
      "Not assembled reads ...............: 1,477 / 141,164 (1.046%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-41M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-41M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-41M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-41M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-41M-2_CGAGCGAC-ACTATCTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-41M-2_CGAGCGAC-ACTATCTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214794\n",
      "  C: 0.289786\n",
      "  G: 0.280143\n",
      "  T: 0.215277\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 99%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 79,675 / 80,398 (99.101%)\n",
      "Discarded reads ...................: 0 / 80,398 (0.000%)\n",
      "Not assembled reads ...............: 723 / 80,398 (0.899%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-41M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-41M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-41M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-41M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-41O-1_TCTCTATG-CGTTACTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-41O-1_TCTCTATG-CGTTACTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.228005\n",
      "  C: 0.276346\n",
      "  G: 0.266897\n",
      "  T: 0.228751\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 44,459 / 44,782 (99.279%)\n",
      "Discarded reads ...................: 0 / 44,782 (0.000%)\n",
      "Not assembled reads ...............: 323 / 44,782 (0.721%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-41O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-41O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-41O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-41O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-41O-2_CGTAGCGA-ATATACAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-41O-2_CGTAGCGA-ATATACAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.227505\n",
      "  C: 0.277031\n",
      "  G: 0.267090\n",
      "  T: 0.228374\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 31,123 / 31,375 (99.197%)\n",
      "Discarded reads ...................: 0 / 31,375 (0.000%)\n",
      "Not assembled reads ...............: 252 / 31,375 (0.803%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-41O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-41O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-41O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-41O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-42M-1_CGAGAGTT-TAGCGAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-42M-1_CGAGAGTT-TAGCGAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214073\n",
      "  C: 0.290514\n",
      "  G: 0.280352\n",
      "  T: 0.215061\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 88%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 89,311 / 90,294 (98.911%)\n",
      "Discarded reads ...................: 0 / 90,294 (0.000%)\n",
      "Not assembled reads ...............: 983 / 90,294 (1.089%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-42M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-42M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-42M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-42M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-42M-2_ACTACGAC-AGAGTCAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-42M-2_ACTACGAC-AGAGTCAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213877\n",
      "  C: 0.290280\n",
      "  G: 0.281001\n",
      "  T: 0.214842\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 25,613 / 25,887 (98.942%)\n",
      "Discarded reads ...................: 0 / 25,887 (0.000%)\n",
      "Not assembled reads ...............: 274 / 25,887 (1.058%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-42M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-42M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-42M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-42M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-42O-1_GATCTACG-CGTCGCTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-42O-1_GATCTACG-CGTCGCTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.235602\n",
      "  C: 0.268642\n",
      "  G: 0.258467\n",
      "  T: 0.237289\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 10,564 / 10,641 (99.276%)\n",
      "Discarded reads ...................: 0 / 10,641 (0.000%)\n",
      "Not assembled reads ...............: 77 / 10,641 (0.724%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-42O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-42O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-42O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-42O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-42O-2_GACATAGT-CTGCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-42O-2_GACATAGT-CTGCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.231324\n",
      "  C: 0.273066\n",
      "  G: 0.262746\n",
      "  T: 0.232864\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 9,163 / 9,241 (99.156%)\n",
      "Discarded reads ...................: 0 / 9,241 (0.000%)\n",
      "Not assembled reads ...............: 78 / 9,241 (0.844%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-42O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-42O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-42O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-42O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-43M-1_CTCGACTT-GATCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-43M-1_CTCGACTT-GATCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215136\n",
      "  C: 0.289323\n",
      "  G: 0.279657\n",
      "  T: 0.215884\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 11,262 / 11,372 (99.033%)\n",
      "Discarded reads ...................: 0 / 11,372 (0.000%)\n",
      "Not assembled reads ...............: 110 / 11,372 (0.967%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-43M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-43M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-43M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-43M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-43M-2_GCGTATAC-TGCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-43M-2_GCGTATAC-TGCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215577\n",
      "  C: 0.288228\n",
      "  G: 0.279592\n",
      "  T: 0.216604\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 16,087 / 16,228 (99.131%)\n",
      "Discarded reads ...................: 0 / 16,228 (0.000%)\n",
      "Not assembled reads ...............: 141 / 16,228 (0.869%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-43M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-43M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-43M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-43M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-43O-1_GTAACGAG-CTAGAGCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-43O-1_GTAACGAG-CTAGAGCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218170\n",
      "  C: 0.286126\n",
      "  G: 0.276771\n",
      "  T: 0.218933\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 6,821 / 6,868 (99.316%)\n",
      "Discarded reads ...................: 0 / 6,868 (0.000%)\n",
      "Not assembled reads ...............: 47 / 6,868 (0.684%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-43O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-43O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-43O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-43O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-43O-2_ACGCTACT-TCATCGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-43O-2_ACGCTACT-TCATCGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217565\n",
      "  C: 0.286561\n",
      "  G: 0.277740\n",
      "  T: 0.218134\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 12,563 / 12,661 (99.226%)\n",
      "Discarded reads ...................: 0 / 12,661 (0.000%)\n",
      "Not assembled reads ...............: 98 / 12,661 (0.774%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-43O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-43O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-43O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-43O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-44M-1_GTCTATGA-ACGTCTCG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-44M-1_GTCTATGA-ACGTCTCG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215150\n",
      "  C: 0.289645\n",
      "  G: 0.278798\n",
      "  T: 0.216407\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 10,900 / 11,034 (98.786%)\n",
      "Discarded reads ...................: 0 / 11,034 (0.000%)\n",
      "Not assembled reads ...............: 134 / 11,034 (1.214%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-44M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-44M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-44M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-44M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-44M-2_TATAGCGA-TCGACGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-44M-2_TATAGCGA-TCGACGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217268\n",
      "  C: 0.286936\n",
      "  G: 0.277781\n",
      "  T: 0.218015\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 32,228 / 32,483 (99.215%)\n",
      "Discarded reads ...................: 0 / 32,483 (0.000%)\n",
      "Not assembled reads ...............: 255 / 32,483 (0.785%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-44M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-44M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-44M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-44M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-44O-1_ATAGTACC-GACACTGA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-44O-1_ATAGTACC-GACACTGA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217584\n",
      "  C: 0.286785\n",
      "  G: 0.277314\n",
      "  T: 0.218317\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 19,565 / 19,705 (99.290%)\n",
      "Discarded reads ...................: 0 / 19,705 (0.000%)\n",
      "Not assembled reads ...............: 140 / 19,705 (0.710%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-44O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-44O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-44O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-44O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-44O-2_TGAGTACG-GGATATCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-44O-2_TGAGTACG-GGATATCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216926\n",
      "  C: 0.288724\n",
      "  G: 0.276392\n",
      "  T: 0.217958\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 33,330 / 33,669 (98.993%)\n",
      "Discarded reads ...................: 0 / 33,669 (0.000%)\n",
      "Not assembled reads ...............: 339 / 33,669 (1.007%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-44O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-44O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-44O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-44O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-45O-1_CTGCGTAG-GACACCGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-45O-1_CTGCGTAG-GACACCGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.221256\n",
      "  C: 0.283605\n",
      "  G: 0.273358\n",
      "  T: 0.221780\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 37,421 / 37,685 (99.299%)\n",
      "Discarded reads ...................: 0 / 37,685 (0.000%)\n",
      "Not assembled reads ...............: 264 / 37,685 (0.701%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-45O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-45O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-45O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-45O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-45O-2_CGAAGTAT-GTCAGATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-45O-2_CGAAGTAT-GTCAGATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.219661\n",
      "  C: 0.285069\n",
      "  G: 0.274853\n",
      "  T: 0.220417\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 18,087 / 18,216 (99.292%)\n",
      "Discarded reads ...................: 0 / 18,216 (0.000%)\n",
      "Not assembled reads ...............: 129 / 18,216 (0.708%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-45O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-45O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-45O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-45O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-46M-1_TGCTCGTA-TAGTGTAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-46M-1_TGCTCGTA-TAGTGTAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216652\n",
      "  C: 0.288920\n",
      "  G: 0.276484\n",
      "  T: 0.217945\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 30,148 / 30,615 (98.475%)\n",
      "Discarded reads ...................: 0 / 30,615 (0.000%)\n",
      "Not assembled reads ...............: 467 / 30,615 (1.525%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-46M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-46M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-46M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-46M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-46M-2_TGCTCGTA-ATCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-46M-2_TGCTCGTA-ATCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216847\n",
      "  C: 0.287997\n",
      "  G: 0.277291\n",
      "  T: 0.217864\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 12,946 / 13,134 (98.569%)\n",
      "Discarded reads ...................: 0 / 13,134 (0.000%)\n",
      "Not assembled reads ...............: 188 / 13,134 (1.431%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-46M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-46M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-46M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-46M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-47M-1_TAGCAGCT-ACGACGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-47M-1_TAGCAGCT-ACGACGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215088\n",
      "  C: 0.289264\n",
      "  G: 0.280083\n",
      "  T: 0.215565\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 17,899 / 18,024 (99.306%)\n",
      "Discarded reads ...................: 0 / 18,024 (0.000%)\n",
      "Not assembled reads ...............: 125 / 18,024 (0.694%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-47M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-47M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-47M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-47M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-47M-2_AACGCTGA-ACTATCTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-47M-2_AACGCTGA-ACTATCTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214506\n",
      "  C: 0.289882\n",
      "  G: 0.280481\n",
      "  T: 0.215131\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 12,632 / 12,735 (99.191%)\n",
      "Discarded reads ...................: 0 / 12,735 (0.000%)\n",
      "Not assembled reads ...............: 103 / 12,735 (0.809%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-47M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-47M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-47M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-47M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-47O-1_CGAGCGAC-CGTTACTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-47O-1_CGAGCGAC-CGTTACTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217532\n",
      "  C: 0.286800\n",
      "  G: 0.277117\n",
      "  T: 0.218550\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,173 / 22,357 (99.177%)\n",
      "Discarded reads ...................: 0 / 22,357 (0.000%)\n",
      "Not assembled reads ...............: 184 / 22,357 (0.823%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-47O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-47O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-47O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-47O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-47O-2_TCTCTATG-ATATACAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-47O-2_TCTCTATG-ATATACAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217183\n",
      "  C: 0.287135\n",
      "  G: 0.277651\n",
      "  T: 0.218031\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,908 / 23,097 (99.182%)\n",
      "Discarded reads ...................: 0 / 23,097 (0.000%)\n",
      "Not assembled reads ...............: 189 / 23,097 (0.818%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-47O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-47O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-47O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-47O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-48O-1_CGTAGCGA-TAGCGAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-48O-1_CGTAGCGA-TAGCGAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216989\n",
      "  C: 0.287337\n",
      "  G: 0.277567\n",
      "  T: 0.218107\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 20,915 / 21,060 (99.311%)\n",
      "Discarded reads ...................: 0 / 21,060 (0.000%)\n",
      "Not assembled reads ...............: 145 / 21,060 (0.689%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-48O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-48O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-48O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-48O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-48O-2_CGAGAGTT-AGAGTCAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-48O-2_CGAGAGTT-AGAGTCAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217135\n",
      "  C: 0.287131\n",
      "  G: 0.277539\n",
      "  T: 0.218195\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 13,371 / 13,481 (99.184%)\n",
      "Discarded reads ...................: 0 / 13,481 (0.000%)\n",
      "Not assembled reads ...............: 110 / 13,481 (0.816%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-48O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-48O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-48O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-48O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-49M-1_ACTACGAC-CGTCGCTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-49M-1_ACTACGAC-CGTCGCTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.220183\n",
      "  C: 0.283408\n",
      "  G: 0.275593\n",
      "  T: 0.220815\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 8,583 / 8,637 (99.375%)\n",
      "Discarded reads ...................: 0 / 8,637 (0.000%)\n",
      "Not assembled reads ...............: 54 / 8,637 (0.625%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-49M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-49M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-49M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-49M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-49M-2_GATCTACG-CTGCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-49M-2_GATCTACG-CTGCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.220465\n",
      "  C: 0.283426\n",
      "  G: 0.275151\n",
      "  T: 0.220958\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 14,567 / 14,693 (99.142%)\n",
      "Discarded reads ...................: 0 / 14,693 (0.000%)\n",
      "Not assembled reads ...............: 126 / 14,693 (0.858%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-49M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-49M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-49M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-49M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-49O-1_GACATAGT-TACGAGAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-49O-1_GACATAGT-TACGAGAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214733\n",
      "  C: 0.289508\n",
      "  G: 0.280284\n",
      "  T: 0.215476\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 17,038 / 17,169 (99.237%)\n",
      "Discarded reads ...................: 0 / 17,169 (0.000%)\n",
      "Not assembled reads ...............: 131 / 17,169 (0.763%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-49O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-49O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-49O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-49O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-49O-2_GTCTGCTA-CTAGAGCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-49O-2_GTCTGCTA-CTAGAGCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214811\n",
      "  C: 0.289755\n",
      "  G: 0.279979\n",
      "  T: 0.215455\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 58,661 / 59,110 (99.240%)\n",
      "Discarded reads ...................: 0 / 59,110 (0.000%)\n",
      "Not assembled reads ...............: 449 / 59,110 (0.760%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-49O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-49O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-49O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-49O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-50O-1_GTAACGAG-TCATCGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-50O-1_GTAACGAG-TCATCGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218438\n",
      "  C: 0.285841\n",
      "  G: 0.276937\n",
      "  T: 0.218784\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 9,447 / 9,506 (99.379%)\n",
      "Discarded reads ...................: 0 / 9,506 (0.000%)\n",
      "Not assembled reads ...............: 59 / 9,506 (0.621%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-50O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-50O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-50O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-50O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-50O-2_ACGCTACT-ACGTCTCG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-50O-2_ACGCTACT-ACGTCTCG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217237\n",
      "  C: 0.287669\n",
      "  G: 0.276947\n",
      "  T: 0.218147\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 4,944 / 5,002 (98.840%)\n",
      "Discarded reads ...................: 0 / 5,002 (0.000%)\n",
      "Not assembled reads ...............: 58 / 5,002 (1.160%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-50O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-50O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-50O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-50O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-51O-1_GTCTATGA-GCTCTAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-51O-1_GTCTATGA-GCTCTAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.222917\n",
      "  C: 0.281812\n",
      "  G: 0.272120\n",
      "  T: 0.223150\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 14,182 / 14,273 (99.362%)\n",
      "Discarded reads ...................: 0 / 14,273 (0.000%)\n",
      "Not assembled reads ...............: 91 / 14,273 (0.638%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-51O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-51O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-51O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-51O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-51O-2_ACGTGCGC-CGTGAGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-51O-2_ACGTGCGC-CGTGAGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.222663\n",
      "  C: 0.281980\n",
      "  G: 0.272256\n",
      "  T: 0.223101\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 9,567 / 9,635 (99.294%)\n",
      "Discarded reads ...................: 0 / 9,635 (0.000%)\n",
      "Not assembled reads ...............: 68 / 9,635 (0.706%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-51O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-51O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-51O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-51O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-52O-1_ACTCACTG-TCGACGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-52O-1_ACTCACTG-TCGACGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.219070\n",
      "  C: 0.285162\n",
      "  G: 0.276254\n",
      "  T: 0.219515\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 10,707 / 10,795 (99.185%)\n",
      "Discarded reads ...................: 0 / 10,795 (0.000%)\n",
      "Not assembled reads ...............: 88 / 10,795 (0.815%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-52O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-52O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-52O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-52O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-52O-2_TATAGCGA-GACACTGA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-52O-2_TATAGCGA-GACACTGA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.219141\n",
      "  C: 0.285105\n",
      "  G: 0.275974\n",
      "  T: 0.219780\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 11,307 / 11,382 (99.341%)\n",
      "Discarded reads ...................: 0 / 11,382 (0.000%)\n",
      "Not assembled reads ...............: 75 / 11,382 (0.659%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-52O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-52O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-52O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-52O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U06O-1_GTCTATGA-CGTGAGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U06O-1_GTCTATGA-CGTGAGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212840\n",
      "  C: 0.291973\n",
      "  G: 0.281828\n",
      "  T: 0.213359\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 9,437 / 9,507 (99.264%)\n",
      "Discarded reads ...................: 0 / 9,507 (0.000%)\n",
      "Not assembled reads ...............: 70 / 9,507 (0.736%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U06O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U06O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U06O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U06O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U06O-2_ACGTGCGC-TCGACGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U06O-2_ACGTGCGC-TCGACGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.210988\n",
      "  C: 0.293526\n",
      "  G: 0.284176\n",
      "  T: 0.211310\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,871 / 23,063 (99.167%)\n",
      "Discarded reads ...................: 0 / 23,063 (0.000%)\n",
      "Not assembled reads ...............: 192 / 23,063 (0.833%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U06O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U06O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U06O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U06O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U07M-1_TATAGCGA-GGATATCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U07M-1_TATAGCGA-GGATATCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212192\n",
      "  C: 0.293308\n",
      "  G: 0.280845\n",
      "  T: 0.213655\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 5,779 / 5,845 (98.871%)\n",
      "Discarded reads ...................: 0 / 5,845 (0.000%)\n",
      "Not assembled reads ...............: 66 / 5,845 (1.129%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U07M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U07M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U07M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U07M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U07M-2_ATAGTACC-GATCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U07M-2_ATAGTACC-GATCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212389\n",
      "  C: 0.292305\n",
      "  G: 0.282270\n",
      "  T: 0.213036\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 16,313 / 16,424 (99.324%)\n",
      "Discarded reads ...................: 0 / 16,424 (0.000%)\n",
      "Not assembled reads ...............: 111 / 16,424 (0.676%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U07M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U07M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U07M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U07M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U07O-1_TGAGTACG-TGCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U07O-1_TGAGTACG-TGCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216638\n",
      "  C: 0.287791\n",
      "  G: 0.278453\n",
      "  T: 0.217118\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 16,475 / 16,589 (99.313%)\n",
      "Discarded reads ...................: 0 / 16,589 (0.000%)\n",
      "Not assembled reads ...............: 114 / 16,589 (0.687%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U07O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U07O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U07O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U07O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U07O-2_CTCGACTT-GACACCGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U07O-2_CTCGACTT-GACACCGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216481\n",
      "  C: 0.287837\n",
      "  G: 0.278597\n",
      "  T: 0.217084\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 14,213 / 14,345 (99.080%)\n",
      "Discarded reads ...................: 0 / 14,345 (0.000%)\n",
      "Not assembled reads ...............: 132 / 14,345 (0.920%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U07O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U07O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U07O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U07O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U08M-1_GCGTATAC-GTCAGATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U08M-1_GCGTATAC-GTCAGATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214324\n",
      "  C: 0.290186\n",
      "  G: 0.280542\n",
      "  T: 0.214948\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 17,028 / 17,157 (99.248%)\n",
      "Discarded reads ...................: 0 / 17,157 (0.000%)\n",
      "Not assembled reads ...............: 129 / 17,157 (0.752%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U08M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U08M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U08M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U08M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U08M-2_CTGCGTAG-TAGTGTAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U08M-2_CTGCGTAG-TAGTGTAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214258\n",
      "  C: 0.291072\n",
      "  G: 0.279538\n",
      "  T: 0.215132\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 21,351 / 21,541 (99.118%)\n",
      "Discarded reads ...................: 0 / 21,541 (0.000%)\n",
      "Not assembled reads ...............: 190 / 21,541 (0.882%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U08M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U08M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U08M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U08M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U08O-1_CTGCGTAG-ATCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U08O-1_CTGCGTAG-ATCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214865\n",
      "  C: 0.289803\n",
      "  G: 0.279702\n",
      "  T: 0.215629\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 9,784 / 9,859 (99.239%)\n",
      "Discarded reads ...................: 0 / 9,859 (0.000%)\n",
      "Not assembled reads ...............: 75 / 9,859 (0.761%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U08O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U08O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U08O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U08O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U08O-2_CGAAGTAT-CTACTATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U08O-2_CGAAGTAT-CTACTATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215003\n",
      "  C: 0.289383\n",
      "  G: 0.279809\n",
      "  T: 0.215805\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 23,368 / 23,599 (99.021%)\n",
      "Discarded reads ...................: 0 / 23,599 (0.000%)\n",
      "Not assembled reads ...............: 231 / 23,599 (0.979%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U08O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U08O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U08O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U08O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U09M-1_TGCTCGTA-ACGACGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U09M-1_TGCTCGTA-ACGACGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212165\n",
      "  C: 0.292087\n",
      "  G: 0.283200\n",
      "  T: 0.212548\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 8,851 / 8,891 (99.550%)\n",
      "Discarded reads ...................: 0 / 8,891 (0.000%)\n",
      "Not assembled reads ...............: 40 / 8,891 (0.450%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U09M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U09M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U09M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U09M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U09M-2_TAGTCTCC-ACTATCTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U09M-2_TAGTCTCC-ACTATCTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.211912\n",
      "  C: 0.292277\n",
      "  G: 0.283423\n",
      "  T: 0.212388\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 15,898 / 15,965 (99.580%)\n",
      "Discarded reads ...................: 0 / 15,965 (0.000%)\n",
      "Not assembled reads ...............: 67 / 15,965 (0.420%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U09M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U09M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U09M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U09M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U09O-1_TAGCAGCT-CGTTACTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U09O-1_TAGCAGCT-CGTTACTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217573\n",
      "  C: 0.286732\n",
      "  G: 0.277527\n",
      "  T: 0.218168\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 18,799 / 18,908 (99.424%)\n",
      "Discarded reads ...................: 0 / 18,908 (0.000%)\n",
      "Not assembled reads ...............: 109 / 18,908 (0.576%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U09O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U09O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U09O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U09O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U09O-2_AACGCTGA-ATATACAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U09O-2_AACGCTGA-ATATACAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217541\n",
      "  C: 0.286856\n",
      "  G: 0.277431\n",
      "  T: 0.218172\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 24,434 / 24,581 (99.402%)\n",
      "Discarded reads ...................: 0 / 24,581 (0.000%)\n",
      "Not assembled reads ...............: 147 / 24,581 (0.598%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U09O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U09O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U09O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U09O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U10M-1_CGTAGCGA-CGTCGCTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U10M-1_CGTAGCGA-CGTCGCTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.211840\n",
      "  C: 0.292757\n",
      "  G: 0.282831\n",
      "  T: 0.212572\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 15,585 / 15,704 (99.242%)\n",
      "Discarded reads ...................: 0 / 15,704 (0.000%)\n",
      "Not assembled reads ...............: 119 / 15,704 (0.758%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U10M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U10M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U10M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U10M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U10M-2_CGAGAGTT-CTGCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U10M-2_CGAGAGTT-CTGCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212317\n",
      "  C: 0.292293\n",
      "  G: 0.282441\n",
      "  T: 0.212949\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 16,878 / 17,027 (99.125%)\n",
      "Discarded reads ...................: 0 / 17,027 (0.000%)\n",
      "Not assembled reads ...............: 149 / 17,027 (0.875%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U10M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U10M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U10M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U10M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U10O-1_ACTACGAC-TACGAGAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U10O-1_ACTACGAC-TACGAGAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214454\n",
      "  C: 0.289537\n",
      "  G: 0.280914\n",
      "  T: 0.215094\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 34,173 / 34,376 (99.409%)\n",
      "Discarded reads ...................: 0 / 34,376 (0.000%)\n",
      "Not assembled reads ...............: 203 / 34,376 (0.591%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U10O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U10O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U10O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U10O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U10O-2_GATCTACG-CTAGAGCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U10O-2_GATCTACG-CTAGAGCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214833\n",
      "  C: 0.289569\n",
      "  G: 0.280286\n",
      "  T: 0.215311\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 35,067 / 35,305 (99.326%)\n",
      "Discarded reads ...................: 0 / 35,305 (0.000%)\n",
      "Not assembled reads ...............: 238 / 35,305 (0.674%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U10O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U10O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U10O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U10O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U11O-1_GACATAGT-TCATCGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U11O-1_GACATAGT-TCATCGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215269\n",
      "  C: 0.288994\n",
      "  G: 0.280168\n",
      "  T: 0.215569\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 31,923 / 32,095 (99.464%)\n",
      "Discarded reads ...................: 0 / 32,095 (0.000%)\n",
      "Not assembled reads ...............: 172 / 32,095 (0.536%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U11O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U11O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U11O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U11O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U11O-2_GTCTGCTA-ACGTCTCG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U11O-2_GTCTGCTA-ACGTCTCG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216267\n",
      "  C: 0.288351\n",
      "  G: 0.278537\n",
      "  T: 0.216844\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 18,171 / 18,288 (99.360%)\n",
      "Discarded reads ...................: 0 / 18,288 (0.000%)\n",
      "Not assembled reads ...............: 117 / 18,288 (0.640%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U11O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U11O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U11O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U11O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U12O-1_GTAACGAG-GCTCTAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U12O-1_GTAACGAG-GCTCTAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215950\n",
      "  C: 0.288197\n",
      "  G: 0.279592\n",
      "  T: 0.216261\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 21,617 / 21,732 (99.471%)\n",
      "Discarded reads ...................: 0 / 21,732 (0.000%)\n",
      "Not assembled reads ...............: 115 / 21,732 (0.529%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U12O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U12O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U12O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U12O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-NT-U12O-2_ACGCTACT-CGTGAGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-NT-U12O-2_ACGCTACT-CGTGAGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213887\n",
      "  C: 0.290491\n",
      "  G: 0.281093\n",
      "  T: 0.214529\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 13,760 / 13,870 (99.207%)\n",
      "Discarded reads ...................: 0 / 13,870 (0.000%)\n",
      "Not assembled reads ...............: 110 / 13,870 (0.793%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U12O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-NT-U12O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U12O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-NT-U12O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-01M-1-10x_CTGCGTAG-CTACTATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-01M-1-10x_CTGCGTAG-CTACTATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212381\n",
      "  C: 0.292230\n",
      "  G: 0.282534\n",
      "  T: 0.212855\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 33,294 / 33,652 (98.936%)\n",
      "Discarded reads ...................: 0 / 33,652 (0.000%)\n",
      "Not assembled reads ...............: 358 / 33,652 (1.064%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-01M-1-10x.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-01M-1-10x.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-01M-1-10x.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-01M-1-10x.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-01M-2_ACTACGAC-CTACTATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-01M-2_ACTACGAC-CTACTATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.210809\n",
      "  C: 0.293387\n",
      "  G: 0.284588\n",
      "  T: 0.211216\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 23,353 / 23,586 (99.012%)\n",
      "Discarded reads ...................: 0 / 23,586 (0.000%)\n",
      "Not assembled reads ...............: 233 / 23,586 (0.988%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-01M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-01M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-01M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-01M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-01O-1-10x_CGAAGTAT-ACGACGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-01O-1-10x_CGAAGTAT-ACGACGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217537\n",
      "  C: 0.287230\n",
      "  G: 0.276494\n",
      "  T: 0.218739\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 27,179 / 27,514 (98.782%)\n",
      "Discarded reads ...................: 0 / 27,514 (0.000%)\n",
      "Not assembled reads ...............: 335 / 27,514 (1.218%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-01O-1-10x.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-01O-1-10x.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-01O-1-10x.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-01O-1-10x.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-01O-2-10x_TGCTCGTA-ACTATCTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-01O-2-10x_TGCTCGTA-ACTATCTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217429\n",
      "  C: 0.286821\n",
      "  G: 0.277475\n",
      "  T: 0.218275\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,345 / 22,554 (99.073%)\n",
      "Discarded reads ...................: 0 / 22,554 (0.000%)\n",
      "Not assembled reads ...............: 209 / 22,554 (0.927%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-01O-2-10x.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-01O-2-10x.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-01O-2-10x.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-01O-2-10x.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-02O-1_GTCTGCTA-CGTTACTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-02O-1_GTCTGCTA-CGTTACTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218108\n",
      "  C: 0.286235\n",
      "  G: 0.276894\n",
      "  T: 0.218763\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 72,970 / 73,407 (99.405%)\n",
      "Discarded reads ...................: 0 / 73,407 (0.000%)\n",
      "Not assembled reads ...............: 437 / 73,407 (0.595%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-02O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-02O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-02O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-02O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-02O-2_GTAACGAG-ATATACAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-02O-2_GTAACGAG-ATATACAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217456\n",
      "  C: 0.286888\n",
      "  G: 0.277798\n",
      "  T: 0.217857\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 41,496 / 41,723 (99.456%)\n",
      "Discarded reads ...................: 0 / 41,723 (0.000%)\n",
      "Not assembled reads ...............: 227 / 41,723 (0.544%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-02O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-02O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-02O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-02O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-04M-1_ACGCTACT-TAGCGAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-04M-1_ACGCTACT-TAGCGAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.224868\n",
      "  C: 0.279100\n",
      "  G: 0.270176\n",
      "  T: 0.225856\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 37,164 / 37,496 (99.115%)\n",
      "Discarded reads ...................: 0 / 37,496 (0.000%)\n",
      "Not assembled reads ...............: 332 / 37,496 (0.885%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-04M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-04M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-04M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-04M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-04M-2_GTCTATGA-AGAGTCAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-04M-2_GTCTATGA-AGAGTCAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.224620\n",
      "  C: 0.279544\n",
      "  G: 0.270058\n",
      "  T: 0.225778\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 34,321 / 34,618 (99.142%)\n",
      "Discarded reads ...................: 0 / 34,618 (0.000%)\n",
      "Not assembled reads ...............: 297 / 34,618 (0.858%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-04M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-04M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-04M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-04M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-05M-1_ACGTGCGC-CGTCGCTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-05M-1_ACGTGCGC-CGTCGCTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215413\n",
      "  C: 0.289095\n",
      "  G: 0.279787\n",
      "  T: 0.215705\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 95%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 83,279 / 83,872 (99.293%)\n",
      "Discarded reads ...................: 0 / 83,872 (0.000%)\n",
      "Not assembled reads ...............: 593 / 83,872 (0.707%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-05M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-05M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-05M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-05M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-05M-2_ACTCACTG-CTGCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-05M-2_ACTCACTG-CTGCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.203132\n",
      "  C: 0.319251\n",
      "  G: 0.260707\n",
      "  T: 0.216910\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 57 / 72 (79.167%)\n",
      "Discarded reads ...................: 0 / 72 (0.000%)\n",
      "Not assembled reads ...............: 15 / 72 (20.833%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-05M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-05M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-05M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-05M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-05O-1_TATAGCGA-TACGAGAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-05O-1_TATAGCGA-TACGAGAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218706\n",
      "  C: 0.285834\n",
      "  G: 0.275393\n",
      "  T: 0.220067\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 62,670 / 63,358 (98.914%)\n",
      "Discarded reads ...................: 0 / 63,358 (0.000%)\n",
      "Not assembled reads ...............: 688 / 63,358 (1.086%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-05O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-05O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-05O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-05O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-05O-2_ATAGTACC-CTAGAGCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-05O-2_ATAGTACC-CTAGAGCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.204746\n",
      "  C: 0.311363\n",
      "  G: 0.273688\n",
      "  T: 0.210203\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22 / 23 (95.652%)\n",
      "Discarded reads ...................: 0 / 23 (0.000%)\n",
      "Not assembled reads ...............: 1 / 23 (4.348%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-05O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-05O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-05O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-05O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-06M-1_TGAGTACG-TCATCGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-06M-1_TGAGTACG-TCATCGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214286\n",
      "  C: 0.290410\n",
      "  G: 0.280989\n",
      "  T: 0.214315\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 53,761 / 54,102 (99.370%)\n",
      "Discarded reads ...................: 0 / 54,102 (0.000%)\n",
      "Not assembled reads ...............: 341 / 54,102 (0.630%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-06M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-06M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-06M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-06M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-06M-2_CTCGACTT-ACGTCTCG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-06M-2_CTCGACTT-ACGTCTCG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213410\n",
      "  C: 0.291486\n",
      "  G: 0.281348\n",
      "  T: 0.213756\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 45,110 / 45,544 (99.047%)\n",
      "Discarded reads ...................: 0 / 45,544 (0.000%)\n",
      "Not assembled reads ...............: 434 / 45,544 (0.953%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-06M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-06M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-06M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-06M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-06O-1_GCGTATAC-GCTCTAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-06O-1_GCGTATAC-GCTCTAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216625\n",
      "  C: 0.287897\n",
      "  G: 0.278618\n",
      "  T: 0.216860\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 38,385 / 38,695 (99.199%)\n",
      "Discarded reads ...................: 0 / 38,695 (0.000%)\n",
      "Not assembled reads ...............: 310 / 38,695 (0.801%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-06O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-06O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-06O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-06O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-06O-2_CTGCGTAG-CGTGAGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-06O-2_CTGCGTAG-CGTGAGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216511\n",
      "  C: 0.288354\n",
      "  G: 0.278343\n",
      "  T: 0.216792\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 29,712 / 29,945 (99.222%)\n",
      "Discarded reads ...................: 0 / 29,945 (0.000%)\n",
      "Not assembled reads ...............: 233 / 29,945 (0.778%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-06O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-06O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-06O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-06O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-07M-1_CGAAGTAT-TCGACGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-07M-1_CGAAGTAT-TCGACGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.209574\n",
      "  C: 0.294857\n",
      "  G: 0.284979\n",
      "  T: 0.210590\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,343 / 22,578 (98.959%)\n",
      "Discarded reads ...................: 0 / 22,578 (0.000%)\n",
      "Not assembled reads ...............: 235 / 22,578 (1.041%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-07M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-07M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-07M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-07M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-07M-2_TGCTCGTA-GACACTGA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-07M-2_TGCTCGTA-GACACTGA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.209963\n",
      "  C: 0.294174\n",
      "  G: 0.285153\n",
      "  T: 0.210710\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 50,782 / 51,233 (99.120%)\n",
      "Discarded reads ...................: 0 / 51,233 (0.000%)\n",
      "Not assembled reads ...............: 451 / 51,233 (0.880%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-07M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-07M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-07M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-07M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-07O-1_TAGTCTCC-GGATATCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-07O-1_TAGTCTCC-GGATATCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.206300\n",
      "  C: 0.308516\n",
      "  G: 0.266559\n",
      "  T: 0.218625\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 29 / 32 (90.625%)\n",
      "Discarded reads ...................: 0 / 32 (0.000%)\n",
      "Not assembled reads ...............: 3 / 32 (9.375%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-07O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-07O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-07O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-07O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-07O-2_TAGCAGCT-GATCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-07O-2_TAGCAGCT-GATCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213520\n",
      "  C: 0.291337\n",
      "  G: 0.280601\n",
      "  T: 0.214541\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 55%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 141,460 / 143,104 (98.851%)\n",
      "Discarded reads ...................: 0 / 143,104 (0.000%)\n",
      "Not assembled reads ...............: 1,644 / 143,104 (1.149%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-07O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-07O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-07O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-07O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-08M-1_AACGCTGA-TGCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-08M-1_AACGCTGA-TGCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.205439\n",
      "  C: 0.307119\n",
      "  G: 0.272215\n",
      "  T: 0.215226\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 20 / 23 (86.957%)\n",
      "Discarded reads ...................: 0 / 23 (0.000%)\n",
      "Not assembled reads ...............: 3 / 23 (13.043%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-08M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-08M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-08M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-08M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-08M-2_CGAGCGAC-GACACCGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-08M-2_CGAGCGAC-GACACCGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214422\n",
      "  C: 0.290157\n",
      "  G: 0.279736\n",
      "  T: 0.215685\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 43,586 / 44,233 (98.537%)\n",
      "Discarded reads ...................: 0 / 44,233 (0.000%)\n",
      "Not assembled reads ...............: 647 / 44,233 (1.463%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-08M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-08M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-08M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-08M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-08O-1_TCTCTATG-GTCAGATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-08O-1_TCTCTATG-GTCAGATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215124\n",
      "  C: 0.290046\n",
      "  G: 0.278666\n",
      "  T: 0.216164\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 33,589 / 34,062 (98.611%)\n",
      "Discarded reads ...................: 0 / 34,062 (0.000%)\n",
      "Not assembled reads ...............: 473 / 34,062 (1.389%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-08O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-08O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-08O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-08O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-08O-2_CGTAGCGA-TAGTGTAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-08O-2_CGTAGCGA-TAGTGTAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214398\n",
      "  C: 0.291546\n",
      "  G: 0.277835\n",
      "  T: 0.216221\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 69,833 / 71,146 (98.154%)\n",
      "Discarded reads ...................: 0 / 71,146 (0.000%)\n",
      "Not assembled reads ...............: 1,313 / 71,146 (1.846%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-08O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-08O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-08O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-08O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-09M-1-R_ACGTGCGC-GACACTGA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-09M-1-R_ACGTGCGC-GACACTGA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215151\n",
      "  C: 0.289127\n",
      "  G: 0.279733\n",
      "  T: 0.215989\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 8,080 / 8,157 (99.056%)\n",
      "Discarded reads ...................: 0 / 8,157 (0.000%)\n",
      "Not assembled reads ...............: 77 / 8,157 (0.944%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-09M-1-R.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-09M-1-R.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-09M-1-R.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-09M-1-R.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-09M-2-R_ACTCACTG-GGATATCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-09M-2-R_ACTCACTG-GGATATCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214468\n",
      "  C: 0.290615\n",
      "  G: 0.279033\n",
      "  T: 0.215885\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 35,211 / 35,722 (98.570%)\n",
      "Discarded reads ...................: 0 / 35,722 (0.000%)\n",
      "Not assembled reads ...............: 511 / 35,722 (1.430%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-09M-2-R.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-09M-2-R.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-09M-2-R.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-09M-2-R.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-09O-1-R_TATAGCGA-GATCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-09O-1-R_TATAGCGA-GATCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215566\n",
      "  C: 0.289089\n",
      "  G: 0.278730\n",
      "  T: 0.216615\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 27,731 / 28,011 (99.000%)\n",
      "Discarded reads ...................: 0 / 28,011 (0.000%)\n",
      "Not assembled reads ...............: 280 / 28,011 (1.000%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-09O-1-R.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-09O-1-R.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-09O-1-R.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-09O-1-R.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-09O-2-R_ATAGTACC-TGCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-09O-2-R_ATAGTACC-TGCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215372\n",
      "  C: 0.288506\n",
      "  G: 0.279565\n",
      "  T: 0.216557\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 20,027 / 20,236 (98.967%)\n",
      "Discarded reads ...................: 0 / 20,236 (0.000%)\n",
      "Not assembled reads ...............: 209 / 20,236 (1.033%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-09O-2-R.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-09O-2-R.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-09O-2-R.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-09O-2-R.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-10M-1_GTCTATGA-CGTCGCTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-10M-1_GTCTATGA-CGTCGCTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215052\n",
      "  C: 0.289899\n",
      "  G: 0.279517\n",
      "  T: 0.215532\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 37,729 / 38,077 (99.086%)\n",
      "Discarded reads ...................: 0 / 38,077 (0.000%)\n",
      "Not assembled reads ...............: 348 / 38,077 (0.914%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-10M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-10M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-10M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-10M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-10M-2_ACGTGCGC-CTGCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-10M-2_ACGTGCGC-CTGCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215258\n",
      "  C: 0.289409\n",
      "  G: 0.279764\n",
      "  T: 0.215570\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 47,808 / 48,258 (99.068%)\n",
      "Discarded reads ...................: 0 / 48,258 (0.000%)\n",
      "Not assembled reads ...............: 450 / 48,258 (0.932%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-10M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-10M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-10M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-10M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-10O-1_ACTCACTG-TACGAGAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-10O-1_ACTCACTG-TACGAGAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215837\n",
      "  C: 0.288918\n",
      "  G: 0.278096\n",
      "  T: 0.217149\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 18,341 / 18,626 (98.470%)\n",
      "Discarded reads ...................: 0 / 18,626 (0.000%)\n",
      "Not assembled reads ...............: 285 / 18,626 (1.530%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-10O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-10O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-10O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-10O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-10O-2_TATAGCGA-CTAGAGCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-10O-2_TATAGCGA-CTAGAGCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216103\n",
      "  C: 0.288938\n",
      "  G: 0.277712\n",
      "  T: 0.217247\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 21,062 / 21,331 (98.739%)\n",
      "Discarded reads ...................: 0 / 21,331 (0.000%)\n",
      "Not assembled reads ...............: 269 / 21,331 (1.261%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-10O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-10O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-10O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-10O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-11M-1_ATAGTACC-TCATCGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-11M-1_ATAGTACC-TCATCGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.221316\n",
      "  C: 0.282978\n",
      "  G: 0.273926\n",
      "  T: 0.221780\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 71,865 / 72,228 (99.497%)\n",
      "Discarded reads ...................: 0 / 72,228 (0.000%)\n",
      "Not assembled reads ...............: 363 / 72,228 (0.503%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-11M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-11M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-11M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-11M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-11M-2_TGAGTACG-ACGTCTCG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-11M-2_TGAGTACG-ACGTCTCG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.219950\n",
      "  C: 0.285251\n",
      "  G: 0.274402\n",
      "  T: 0.220398\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 58,350 / 58,852 (99.147%)\n",
      "Discarded reads ...................: 0 / 58,852 (0.000%)\n",
      "Not assembled reads ...............: 502 / 58,852 (0.853%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-11M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-11M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-11M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-11M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-11O-1_CTCGACTT-GCTCTAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-11O-1_CTCGACTT-GCTCTAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.220864\n",
      "  C: 0.283578\n",
      "  G: 0.273958\n",
      "  T: 0.221600\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 25,084 / 25,243 (99.370%)\n",
      "Discarded reads ...................: 0 / 25,243 (0.000%)\n",
      "Not assembled reads ...............: 159 / 25,243 (0.630%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-11O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-11O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-11O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-11O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-11O-2_GCGTATAC-CGTGAGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-11O-2_GCGTATAC-CGTGAGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.221423\n",
      "  C: 0.283164\n",
      "  G: 0.273026\n",
      "  T: 0.222388\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 26,223 / 26,393 (99.356%)\n",
      "Discarded reads ...................: 0 / 26,393 (0.000%)\n",
      "Not assembled reads ...............: 170 / 26,393 (0.644%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-11O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-11O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-11O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-11O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-12M-1_CTGCGTAG-TCGACGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-12M-1_CTGCGTAG-TCGACGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217702\n",
      "  C: 0.286978\n",
      "  G: 0.276818\n",
      "  T: 0.218502\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 86%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 91,341 / 92,438 (98.813%)\n",
      "Discarded reads ...................: 0 / 92,438 (0.000%)\n",
      "Not assembled reads ...............: 1,097 / 92,438 (1.187%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-12M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-12M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-12M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-12M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-12M-2_CGAAGTAT-GACACTGA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-12M-2_CGAAGTAT-GACACTGA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217930\n",
      "  C: 0.286372\n",
      "  G: 0.276688\n",
      "  T: 0.219010\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 38,305 / 38,737 (98.885%)\n",
      "Discarded reads ...................: 0 / 38,737 (0.000%)\n",
      "Not assembled reads ...............: 432 / 38,737 (1.115%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-12M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-12M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-12M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-12M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-12O-1_TGCTCGTA-GGATATCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-12O-1_TGCTCGTA-GGATATCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215021\n",
      "  C: 0.290900\n",
      "  G: 0.277408\n",
      "  T: 0.216671\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 71%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 110,065 / 111,887 (98.372%)\n",
      "Discarded reads ...................: 0 / 111,887 (0.000%)\n",
      "Not assembled reads ...............: 1,822 / 111,887 (1.628%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-12O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-12O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-12O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-12O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-12O-2_TAGTCTCC-GATCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-12O-2_TAGTCTCC-GATCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216656\n",
      "  C: 0.288616\n",
      "  G: 0.277090\n",
      "  T: 0.217637\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 23,531 / 23,786 (98.928%)\n",
      "Discarded reads ...................: 0 / 23,786 (0.000%)\n",
      "Not assembled reads ...............: 255 / 23,786 (1.072%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-12O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-12O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-12O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-12O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-13O-1_TAGCAGCT-TGCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-13O-1_TAGCAGCT-TGCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218997\n",
      "  C: 0.285558\n",
      "  G: 0.275304\n",
      "  T: 0.220142\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 24,624 / 24,896 (98.907%)\n",
      "Discarded reads ...................: 0 / 24,896 (0.000%)\n",
      "Not assembled reads ...............: 272 / 24,896 (1.093%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-13O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-13O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-13O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-13O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-13O-2_AACGCTGA-GACACCGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-13O-2_AACGCTGA-GACACCGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.219083\n",
      "  C: 0.285442\n",
      "  G: 0.275463\n",
      "  T: 0.220012\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 19,867 / 20,046 (99.107%)\n",
      "Discarded reads ...................: 0 / 20,046 (0.000%)\n",
      "Not assembled reads ...............: 179 / 20,046 (0.893%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-13O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-13O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-13O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-13O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-14M-1_CGAGCGAC-GTCAGATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-14M-1_CGAGCGAC-GTCAGATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217719\n",
      "  C: 0.286733\n",
      "  G: 0.277166\n",
      "  T: 0.218382\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 32,872 / 33,166 (99.114%)\n",
      "Discarded reads ...................: 0 / 33,166 (0.000%)\n",
      "Not assembled reads ...............: 294 / 33,166 (0.886%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-14M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-14M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-14M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-14M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-14M-2_TCTCTATG-TAGTGTAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-14M-2_TCTCTATG-TAGTGTAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218160\n",
      "  C: 0.286900\n",
      "  G: 0.275831\n",
      "  T: 0.219110\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,860 / 23,040 (99.219%)\n",
      "Discarded reads ...................: 0 / 23,040 (0.000%)\n",
      "Not assembled reads ...............: 180 / 23,040 (0.781%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-14M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-14M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-14M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-14M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-14O-1_TCTCTATG-ATCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-14O-1_TCTCTATG-ATCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.239454\n",
      "  C: 0.264627\n",
      "  G: 0.254924\n",
      "  T: 0.240995\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 21,709 / 21,840 (99.400%)\n",
      "Discarded reads ...................: 0 / 21,840 (0.000%)\n",
      "Not assembled reads ...............: 131 / 21,840 (0.600%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-14O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-14O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-14O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-14O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-14O-2_CGTAGCGA-CTACTATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-14O-2_CGTAGCGA-CTACTATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.238263\n",
      "  C: 0.266027\n",
      "  G: 0.255655\n",
      "  T: 0.240054\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 37,847 / 38,065 (99.427%)\n",
      "Discarded reads ...................: 0 / 38,065 (0.000%)\n",
      "Not assembled reads ...............: 218 / 38,065 (0.573%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-14O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-14O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-14O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-14O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-15M-1_CGAGAGTT-ACGACGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-15M-1_CGAGAGTT-ACGACGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214562\n",
      "  C: 0.290082\n",
      "  G: 0.280458\n",
      "  T: 0.214899\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 50,303 / 50,745 (99.129%)\n",
      "Discarded reads ...................: 0 / 50,745 (0.000%)\n",
      "Not assembled reads ...............: 442 / 50,745 (0.871%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-15M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-15M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-15M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-15M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-15M-2_ACTACGAC-ACTATCTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-15M-2_ACTACGAC-ACTATCTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214088\n",
      "  C: 0.290319\n",
      "  G: 0.281223\n",
      "  T: 0.214370\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 27,508 / 27,693 (99.332%)\n",
      "Discarded reads ...................: 0 / 27,693 (0.000%)\n",
      "Not assembled reads ...............: 185 / 27,693 (0.668%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-15M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-15M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-15M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-15M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-15O-1_GACATAGT-ATATACAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-15O-1_GACATAGT-ATATACAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216948\n",
      "  C: 0.287521\n",
      "  G: 0.277757\n",
      "  T: 0.217774\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 34,815 / 35,085 (99.230%)\n",
      "Discarded reads ...................: 0 / 35,085 (0.000%)\n",
      "Not assembled reads ...............: 270 / 35,085 (0.770%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-15O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-15O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-15O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-15O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-15O-2_GTCTGCTA-TAGCGAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-15O-2_GTCTGCTA-TAGCGAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217868\n",
      "  C: 0.286491\n",
      "  G: 0.276789\n",
      "  T: 0.218852\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 40,607 / 40,923 (99.228%)\n",
      "Discarded reads ...................: 0 / 40,923 (0.000%)\n",
      "Not assembled reads ...............: 316 / 40,923 (0.772%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-15O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-15O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-15O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-15O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-16M-1_GTAACGAG-AGAGTCAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-16M-1_GTAACGAG-AGAGTCAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216386\n",
      "  C: 0.287949\n",
      "  G: 0.278857\n",
      "  T: 0.216808\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 63%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 123,682 / 124,496 (99.346%)\n",
      "Discarded reads ...................: 0 / 124,496 (0.000%)\n",
      "Not assembled reads ...............: 814 / 124,496 (0.654%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-16M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-16M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-16M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-16M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-16M-2_ACGCTACT-CGTCGCTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-16M-2_ACGCTACT-CGTCGCTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216211\n",
      "  C: 0.288237\n",
      "  G: 0.278938\n",
      "  T: 0.216614\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 51,247 / 51,605 (99.306%)\n",
      "Discarded reads ...................: 0 / 51,605 (0.000%)\n",
      "Not assembled reads ...............: 358 / 51,605 (0.694%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-16M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-16M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-16M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-16M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-16O-1_GTCTATGA-CTGCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-16O-1_GTCTATGA-CTGCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218611\n",
      "  C: 0.286651\n",
      "  G: 0.275010\n",
      "  T: 0.219728\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 38,786 / 39,312 (98.662%)\n",
      "Discarded reads ...................: 0 / 39,312 (0.000%)\n",
      "Not assembled reads ...............: 526 / 39,312 (1.338%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-16O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-16O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-16O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-16O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-16O-2_ACGTGCGC-TACGAGAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-16O-2_ACGTGCGC-TACGAGAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.220390\n",
      "  C: 0.284149\n",
      "  G: 0.273875\n",
      "  T: 0.221586\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 18,925 / 19,160 (98.773%)\n",
      "Discarded reads ...................: 0 / 19,160 (0.000%)\n",
      "Not assembled reads ...............: 235 / 19,160 (1.227%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-16O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-16O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-16O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-16O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-17M-1_ACTCACTG-CTAGAGCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-17M-1_ACTCACTG-CTAGAGCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216853\n",
      "  C: 0.287898\n",
      "  G: 0.277467\n",
      "  T: 0.217781\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 98%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 80,556 / 81,394 (98.970%)\n",
      "Discarded reads ...................: 0 / 81,394 (0.000%)\n",
      "Not assembled reads ...............: 838 / 81,394 (1.030%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-17M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-17M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-17M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-17M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-17M-2_TATAGCGA-TCATCGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-17M-2_TATAGCGA-TCATCGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217233\n",
      "  C: 0.287154\n",
      "  G: 0.277911\n",
      "  T: 0.217703\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 41,750 / 42,023 (99.350%)\n",
      "Discarded reads ...................: 0 / 42,023 (0.000%)\n",
      "Not assembled reads ...............: 273 / 42,023 (0.650%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-17M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-17M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-17M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-17M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-17O-1_ATAGTACC-ACGTCTCG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-17O-1_ATAGTACC-ACGTCTCG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216479\n",
      "  C: 0.288161\n",
      "  G: 0.277995\n",
      "  T: 0.217365\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 37,716 / 38,056 (99.107%)\n",
      "Discarded reads ...................: 0 / 38,056 (0.000%)\n",
      "Not assembled reads ...............: 340 / 38,056 (0.893%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-17O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-17O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-17O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-17O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-17O-2_GCGTATAC-TCGACGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-17O-2_GCGTATAC-TCGACGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216248\n",
      "  C: 0.288391\n",
      "  G: 0.278444\n",
      "  T: 0.216916\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 35,267 / 35,511 (99.313%)\n",
      "Discarded reads ...................: 0 / 35,511 (0.000%)\n",
      "Not assembled reads ...............: 244 / 35,511 (0.687%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-17O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-17O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-17O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-17O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-18M-1_CTGCGTAG-GACACTGA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-18M-1_CTGCGTAG-GACACTGA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216263\n",
      "  C: 0.288654\n",
      "  G: 0.277718\n",
      "  T: 0.217365\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 50,219 / 50,794 (98.868%)\n",
      "Discarded reads ...................: 0 / 50,794 (0.000%)\n",
      "Not assembled reads ...............: 575 / 50,794 (1.132%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-18M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-18M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-18M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-18M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-18M-2_CGAAGTAT-GGATATCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-18M-2_CGAAGTAT-GGATATCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214904\n",
      "  C: 0.290705\n",
      "  G: 0.277477\n",
      "  T: 0.216914\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 65,308 / 66,264 (98.557%)\n",
      "Discarded reads ...................: 0 / 66,264 (0.000%)\n",
      "Not assembled reads ...............: 956 / 66,264 (1.443%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-18M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-18M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-18M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-18M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U01M-1_TGAGTACG-GATCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U01M-1_TGAGTACG-GATCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214920\n",
      "  C: 0.290399\n",
      "  G: 0.279265\n",
      "  T: 0.215416\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 19,130 / 19,273 (99.258%)\n",
      "Discarded reads ...................: 0 / 19,273 (0.000%)\n",
      "Not assembled reads ...............: 143 / 19,273 (0.742%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U01M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U01M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U01M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U01M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U01M-2_CTCGACTT-TGCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U01M-2_CTCGACTT-TGCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216674\n",
      "  C: 0.287644\n",
      "  G: 0.278210\n",
      "  T: 0.217472\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 12,288 / 12,388 (99.193%)\n",
      "Discarded reads ...................: 0 / 12,388 (0.000%)\n",
      "Not assembled reads ...............: 100 / 12,388 (0.807%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U01M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U01M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U01M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U01M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U01O-1_GCGTATAC-GACACCGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U01O-1_GCGTATAC-GACACCGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216353\n",
      "  C: 0.288391\n",
      "  G: 0.278294\n",
      "  T: 0.216963\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 17,823 / 17,958 (99.248%)\n",
      "Discarded reads ...................: 0 / 17,958 (0.000%)\n",
      "Not assembled reads ...............: 135 / 17,958 (0.752%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U01O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U01O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U01O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U01O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U01O-2_CGAAGTAT-ATCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U01O-2_CGAAGTAT-ATCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216423\n",
      "  C: 0.288048\n",
      "  G: 0.278171\n",
      "  T: 0.217357\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 10,718 / 10,823 (99.030%)\n",
      "Discarded reads ...................: 0 / 10,823 (0.000%)\n",
      "Not assembled reads ...............: 105 / 10,823 (0.970%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U01O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U01O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U01O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U01O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U02O-1_TGCTCGTA-CTACTATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U02O-1_TGCTCGTA-CTACTATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217023\n",
      "  C: 0.287125\n",
      "  G: 0.278626\n",
      "  T: 0.217226\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 12,070 / 12,144 (99.391%)\n",
      "Discarded reads ...................: 0 / 12,144 (0.000%)\n",
      "Not assembled reads ...............: 74 / 12,144 (0.609%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U02O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U02O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U02O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U02O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U02O-2_TAGTCTCC-ACGACGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U02O-2_TAGTCTCC-ACGACGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218345\n",
      "  C: 0.285927\n",
      "  G: 0.277453\n",
      "  T: 0.218275\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 11,956 / 12,028 (99.401%)\n",
      "Discarded reads ...................: 0 / 12,028 (0.000%)\n",
      "Not assembled reads ...............: 72 / 12,028 (0.599%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U02O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U02O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U02O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U02O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U03M-1_TAGCAGCT-ACTATCTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U03M-1_TAGCAGCT-ACTATCTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214298\n",
      "  C: 0.290047\n",
      "  G: 0.280839\n",
      "  T: 0.214816\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 11,800 / 11,891 (99.235%)\n",
      "Discarded reads ...................: 0 / 11,891 (0.000%)\n",
      "Not assembled reads ...............: 91 / 11,891 (0.765%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U03M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U03M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U03M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U03M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U03M-2_AACGCTGA-CGTTACTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U03M-2_AACGCTGA-CGTTACTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215352\n",
      "  C: 0.300080\n",
      "  G: 0.254475\n",
      "  T: 0.230093\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 61 / 75 (81.333%)\n",
      "Discarded reads ...................: 0 / 75 (0.000%)\n",
      "Not assembled reads ...............: 14 / 75 (18.667%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U03M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U03M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U03M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U03M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U03O-1_CGAGCGAC-ATATACAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U03O-1_CGAGCGAC-ATATACAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215223\n",
      "  C: 0.289285\n",
      "  G: 0.279571\n",
      "  T: 0.215921\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 4,680 / 4,718 (99.195%)\n",
      "Discarded reads ...................: 0 / 4,718 (0.000%)\n",
      "Not assembled reads ...............: 38 / 4,718 (0.805%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U03O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U03O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U03O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U03O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U03O-2_TCTCTATG-TAGCGAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U03O-2_TCTCTATG-TAGCGAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216670\n",
      "  C: 0.287691\n",
      "  G: 0.278573\n",
      "  T: 0.217067\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 11,926 / 12,009 (99.309%)\n",
      "Discarded reads ...................: 0 / 12,009 (0.000%)\n",
      "Not assembled reads ...............: 83 / 12,009 (0.691%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U03O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U03O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U03O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U03O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U04M-1_CGTAGCGA-AGAGTCAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U04M-1_CGTAGCGA-AGAGTCAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212925\n",
      "  C: 0.291977\n",
      "  G: 0.281122\n",
      "  T: 0.213976\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 9,018 / 9,113 (98.958%)\n",
      "Discarded reads ...................: 0 / 9,113 (0.000%)\n",
      "Not assembled reads ...............: 95 / 9,113 (1.042%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U04M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U04M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U04M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U04M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U04M-2_CGAGAGTT-CGTCGCTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U04M-2_CGAGAGTT-CGTCGCTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213722\n",
      "  C: 0.290844\n",
      "  G: 0.281110\n",
      "  T: 0.214323\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 8,576 / 8,651 (99.133%)\n",
      "Discarded reads ...................: 0 / 8,651 (0.000%)\n",
      "Not assembled reads ...............: 75 / 8,651 (0.867%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U04M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U04M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U04M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U04M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U04O-1_ACTACGAC-CTGCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U04O-1_ACTACGAC-CTGCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215723\n",
      "  C: 0.288421\n",
      "  G: 0.279549\n",
      "  T: 0.216307\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 13,311 / 13,412 (99.247%)\n",
      "Discarded reads ...................: 0 / 13,412 (0.000%)\n",
      "Not assembled reads ...............: 101 / 13,412 (0.753%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U04O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U04O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U04O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U04O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U04O-2_GATCTACG-TACGAGAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U04O-2_GATCTACG-TACGAGAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215379\n",
      "  C: 0.288719\n",
      "  G: 0.279859\n",
      "  T: 0.216042\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 10,825 / 10,914 (99.185%)\n",
      "Discarded reads ...................: 0 / 10,914 (0.000%)\n",
      "Not assembled reads ...............: 89 / 10,914 (0.815%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U04O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U04O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U04O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U04O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U05M-1_GACATAGT-CTAGAGCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U05M-1_GACATAGT-CTAGAGCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.211064\n",
      "  C: 0.293407\n",
      "  G: 0.283925\n",
      "  T: 0.211604\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 31,212 / 31,412 (99.363%)\n",
      "Discarded reads ...................: 0 / 31,412 (0.000%)\n",
      "Not assembled reads ...............: 200 / 31,412 (0.637%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U05M-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U05M-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U05M-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U05M-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U05M-2_GTCTGCTA-TCATCGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U05M-2_GTCTGCTA-TCATCGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.211711\n",
      "  C: 0.292531\n",
      "  G: 0.283541\n",
      "  T: 0.212216\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 10,705 / 10,761 (99.480%)\n",
      "Discarded reads ...................: 0 / 10,761 (0.000%)\n",
      "Not assembled reads ...............: 56 / 10,761 (0.520%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U05M-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U05M-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U05M-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U05M-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U05O-1_GTAACGAG-ACGTCTCG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U05O-1_GTAACGAG-ACGTCTCG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213009\n",
      "  C: 0.291535\n",
      "  G: 0.281688\n",
      "  T: 0.213768\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 13,507 / 13,595 (99.353%)\n",
      "Discarded reads ...................: 0 / 13,595 (0.000%)\n",
      "Not assembled reads ...............: 88 / 13,595 (0.647%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U05O-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U05O-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U05O-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U05O-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/15S-WB-U05O-2_ACGCTACT-GCTCTAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/15S-WB-U05O-2_ACGCTACT-GCTCTAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213906\n",
      "  C: 0.290409\n",
      "  G: 0.281305\n",
      "  T: 0.214380\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 6,182 / 6,238 (99.102%)\n",
      "Discarded reads ...................: 0 / 6,238 (0.000%)\n",
      "Not assembled reads ...............: 56 / 6,238 (0.898%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U05O-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/15S-WB-U05O-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U05O-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/15S-WB-U05O-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/Blank-10_GTCTGCTA-CTGCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/Blank-10_GTCTGCTA-CTGCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214308\n",
      "  C: 0.293884\n",
      "  G: 0.273753\n",
      "  T: 0.218055\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 286 / 302 (94.702%)\n",
      "Discarded reads ...................: 0 / 302 (0.000%)\n",
      "Not assembled reads ...............: 16 / 302 (5.298%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/Blank-10.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/Blank-10.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/Blank-10.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/Blank-10.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/Blank-11_GTCTGCTA-TACGAGAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/Blank-11_GTCTGCTA-TACGAGAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215999\n",
      "  C: 0.290427\n",
      "  G: 0.275815\n",
      "  T: 0.217759\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 534 / 549 (97.268%)\n",
      "Discarded reads ...................: 0 / 549 (0.000%)\n",
      "Not assembled reads ...............: 15 / 549 (2.732%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/Blank-11.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/Blank-11.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/Blank-11.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/Blank-11.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/Blank-13_TAGTCTCC-CTACTATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/Blank-13_TAGTCTCC-CTACTATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.218275\n",
      "  C: 0.287149\n",
      "  G: 0.274036\n",
      "  T: 0.220540\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 244 / 249 (97.992%)\n",
      "Discarded reads ...................: 0 / 249 (0.000%)\n",
      "Not assembled reads ...............: 5 / 249 (2.008%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/Blank-13.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/Blank-13.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/Blank-13.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/Blank-13.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/Blank-16_ATAGTACC-GGATATCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/Blank-16_ATAGTACC-GGATATCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.216979\n",
      "  C: 0.289327\n",
      "  G: 0.273870\n",
      "  T: 0.219825\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 1,063 / 1,094 (97.166%)\n",
      "Discarded reads ...................: 0 / 1,094 (0.000%)\n",
      "Not assembled reads ...............: 31 / 1,094 (2.834%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/Blank-16.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/Blank-16.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/Blank-16.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/Blank-16.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/Blank-17_ACTCACTG-GACACTGA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/Blank-17_ACTCACTG-GACACTGA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.219980\n",
      "  C: 0.284283\n",
      "  G: 0.273135\n",
      "  T: 0.222602\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 190 / 193 (98.446%)\n",
      "Discarded reads ...................: 0 / 193 (0.000%)\n",
      "Not assembled reads ...............: 3 / 193 (1.554%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/Blank-17.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/Blank-17.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/Blank-17.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/Blank-17.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/Blank-1_CGTAGCGA-ATCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/Blank-1_CGTAGCGA-ATCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217821\n",
      "  C: 0.289074\n",
      "  G: 0.273598\n",
      "  T: 0.219507\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 24 / 26 (92.308%)\n",
      "Discarded reads ...................: 0 / 26 (0.000%)\n",
      "Not assembled reads ...............: 2 / 26 (7.692%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/Blank-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/Blank-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/Blank-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/Blank-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/Blank-22_GTCTATGA-TCGACGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/Blank-22_GTCTATGA-TCGACGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.224827\n",
      "  C: 0.280451\n",
      "  G: 0.266268\n",
      "  T: 0.228454\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 116 / 117 (99.145%)\n",
      "Discarded reads ...................: 0 / 117 (0.000%)\n",
      "Not assembled reads ...............: 1 / 117 (0.855%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/Blank-22.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/Blank-22.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/Blank-22.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/Blank-22.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/Blank-4_GATCTACG-CGTTACTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/Blank-4_GATCTACG-CGTTACTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.219301\n",
      "  C: 0.286664\n",
      "  G: 0.273700\n",
      "  T: 0.220335\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 358 / 370 (96.757%)\n",
      "Discarded reads ...................: 0 / 370 (0.000%)\n",
      "Not assembled reads ...............: 12 / 370 (3.243%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/Blank-4.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/Blank-4.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/Blank-4.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/Blank-4.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/Blank-6_CGAGAGTT-ACTATCTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/Blank-6_CGAGAGTT-ACTATCTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217578\n",
      "  C: 0.286787\n",
      "  G: 0.277545\n",
      "  T: 0.218090\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 26,952 / 27,122 (99.373%)\n",
      "Discarded reads ...................: 0 / 27,122 (0.000%)\n",
      "Not assembled reads ...............: 170 / 27,122 (0.627%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/Blank-6.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/Blank-6.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/Blank-6.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/Blank-6.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/Blank-8_GATCTACG-TAGCGAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/Blank-8_GATCTACG-TAGCGAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.217509\n",
      "  C: 0.286977\n",
      "  G: 0.277295\n",
      "  T: 0.218219\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 1,070 / 1,088 (98.346%)\n",
      "Discarded reads ...................: 0 / 1,088 (0.000%)\n",
      "Not assembled reads ...............: 18 / 1,088 (1.654%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/Blank-8.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/Blank-8.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/Blank-8.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/Blank-8.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/DOE-BS-1_TAGTCTCC-CGTTACTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/DOE-BS-1_TAGTCTCC-CGTTACTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.209749\n",
      "  C: 0.294360\n",
      "  G: 0.285514\n",
      "  T: 0.210377\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 39,683 / 39,997 (99.215%)\n",
      "Discarded reads ...................: 0 / 39,997 (0.000%)\n",
      "Not assembled reads ...............: 314 / 39,997 (0.785%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/DOE-BS-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/DOE-BS-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/DOE-BS-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/DOE-BS-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/DOE-BS-2_TAGCAGCT-ATATACAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/DOE-BS-2_TAGCAGCT-ATATACAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.209688\n",
      "  C: 0.294619\n",
      "  G: 0.285419\n",
      "  T: 0.210274\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 18,401 / 18,556 (99.165%)\n",
      "Discarded reads ...................: 0 / 18,556 (0.000%)\n",
      "Not assembled reads ...............: 155 / 18,556 (0.835%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/DOE-BS-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/DOE-BS-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/DOE-BS-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/DOE-BS-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/DOE-BS-3_AACGCTGA-TAGCGAGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/DOE-BS-3_AACGCTGA-TAGCGAGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.209685\n",
      "  C: 0.294656\n",
      "  G: 0.285067\n",
      "  T: 0.210592\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 25,228 / 25,454 (99.112%)\n",
      "Discarded reads ...................: 0 / 25,454 (0.000%)\n",
      "Not assembled reads ...............: 226 / 25,454 (0.888%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/DOE-BS-3.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/DOE-BS-3.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/DOE-BS-3.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/DOE-BS-3.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/DOE-BS-4_CGAGCGAC-AGAGTCAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/DOE-BS-4_CGAGCGAC-AGAGTCAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.210447\n",
      "  C: 0.294050\n",
      "  G: 0.284216\n",
      "  T: 0.211286\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 20,945 / 21,152 (99.021%)\n",
      "Discarded reads ...................: 0 / 21,152 (0.000%)\n",
      "Not assembled reads ...............: 207 / 21,152 (0.979%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/DOE-BS-4.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/DOE-BS-4.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/DOE-BS-4.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/DOE-BS-4.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/DOE-Blank_GATCTACG-TCATCGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/DOE-Blank_GATCTACG-TCATCGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.215320\n",
      "  C: 0.288695\n",
      "  G: 0.280263\n",
      "  T: 0.215722\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 11,059 / 11,141 (99.264%)\n",
      "Discarded reads ...................: 0 / 11,141 (0.000%)\n",
      "Not assembled reads ...............: 82 / 11,141 (0.736%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/DOE-Blank.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/DOE-Blank.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/DOE-Blank.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/DOE-Blank.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/DOE-UBS-1_TCTCTATG-CGTCGCTA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/DOE-UBS-1_TCTCTATG-CGTCGCTA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.209344\n",
      "  C: 0.295204\n",
      "  G: 0.285576\n",
      "  T: 0.209876\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 30,505 / 30,786 (99.087%)\n",
      "Discarded reads ...................: 0 / 30,786 (0.000%)\n",
      "Not assembled reads ...............: 281 / 30,786 (0.913%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/DOE-UBS-1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/DOE-UBS-1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/DOE-UBS-1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/DOE-UBS-1.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/DOE-UBS-2_CGTAGCGA-CTGCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/DOE-UBS-2_CGTAGCGA-CTGCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.208994\n",
      "  C: 0.296147\n",
      "  G: 0.285105\n",
      "  T: 0.209754\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 26,134 / 26,427 (98.891%)\n",
      "Discarded reads ...................: 0 / 26,427 (0.000%)\n",
      "Not assembled reads ...............: 293 / 26,427 (1.109%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/DOE-UBS-2.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/DOE-UBS-2.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/DOE-UBS-2.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/DOE-UBS-2.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/DOE-UBS-3_CGAGAGTT-TACGAGAC_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/DOE-UBS-3_CGAGAGTT-TACGAGAC_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.210601\n",
      "  C: 0.293848\n",
      "  G: 0.284260\n",
      "  T: 0.211291\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 19,920 / 20,142 (98.898%)\n",
      "Discarded reads ...................: 0 / 20,142 (0.000%)\n",
      "Not assembled reads ...............: 222 / 20,142 (1.102%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/DOE-UBS-3.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/DOE-UBS-3.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/DOE-UBS-3.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/DOE-UBS-3.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/DOE-UBS-4_ACTACGAC-CTAGAGCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/DOE-UBS-4_ACTACGAC-CTAGAGCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.210589\n",
      "  C: 0.293863\n",
      "  G: 0.284446\n",
      "  T: 0.211102\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 41,029 / 41,405 (99.092%)\n",
      "Discarded reads ...................: 0 / 41,405 (0.000%)\n",
      "Not assembled reads ...............: 376 / 41,405 (0.908%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/DOE-UBS-4.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/DOE-UBS-4.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/DOE-UBS-4.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/DOE-UBS-4.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-09M_GTAACGAG-CGTGAGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-09M_GTAACGAG-CGTGAGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.211895\n",
      "  C: 0.292616\n",
      "  G: 0.283185\n",
      "  T: 0.212304\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 16,465 / 16,565 (99.396%)\n",
      "Discarded reads ...................: 0 / 16,565 (0.000%)\n",
      "Not assembled reads ...............: 100 / 16,565 (0.604%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-09M.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-09M.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-09M.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-09M.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-09T_ACGCTACT-TCGACGAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-09T_ACGCTACT-TCGACGAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212985\n",
      "  C: 0.291602\n",
      "  G: 0.281990\n",
      "  T: 0.213423\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 18,243 / 18,403 (99.131%)\n",
      "Discarded reads ...................: 0 / 18,403 (0.000%)\n",
      "Not assembled reads ...............: 160 / 18,403 (0.869%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-09T.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-09T.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-09T.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-09T.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-11M_GTCTATGA-GACACTGA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-11M_GTCTATGA-GACACTGA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212308\n",
      "  C: 0.292356\n",
      "  G: 0.282562\n",
      "  T: 0.212774\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 13,716 / 13,841 (99.097%)\n",
      "Discarded reads ...................: 0 / 13,841 (0.000%)\n",
      "Not assembled reads ...............: 125 / 13,841 (0.903%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-11M.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-11M.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-11M.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-11M.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-11T_ACGTGCGC-GGATATCT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-11T_ACGTGCGC-GGATATCT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212277\n",
      "  C: 0.293146\n",
      "  G: 0.281513\n",
      "  T: 0.213063\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 14,638 / 14,829 (98.712%)\n",
      "Discarded reads ...................: 0 / 14,829 (0.000%)\n",
      "Not assembled reads ...............: 191 / 14,829 (1.288%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-11T.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-11T.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-11T.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-11T.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-17M_ACTCACTG-GATCGTGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-17M_ACTCACTG-GATCGTGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213719\n",
      "  C: 0.291184\n",
      "  G: 0.280827\n",
      "  T: 0.214270\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,558 / 22,737 (99.213%)\n",
      "Discarded reads ...................: 0 / 22,737 (0.000%)\n",
      "Not assembled reads ...............: 179 / 22,737 (0.787%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-17M.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-17M.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-17M.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-17M.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-17T_TATAGCGA-TGCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-17T_TATAGCGA-TGCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.211872\n",
      "  C: 0.292429\n",
      "  G: 0.283076\n",
      "  T: 0.212623\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 12,970 / 13,065 (99.273%)\n",
      "Discarded reads ...................: 0 / 13,065 (0.000%)\n",
      "Not assembled reads ...............: 95 / 13,065 (0.727%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-17T.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-17T.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-17T.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-17T.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-23M_ATAGTACC-GACACCGT_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-23M_ATAGTACC-GACACCGT_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212475\n",
      "  C: 0.291772\n",
      "  G: 0.282742\n",
      "  T: 0.213011\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 22,842 / 23,008 (99.279%)\n",
      "Discarded reads ...................: 0 / 23,008 (0.000%)\n",
      "Not assembled reads ...............: 166 / 23,008 (0.721%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-23M.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-23M.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-23M.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-23M.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-23T_TGAGTACG-GTCAGATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-23T_TGAGTACG-GTCAGATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.212703\n",
      "  C: 0.291901\n",
      "  G: 0.282385\n",
      "  T: 0.213010\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 37,216 / 37,464 (99.338%)\n",
      "Discarded reads ...................: 0 / 37,464 (0.000%)\n",
      "Not assembled reads ...............: 248 / 37,464 (0.662%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-23T.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-23T.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-23T.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-23T.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-30M_CTCGACTT-TAGTGTAG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-30M_CTCGACTT-TAGTGTAG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213847\n",
      "  C: 0.291188\n",
      "  G: 0.280577\n",
      "  T: 0.214389\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 12,869 / 12,962 (99.283%)\n",
      "Discarded reads ...................: 0 / 12,962 (0.000%)\n",
      "Not assembled reads ...............: 93 / 12,962 (0.717%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-30M.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-30M.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-30M.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-30M.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-30T_CTCGACTT-ATCGTACG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-30T_CTCGACTT-ATCGTACG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.213990\n",
      "  C: 0.290334\n",
      "  G: 0.281324\n",
      "  T: 0.214353\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 25,698 / 25,872 (99.327%)\n",
      "Discarded reads ...................: 0 / 25,872 (0.000%)\n",
      "Not assembled reads ...............: 174 / 25,872 (0.673%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-30T.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-30T.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-30T.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-30T.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-53M_GCGTATAC-CTACTATA_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-53M_GCGTATAC-CTACTATA_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.214637\n",
      "  C: 0.289826\n",
      "  G: 0.280489\n",
      "  T: 0.215048\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 11,196 / 11,301 (99.071%)\n",
      "Discarded reads ...................: 0 / 11,301 (0.000%)\n",
      "Not assembled reads ...............: 105 / 11,301 (0.929%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-53M.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-53M.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-53M.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-53M.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-53T_CTGCGTAG-ACGACGTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-53T_CTGCGTAG-ACGACGTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.224625\n",
      "  C: 0.279673\n",
      "  G: 0.270581\n",
      "  T: 0.225121\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 12,622 / 12,706 (99.339%)\n",
      "Discarded reads ...................: 0 / 12,706 (0.000%)\n",
      "Not assembled reads ...............: 84 / 12,706 (0.661%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-53T.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-53T.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-53T.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-53T.unassembled.reverse.fastq\n",
      " ____  _____    _    ____ \n",
      "|  _ \\| ____|  / \\  |  _ \\\n",
      "| |_) |  _|   / _ \\ | |_) |\n",
      "|  __/| |___ / ___ \\|  _ <\n",
      "|_|   |_____/_/   \\_\\_| \\_\\\n",
      "\n",
      "PEAR v0.9.6 [January 15, 2015]\n",
      "\n",
      "Citation - PEAR: a fast and accurate Illumina Paired-End reAd mergeR\n",
      "Zhang et al (2014) Bioinformatics 30(5): 614-620 | doi:10.1093/bioinformatics/btt593\n",
      "\n",
      "Forward reads file.................: ../data/Seq_data/Seqs/JP-CFB-Blank1_CGAAGTAT-ACTATCTG_L001_R1_001.fastq.gz\n",
      "Reverse reads file.................: ../data/Seq_data/Seqs/JP-CFB-Blank1_CGAAGTAT-ACTATCTG_L001_R2_001.fastq.gz\n",
      "PHRED..............................: 33\n",
      "Using empirical frequencies........: YES\n",
      "Statistical method.................: OES\n",
      "Maximum assembly length............: 500\n",
      "Minimum assembly length............: 50\n",
      "p-value............................: 0.010000\n",
      "Quality score threshold (trimming).: 0\n",
      "Minimum read size after trimming...: 1\n",
      "Maximal ratio of uncalled bases....: 1.000000\n",
      "Minimum overlap....................: 10\n",
      "Scoring method.....................: Scaled score\n",
      "Threads............................: 4\n",
      "\n",
      "Allocating memory..................: 200,000,000 bytes\n",
      "Computing empirical frequencies....: DONE\n",
      "  A: 0.220614\n",
      "  C: 0.283905\n",
      "  G: 0.274113\n",
      "  T: 0.221368\n",
      "  0 uncalled bases\n",
      "Assemblying reads: 0%\r",
      "Assemblying reads: 100%\n",
      "\n",
      "Assembled reads ...................: 3,485 / 3,508 (99.344%)\n",
      "Discarded reads ...................: 0 / 3,508 (0.000%)\n",
      "Not assembled reads ...............: 23 / 3,508 (0.656%)\n",
      "Assembled reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-Blank1.assembled.fastq\n",
      "Discarded reads file...............: ../data/Seq_data/MergedSeqs/JP-CFB-Blank1.discarded.fastq\n",
      "Unassembled forward reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-Blank1.unassembled.forward.fastq\n",
      "Unassembled reverse reads file.....: ../data/Seq_data/MergedSeqs/JP-CFB-Blank1.unassembled.reverse.fastq\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "# Using \"bash magic\" - to run bash code in iPython notebook\n",
    "# Note: fwd and rev read files need to be complementary, in the right order, etc.\n",
    "\n",
    "r1=../data/Seq_data/Seqs/*R1*\n",
    "fwdreads=($r1)\n",
    "# Setting array of fwd read files\n",
    "\n",
    "r2=../data/Seq_data/Seqs/*R2*\n",
    "revreads=($r2)\n",
    "# Setting array of rev read files\n",
    "\n",
    "# For loop to run pear merge fwd and rev reads on all datasets.\n",
    "# Most of this code just pulls out the sample's name for each pair of seqeunce files.\n",
    "for ((i = 0; i < ${#fwdreads[@]}; ++i)); do\n",
    "    filepath=${fwdreads[i]}\n",
    "    # Take the full filepath for each sample (using read 1 arbitrarily)\n",
    "    splitfilepath=(${filepath//\\// })\n",
    "    # Split it up by /s\n",
    "    filename=${splitfilepath[${#splitfilepath[@]} - 1]}\n",
    "    # Take the last element from that filepath (i.e., the filename for read 1)\n",
    "    splitfilename=(${filename//_/ })\n",
    "    # Split the filename up using the underscore\n",
    "    outputname=${splitfilename[$1]}\n",
    "    # Take the first element of the split filename - i.e., everything before the first underscore - i.e., the sampleID\n",
    "    pear -m 500 -j 4 -f \"${fwdreads[i]}\" -r \"${revreads[i]}\" -o ../data/Seq_data/MergedSeqs/$outputname\n",
    "    # These pear settings restrict merged length to 500bp (max length of 2x250 paired ends sequencing)\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "15S-NT-19O-1.assembled.fastq\r\n",
      "15S-NT-19O-1.discarded.fastq\r\n",
      "15S-NT-19O-1.unassembled.forward.fastq\r\n",
      "15S-NT-19O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-19O-2.assembled.fastq\r\n",
      "15S-NT-19O-2.discarded.fastq\r\n",
      "15S-NT-19O-2.unassembled.forward.fastq\r\n",
      "15S-NT-19O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-20O-A-1.assembled.fastq\r\n",
      "15S-NT-20O-A-1.discarded.fastq\r\n",
      "15S-NT-20O-A-1.unassembled.forward.fastq\r\n",
      "15S-NT-20O-A-1.unassembled.reverse.fastq\r\n",
      "15S-NT-20O-A-2.assembled.fastq\r\n",
      "15S-NT-20O-A-2.discarded.fastq\r\n",
      "15S-NT-20O-A-2.unassembled.forward.fastq\r\n",
      "15S-NT-20O-A-2.unassembled.reverse.fastq\r\n",
      "15S-NT-20O-B-1.assembled.fastq\r\n",
      "15S-NT-20O-B-1.discarded.fastq\r\n",
      "15S-NT-20O-B-1.unassembled.forward.fastq\r\n",
      "15S-NT-20O-B-1.unassembled.reverse.fastq\r\n",
      "15S-NT-20O-B-2.assembled.fastq\r\n",
      "15S-NT-20O-B-2.discarded.fastq\r\n",
      "15S-NT-20O-B-2.unassembled.forward.fastq\r\n",
      "15S-NT-20O-B-2.unassembled.reverse.fastq\r\n",
      "15S-NT-21M-1.assembled.fastq\r\n",
      "15S-NT-21M-1.discarded.fastq\r\n",
      "15S-NT-21M-1.unassembled.forward.fastq\r\n",
      "15S-NT-21M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-21M-2.assembled.fastq\r\n",
      "15S-NT-21M-2.discarded.fastq\r\n",
      "15S-NT-21M-2.unassembled.forward.fastq\r\n",
      "15S-NT-21M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-21O-1.assembled.fastq\r\n",
      "15S-NT-21O-1.discarded.fastq\r\n",
      "15S-NT-21O-1.unassembled.forward.fastq\r\n",
      "15S-NT-21O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-21O-2.assembled.fastq\r\n",
      "15S-NT-21O-2.discarded.fastq\r\n",
      "15S-NT-21O-2.unassembled.forward.fastq\r\n",
      "15S-NT-21O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-22M-1.assembled.fastq\r\n",
      "15S-NT-22M-1.discarded.fastq\r\n",
      "15S-NT-22M-1.unassembled.forward.fastq\r\n",
      "15S-NT-22M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-22M-2.assembled.fastq\r\n",
      "15S-NT-22M-2.discarded.fastq\r\n",
      "15S-NT-22M-2.unassembled.forward.fastq\r\n",
      "15S-NT-22M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-22O-1.assembled.fastq\r\n",
      "15S-NT-22O-1.discarded.fastq\r\n",
      "15S-NT-22O-1.unassembled.forward.fastq\r\n",
      "15S-NT-22O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-22O-2.assembled.fastq\r\n",
      "15S-NT-22O-2.discarded.fastq\r\n",
      "15S-NT-22O-2.unassembled.forward.fastq\r\n",
      "15S-NT-22O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-23M-1.assembled.fastq\r\n",
      "15S-NT-23M-1.discarded.fastq\r\n",
      "15S-NT-23M-1.unassembled.forward.fastq\r\n",
      "15S-NT-23M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-23M-2.assembled.fastq\r\n",
      "15S-NT-23M-2.discarded.fastq\r\n",
      "15S-NT-23M-2.unassembled.forward.fastq\r\n",
      "15S-NT-23M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-23O-1.assembled.fastq\r\n",
      "15S-NT-23O-1.discarded.fastq\r\n",
      "15S-NT-23O-1.unassembled.forward.fastq\r\n",
      "15S-NT-23O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-23O-2.assembled.fastq\r\n",
      "15S-NT-23O-2.discarded.fastq\r\n",
      "15S-NT-23O-2.unassembled.forward.fastq\r\n",
      "15S-NT-23O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-24M-1.assembled.fastq\r\n",
      "15S-NT-24M-1.discarded.fastq\r\n",
      "15S-NT-24M-1.unassembled.forward.fastq\r\n",
      "15S-NT-24M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-24M-2.assembled.fastq\r\n",
      "15S-NT-24M-2.discarded.fastq\r\n",
      "15S-NT-24M-2.unassembled.forward.fastq\r\n",
      "15S-NT-24M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-24O-1.assembled.fastq\r\n",
      "15S-NT-24O-1.discarded.fastq\r\n",
      "15S-NT-24O-1.unassembled.forward.fastq\r\n",
      "15S-NT-24O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-24O-2.assembled.fastq\r\n",
      "15S-NT-24O-2.discarded.fastq\r\n",
      "15S-NT-24O-2.unassembled.forward.fastq\r\n",
      "15S-NT-24O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-25O-1.assembled.fastq\r\n",
      "15S-NT-25O-1.discarded.fastq\r\n",
      "15S-NT-25O-1.unassembled.forward.fastq\r\n",
      "15S-NT-25O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-25O-2.assembled.fastq\r\n",
      "15S-NT-25O-2.discarded.fastq\r\n",
      "15S-NT-25O-2.unassembled.forward.fastq\r\n",
      "15S-NT-25O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-26O-1.assembled.fastq\r\n",
      "15S-NT-26O-1.discarded.fastq\r\n",
      "15S-NT-26O-1.unassembled.forward.fastq\r\n",
      "15S-NT-26O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-26O-2.assembled.fastq\r\n",
      "15S-NT-26O-2.discarded.fastq\r\n",
      "15S-NT-26O-2.unassembled.forward.fastq\r\n",
      "15S-NT-26O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-27M-1.assembled.fastq\r\n",
      "15S-NT-27M-1.discarded.fastq\r\n",
      "15S-NT-27M-1.unassembled.forward.fastq\r\n",
      "15S-NT-27M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-27M-2.assembled.fastq\r\n",
      "15S-NT-27M-2.discarded.fastq\r\n",
      "15S-NT-27M-2.unassembled.forward.fastq\r\n",
      "15S-NT-27M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-27O-1.assembled.fastq\r\n",
      "15S-NT-27O-1.discarded.fastq\r\n",
      "15S-NT-27O-1.unassembled.forward.fastq\r\n",
      "15S-NT-27O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-27O-2.assembled.fastq\r\n",
      "15S-NT-27O-2.discarded.fastq\r\n",
      "15S-NT-27O-2.unassembled.forward.fastq\r\n",
      "15S-NT-27O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-28M-1.assembled.fastq\r\n",
      "15S-NT-28M-1.discarded.fastq\r\n",
      "15S-NT-28M-1.unassembled.forward.fastq\r\n",
      "15S-NT-28M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-28M-2.assembled.fastq\r\n",
      "15S-NT-28M-2.discarded.fastq\r\n",
      "15S-NT-28M-2.unassembled.forward.fastq\r\n",
      "15S-NT-28M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-28O-1.assembled.fastq\r\n",
      "15S-NT-28O-1.discarded.fastq\r\n",
      "15S-NT-28O-1.unassembled.forward.fastq\r\n",
      "15S-NT-28O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-28O-2.assembled.fastq\r\n",
      "15S-NT-28O-2.discarded.fastq\r\n",
      "15S-NT-28O-2.unassembled.forward.fastq\r\n",
      "15S-NT-28O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-29O-1.assembled.fastq\r\n",
      "15S-NT-29O-1.discarded.fastq\r\n",
      "15S-NT-29O-1.unassembled.forward.fastq\r\n",
      "15S-NT-29O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-29O-2.assembled.fastq\r\n",
      "15S-NT-29O-2.discarded.fastq\r\n",
      "15S-NT-29O-2.unassembled.forward.fastq\r\n",
      "15S-NT-29O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-30M-1.assembled.fastq\r\n",
      "15S-NT-30M-1.discarded.fastq\r\n",
      "15S-NT-30M-1.unassembled.forward.fastq\r\n",
      "15S-NT-30M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-30M-2.assembled.fastq\r\n",
      "15S-NT-30M-2.discarded.fastq\r\n",
      "15S-NT-30M-2.unassembled.forward.fastq\r\n",
      "15S-NT-30M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-30O-1.assembled.fastq\r\n",
      "15S-NT-30O-1.discarded.fastq\r\n",
      "15S-NT-30O-1.unassembled.forward.fastq\r\n",
      "15S-NT-30O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-30O-2.assembled.fastq\r\n",
      "15S-NT-30O-2.discarded.fastq\r\n",
      "15S-NT-30O-2.unassembled.forward.fastq\r\n",
      "15S-NT-30O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-31M-1.assembled.fastq\r\n",
      "15S-NT-31M-1.discarded.fastq\r\n",
      "15S-NT-31M-1.unassembled.forward.fastq\r\n",
      "15S-NT-31M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-31M-2.assembled.fastq\r\n",
      "15S-NT-31M-2.discarded.fastq\r\n",
      "15S-NT-31M-2.unassembled.forward.fastq\r\n",
      "15S-NT-31M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-31O-A-1.assembled.fastq\r\n",
      "15S-NT-31O-A-1.discarded.fastq\r\n",
      "15S-NT-31O-A-1.unassembled.forward.fastq\r\n",
      "15S-NT-31O-A-1.unassembled.reverse.fastq\r\n",
      "15S-NT-31O-A-2.assembled.fastq\r\n",
      "15S-NT-31O-A-2.discarded.fastq\r\n",
      "15S-NT-31O-A-2.unassembled.forward.fastq\r\n",
      "15S-NT-31O-A-2.unassembled.reverse.fastq\r\n",
      "15S-NT-31O-B-1.assembled.fastq\r\n",
      "15S-NT-31O-B-1.discarded.fastq\r\n",
      "15S-NT-31O-B-1.unassembled.forward.fastq\r\n",
      "15S-NT-31O-B-1.unassembled.reverse.fastq\r\n",
      "15S-NT-31O-B-2.assembled.fastq\r\n",
      "15S-NT-31O-B-2.discarded.fastq\r\n",
      "15S-NT-31O-B-2.unassembled.forward.fastq\r\n",
      "15S-NT-31O-B-2.unassembled.reverse.fastq\r\n",
      "15S-NT-32O-1.assembled.fastq\r\n",
      "15S-NT-32O-1.discarded.fastq\r\n",
      "15S-NT-32O-1.unassembled.forward.fastq\r\n",
      "15S-NT-32O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-32O-2.assembled.fastq\r\n",
      "15S-NT-32O-2.discarded.fastq\r\n",
      "15S-NT-32O-2.unassembled.forward.fastq\r\n",
      "15S-NT-32O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-33M-1.assembled.fastq\r\n",
      "15S-NT-33M-1.discarded.fastq\r\n",
      "15S-NT-33M-1.unassembled.forward.fastq\r\n",
      "15S-NT-33M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-33M-2.assembled.fastq\r\n",
      "15S-NT-33M-2.discarded.fastq\r\n",
      "15S-NT-33M-2.unassembled.forward.fastq\r\n",
      "15S-NT-33M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-33O-1.assembled.fastq\r\n",
      "15S-NT-33O-1.discarded.fastq\r\n",
      "15S-NT-33O-1.unassembled.forward.fastq\r\n",
      "15S-NT-33O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-33O-2.assembled.fastq\r\n",
      "15S-NT-33O-2.discarded.fastq\r\n",
      "15S-NT-33O-2.unassembled.forward.fastq\r\n",
      "15S-NT-33O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-34M-1.assembled.fastq\r\n",
      "15S-NT-34M-1.discarded.fastq\r\n",
      "15S-NT-34M-1.unassembled.forward.fastq\r\n",
      "15S-NT-34M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-34M-2.assembled.fastq\r\n",
      "15S-NT-34M-2.discarded.fastq\r\n",
      "15S-NT-34M-2.unassembled.forward.fastq\r\n",
      "15S-NT-34M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-34O-1.assembled.fastq\r\n",
      "15S-NT-34O-1.discarded.fastq\r\n",
      "15S-NT-34O-1.unassembled.forward.fastq\r\n",
      "15S-NT-34O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-34O-2.assembled.fastq\r\n",
      "15S-NT-34O-2.discarded.fastq\r\n",
      "15S-NT-34O-2.unassembled.forward.fastq\r\n",
      "15S-NT-34O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-35O-1.assembled.fastq\r\n",
      "15S-NT-35O-1.discarded.fastq\r\n",
      "15S-NT-35O-1.unassembled.forward.fastq\r\n",
      "15S-NT-35O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-35O-2.assembled.fastq\r\n",
      "15S-NT-35O-2.discarded.fastq\r\n",
      "15S-NT-35O-2.unassembled.forward.fastq\r\n",
      "15S-NT-35O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-36O-1.assembled.fastq\r\n",
      "15S-NT-36O-1.discarded.fastq\r\n",
      "15S-NT-36O-1.unassembled.forward.fastq\r\n",
      "15S-NT-36O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-36O-2.assembled.fastq\r\n",
      "15S-NT-36O-2.discarded.fastq\r\n",
      "15S-NT-36O-2.unassembled.forward.fastq\r\n",
      "15S-NT-36O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-38M-1.assembled.fastq\r\n",
      "15S-NT-38M-1.discarded.fastq\r\n",
      "15S-NT-38M-1.unassembled.forward.fastq\r\n",
      "15S-NT-38M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-38M-2.assembled.fastq\r\n",
      "15S-NT-38M-2.discarded.fastq\r\n",
      "15S-NT-38M-2.unassembled.forward.fastq\r\n",
      "15S-NT-38M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-38O-1.assembled.fastq\r\n",
      "15S-NT-38O-1.discarded.fastq\r\n",
      "15S-NT-38O-1.unassembled.forward.fastq\r\n",
      "15S-NT-38O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-38O-2.assembled.fastq\r\n",
      "15S-NT-38O-2.discarded.fastq\r\n",
      "15S-NT-38O-2.unassembled.forward.fastq\r\n",
      "15S-NT-38O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-39M-1.assembled.fastq\r\n",
      "15S-NT-39M-1.discarded.fastq\r\n",
      "15S-NT-39M-1.unassembled.forward.fastq\r\n",
      "15S-NT-39M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-39M-2.assembled.fastq\r\n",
      "15S-NT-39M-2.discarded.fastq\r\n",
      "15S-NT-39M-2.unassembled.forward.fastq\r\n",
      "15S-NT-39M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-39O-1.assembled.fastq\r\n",
      "15S-NT-39O-1.discarded.fastq\r\n",
      "15S-NT-39O-1.unassembled.forward.fastq\r\n",
      "15S-NT-39O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-39O-2.assembled.fastq\r\n",
      "15S-NT-39O-2.discarded.fastq\r\n",
      "15S-NT-39O-2.unassembled.forward.fastq\r\n",
      "15S-NT-39O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-40M-1.assembled.fastq\r\n",
      "15S-NT-40M-1.discarded.fastq\r\n",
      "15S-NT-40M-1.unassembled.forward.fastq\r\n",
      "15S-NT-40M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-40M-2.assembled.fastq\r\n",
      "15S-NT-40M-2.discarded.fastq\r\n",
      "15S-NT-40M-2.unassembled.forward.fastq\r\n",
      "15S-NT-40M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-40O-1.assembled.fastq\r\n",
      "15S-NT-40O-1.discarded.fastq\r\n",
      "15S-NT-40O-1.unassembled.forward.fastq\r\n",
      "15S-NT-40O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-40O-2.assembled.fastq\r\n",
      "15S-NT-40O-2.discarded.fastq\r\n",
      "15S-NT-40O-2.unassembled.forward.fastq\r\n",
      "15S-NT-40O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-41M-1.assembled.fastq\r\n",
      "15S-NT-41M-1.discarded.fastq\r\n",
      "15S-NT-41M-1.unassembled.forward.fastq\r\n",
      "15S-NT-41M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-41M-2.assembled.fastq\r\n",
      "15S-NT-41M-2.discarded.fastq\r\n",
      "15S-NT-41M-2.unassembled.forward.fastq\r\n",
      "15S-NT-41M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-41O-1.assembled.fastq\r\n",
      "15S-NT-41O-1.discarded.fastq\r\n",
      "15S-NT-41O-1.unassembled.forward.fastq\r\n",
      "15S-NT-41O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-41O-2.assembled.fastq\r\n",
      "15S-NT-41O-2.discarded.fastq\r\n",
      "15S-NT-41O-2.unassembled.forward.fastq\r\n",
      "15S-NT-41O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-42M-1.assembled.fastq\r\n",
      "15S-NT-42M-1.discarded.fastq\r\n",
      "15S-NT-42M-1.unassembled.forward.fastq\r\n",
      "15S-NT-42M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-42M-2.assembled.fastq\r\n",
      "15S-NT-42M-2.discarded.fastq\r\n",
      "15S-NT-42M-2.unassembled.forward.fastq\r\n",
      "15S-NT-42M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-42O-1.assembled.fastq\r\n",
      "15S-NT-42O-1.discarded.fastq\r\n",
      "15S-NT-42O-1.unassembled.forward.fastq\r\n",
      "15S-NT-42O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-42O-2.assembled.fastq\r\n",
      "15S-NT-42O-2.discarded.fastq\r\n",
      "15S-NT-42O-2.unassembled.forward.fastq\r\n",
      "15S-NT-42O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-43M-1.assembled.fastq\r\n",
      "15S-NT-43M-1.discarded.fastq\r\n",
      "15S-NT-43M-1.unassembled.forward.fastq\r\n",
      "15S-NT-43M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-43M-2.assembled.fastq\r\n",
      "15S-NT-43M-2.discarded.fastq\r\n",
      "15S-NT-43M-2.unassembled.forward.fastq\r\n",
      "15S-NT-43M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-43O-1.assembled.fastq\r\n",
      "15S-NT-43O-1.discarded.fastq\r\n",
      "15S-NT-43O-1.unassembled.forward.fastq\r\n",
      "15S-NT-43O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-43O-2.assembled.fastq\r\n",
      "15S-NT-43O-2.discarded.fastq\r\n",
      "15S-NT-43O-2.unassembled.forward.fastq\r\n",
      "15S-NT-43O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-44M-1.assembled.fastq\r\n",
      "15S-NT-44M-1.discarded.fastq\r\n",
      "15S-NT-44M-1.unassembled.forward.fastq\r\n",
      "15S-NT-44M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-44M-2.assembled.fastq\r\n",
      "15S-NT-44M-2.discarded.fastq\r\n",
      "15S-NT-44M-2.unassembled.forward.fastq\r\n",
      "15S-NT-44M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-44O-1.assembled.fastq\r\n",
      "15S-NT-44O-1.discarded.fastq\r\n",
      "15S-NT-44O-1.unassembled.forward.fastq\r\n",
      "15S-NT-44O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-44O-2.assembled.fastq\r\n",
      "15S-NT-44O-2.discarded.fastq\r\n",
      "15S-NT-44O-2.unassembled.forward.fastq\r\n",
      "15S-NT-44O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-45O-1.assembled.fastq\r\n",
      "15S-NT-45O-1.discarded.fastq\r\n",
      "15S-NT-45O-1.unassembled.forward.fastq\r\n",
      "15S-NT-45O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-45O-2.assembled.fastq\r\n",
      "15S-NT-45O-2.discarded.fastq\r\n",
      "15S-NT-45O-2.unassembled.forward.fastq\r\n",
      "15S-NT-45O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-46M-1.assembled.fastq\r\n",
      "15S-NT-46M-1.discarded.fastq\r\n",
      "15S-NT-46M-1.unassembled.forward.fastq\r\n",
      "15S-NT-46M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-46M-2.assembled.fastq\r\n",
      "15S-NT-46M-2.discarded.fastq\r\n",
      "15S-NT-46M-2.unassembled.forward.fastq\r\n",
      "15S-NT-46M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-47M-1.assembled.fastq\r\n",
      "15S-NT-47M-1.discarded.fastq\r\n",
      "15S-NT-47M-1.unassembled.forward.fastq\r\n",
      "15S-NT-47M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-47M-2.assembled.fastq\r\n",
      "15S-NT-47M-2.discarded.fastq\r\n",
      "15S-NT-47M-2.unassembled.forward.fastq\r\n",
      "15S-NT-47M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-47O-1.assembled.fastq\r\n",
      "15S-NT-47O-1.discarded.fastq\r\n",
      "15S-NT-47O-1.unassembled.forward.fastq\r\n",
      "15S-NT-47O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-47O-2.assembled.fastq\r\n",
      "15S-NT-47O-2.discarded.fastq\r\n",
      "15S-NT-47O-2.unassembled.forward.fastq\r\n",
      "15S-NT-47O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-48O-1.assembled.fastq\r\n",
      "15S-NT-48O-1.discarded.fastq\r\n",
      "15S-NT-48O-1.unassembled.forward.fastq\r\n",
      "15S-NT-48O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-48O-2.assembled.fastq\r\n",
      "15S-NT-48O-2.discarded.fastq\r\n",
      "15S-NT-48O-2.unassembled.forward.fastq\r\n",
      "15S-NT-48O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-49M-1.assembled.fastq\r\n",
      "15S-NT-49M-1.discarded.fastq\r\n",
      "15S-NT-49M-1.unassembled.forward.fastq\r\n",
      "15S-NT-49M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-49M-2.assembled.fastq\r\n",
      "15S-NT-49M-2.discarded.fastq\r\n",
      "15S-NT-49M-2.unassembled.forward.fastq\r\n",
      "15S-NT-49M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-49O-1.assembled.fastq\r\n",
      "15S-NT-49O-1.discarded.fastq\r\n",
      "15S-NT-49O-1.unassembled.forward.fastq\r\n",
      "15S-NT-49O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-49O-2.assembled.fastq\r\n",
      "15S-NT-49O-2.discarded.fastq\r\n",
      "15S-NT-49O-2.unassembled.forward.fastq\r\n",
      "15S-NT-49O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-50O-1.assembled.fastq\r\n",
      "15S-NT-50O-1.discarded.fastq\r\n",
      "15S-NT-50O-1.unassembled.forward.fastq\r\n",
      "15S-NT-50O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-50O-2.assembled.fastq\r\n",
      "15S-NT-50O-2.discarded.fastq\r\n",
      "15S-NT-50O-2.unassembled.forward.fastq\r\n",
      "15S-NT-50O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-51O-1.assembled.fastq\r\n",
      "15S-NT-51O-1.discarded.fastq\r\n",
      "15S-NT-51O-1.unassembled.forward.fastq\r\n",
      "15S-NT-51O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-51O-2.assembled.fastq\r\n",
      "15S-NT-51O-2.discarded.fastq\r\n",
      "15S-NT-51O-2.unassembled.forward.fastq\r\n",
      "15S-NT-51O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-52O-1.assembled.fastq\r\n",
      "15S-NT-52O-1.discarded.fastq\r\n",
      "15S-NT-52O-1.unassembled.forward.fastq\r\n",
      "15S-NT-52O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-52O-2.assembled.fastq\r\n",
      "15S-NT-52O-2.discarded.fastq\r\n",
      "15S-NT-52O-2.unassembled.forward.fastq\r\n",
      "15S-NT-52O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-U06O-1.assembled.fastq\r\n",
      "15S-NT-U06O-1.discarded.fastq\r\n",
      "15S-NT-U06O-1.unassembled.forward.fastq\r\n",
      "15S-NT-U06O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-U06O-2.assembled.fastq\r\n",
      "15S-NT-U06O-2.discarded.fastq\r\n",
      "15S-NT-U06O-2.unassembled.forward.fastq\r\n",
      "15S-NT-U06O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-U07M-1.assembled.fastq\r\n",
      "15S-NT-U07M-1.discarded.fastq\r\n",
      "15S-NT-U07M-1.unassembled.forward.fastq\r\n",
      "15S-NT-U07M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-U07M-2.assembled.fastq\r\n",
      "15S-NT-U07M-2.discarded.fastq\r\n",
      "15S-NT-U07M-2.unassembled.forward.fastq\r\n",
      "15S-NT-U07M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-U07O-1.assembled.fastq\r\n",
      "15S-NT-U07O-1.discarded.fastq\r\n",
      "15S-NT-U07O-1.unassembled.forward.fastq\r\n",
      "15S-NT-U07O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-U07O-2.assembled.fastq\r\n",
      "15S-NT-U07O-2.discarded.fastq\r\n",
      "15S-NT-U07O-2.unassembled.forward.fastq\r\n",
      "15S-NT-U07O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-U08M-1.assembled.fastq\r\n",
      "15S-NT-U08M-1.discarded.fastq\r\n",
      "15S-NT-U08M-1.unassembled.forward.fastq\r\n",
      "15S-NT-U08M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-U08M-2.assembled.fastq\r\n",
      "15S-NT-U08M-2.discarded.fastq\r\n",
      "15S-NT-U08M-2.unassembled.forward.fastq\r\n",
      "15S-NT-U08M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-U08O-1.assembled.fastq\r\n",
      "15S-NT-U08O-1.discarded.fastq\r\n",
      "15S-NT-U08O-1.unassembled.forward.fastq\r\n",
      "15S-NT-U08O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-U08O-2.assembled.fastq\r\n",
      "15S-NT-U08O-2.discarded.fastq\r\n",
      "15S-NT-U08O-2.unassembled.forward.fastq\r\n",
      "15S-NT-U08O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-U09M-1.assembled.fastq\r\n",
      "15S-NT-U09M-1.discarded.fastq\r\n",
      "15S-NT-U09M-1.unassembled.forward.fastq\r\n",
      "15S-NT-U09M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-U09M-2.assembled.fastq\r\n",
      "15S-NT-U09M-2.discarded.fastq\r\n",
      "15S-NT-U09M-2.unassembled.forward.fastq\r\n",
      "15S-NT-U09M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-U09O-1.assembled.fastq\r\n",
      "15S-NT-U09O-1.discarded.fastq\r\n",
      "15S-NT-U09O-1.unassembled.forward.fastq\r\n",
      "15S-NT-U09O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-U09O-2.assembled.fastq\r\n",
      "15S-NT-U09O-2.discarded.fastq\r\n",
      "15S-NT-U09O-2.unassembled.forward.fastq\r\n",
      "15S-NT-U09O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-U10M-1.assembled.fastq\r\n",
      "15S-NT-U10M-1.discarded.fastq\r\n",
      "15S-NT-U10M-1.unassembled.forward.fastq\r\n",
      "15S-NT-U10M-1.unassembled.reverse.fastq\r\n",
      "15S-NT-U10M-2.assembled.fastq\r\n",
      "15S-NT-U10M-2.discarded.fastq\r\n",
      "15S-NT-U10M-2.unassembled.forward.fastq\r\n",
      "15S-NT-U10M-2.unassembled.reverse.fastq\r\n",
      "15S-NT-U10O-1.assembled.fastq\r\n",
      "15S-NT-U10O-1.discarded.fastq\r\n",
      "15S-NT-U10O-1.unassembled.forward.fastq\r\n",
      "15S-NT-U10O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-U10O-2.assembled.fastq\r\n",
      "15S-NT-U10O-2.discarded.fastq\r\n",
      "15S-NT-U10O-2.unassembled.forward.fastq\r\n",
      "15S-NT-U10O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-U11O-1.assembled.fastq\r\n",
      "15S-NT-U11O-1.discarded.fastq\r\n",
      "15S-NT-U11O-1.unassembled.forward.fastq\r\n",
      "15S-NT-U11O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-U11O-2.assembled.fastq\r\n",
      "15S-NT-U11O-2.discarded.fastq\r\n",
      "15S-NT-U11O-2.unassembled.forward.fastq\r\n",
      "15S-NT-U11O-2.unassembled.reverse.fastq\r\n",
      "15S-NT-U12O-1.assembled.fastq\r\n",
      "15S-NT-U12O-1.discarded.fastq\r\n",
      "15S-NT-U12O-1.unassembled.forward.fastq\r\n",
      "15S-NT-U12O-1.unassembled.reverse.fastq\r\n",
      "15S-NT-U12O-2.assembled.fastq\r\n",
      "15S-NT-U12O-2.discarded.fastq\r\n",
      "15S-NT-U12O-2.unassembled.forward.fastq\r\n",
      "15S-NT-U12O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-01M-1-10x.assembled.fastq\r\n",
      "15S-WB-01M-1-10x.discarded.fastq\r\n",
      "15S-WB-01M-1-10x.unassembled.forward.fastq\r\n",
      "15S-WB-01M-1-10x.unassembled.reverse.fastq\r\n",
      "15S-WB-01M-2.assembled.fastq\r\n",
      "15S-WB-01M-2.discarded.fastq\r\n",
      "15S-WB-01M-2.unassembled.forward.fastq\r\n",
      "15S-WB-01M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-01O-1-10x.assembled.fastq\r\n",
      "15S-WB-01O-1-10x.discarded.fastq\r\n",
      "15S-WB-01O-1-10x.unassembled.forward.fastq\r\n",
      "15S-WB-01O-1-10x.unassembled.reverse.fastq\r\n",
      "15S-WB-01O-2-10x.assembled.fastq\r\n",
      "15S-WB-01O-2-10x.discarded.fastq\r\n",
      "15S-WB-01O-2-10x.unassembled.forward.fastq\r\n",
      "15S-WB-01O-2-10x.unassembled.reverse.fastq\r\n",
      "15S-WB-02O-1.assembled.fastq\r\n",
      "15S-WB-02O-1.discarded.fastq\r\n",
      "15S-WB-02O-1.unassembled.forward.fastq\r\n",
      "15S-WB-02O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-02O-2.assembled.fastq\r\n",
      "15S-WB-02O-2.discarded.fastq\r\n",
      "15S-WB-02O-2.unassembled.forward.fastq\r\n",
      "15S-WB-02O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-04M-1.assembled.fastq\r\n",
      "15S-WB-04M-1.discarded.fastq\r\n",
      "15S-WB-04M-1.unassembled.forward.fastq\r\n",
      "15S-WB-04M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-04M-2.assembled.fastq\r\n",
      "15S-WB-04M-2.discarded.fastq\r\n",
      "15S-WB-04M-2.unassembled.forward.fastq\r\n",
      "15S-WB-04M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-05M-1.assembled.fastq\r\n",
      "15S-WB-05M-1.discarded.fastq\r\n",
      "15S-WB-05M-1.unassembled.forward.fastq\r\n",
      "15S-WB-05M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-05M-2.assembled.fastq\r\n",
      "15S-WB-05M-2.discarded.fastq\r\n",
      "15S-WB-05M-2.unassembled.forward.fastq\r\n",
      "15S-WB-05M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-05O-1.assembled.fastq\r\n",
      "15S-WB-05O-1.discarded.fastq\r\n",
      "15S-WB-05O-1.unassembled.forward.fastq\r\n",
      "15S-WB-05O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-05O-2.assembled.fastq\r\n",
      "15S-WB-05O-2.discarded.fastq\r\n",
      "15S-WB-05O-2.unassembled.forward.fastq\r\n",
      "15S-WB-05O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-06M-1.assembled.fastq\r\n",
      "15S-WB-06M-1.discarded.fastq\r\n",
      "15S-WB-06M-1.unassembled.forward.fastq\r\n",
      "15S-WB-06M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-06M-2.assembled.fastq\r\n",
      "15S-WB-06M-2.discarded.fastq\r\n",
      "15S-WB-06M-2.unassembled.forward.fastq\r\n",
      "15S-WB-06M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-06O-1.assembled.fastq\r\n",
      "15S-WB-06O-1.discarded.fastq\r\n",
      "15S-WB-06O-1.unassembled.forward.fastq\r\n",
      "15S-WB-06O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-06O-2.assembled.fastq\r\n",
      "15S-WB-06O-2.discarded.fastq\r\n",
      "15S-WB-06O-2.unassembled.forward.fastq\r\n",
      "15S-WB-06O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-07M-1.assembled.fastq\r\n",
      "15S-WB-07M-1.discarded.fastq\r\n",
      "15S-WB-07M-1.unassembled.forward.fastq\r\n",
      "15S-WB-07M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-07M-2.assembled.fastq\r\n",
      "15S-WB-07M-2.discarded.fastq\r\n",
      "15S-WB-07M-2.unassembled.forward.fastq\r\n",
      "15S-WB-07M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-07O-1.assembled.fastq\r\n",
      "15S-WB-07O-1.discarded.fastq\r\n",
      "15S-WB-07O-1.unassembled.forward.fastq\r\n",
      "15S-WB-07O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-07O-2.assembled.fastq\r\n",
      "15S-WB-07O-2.discarded.fastq\r\n",
      "15S-WB-07O-2.unassembled.forward.fastq\r\n",
      "15S-WB-07O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-08M-1.assembled.fastq\r\n",
      "15S-WB-08M-1.discarded.fastq\r\n",
      "15S-WB-08M-1.unassembled.forward.fastq\r\n",
      "15S-WB-08M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-08M-2.assembled.fastq\r\n",
      "15S-WB-08M-2.discarded.fastq\r\n",
      "15S-WB-08M-2.unassembled.forward.fastq\r\n",
      "15S-WB-08M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-08O-1.assembled.fastq\r\n",
      "15S-WB-08O-1.discarded.fastq\r\n",
      "15S-WB-08O-1.unassembled.forward.fastq\r\n",
      "15S-WB-08O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-08O-2.assembled.fastq\r\n",
      "15S-WB-08O-2.discarded.fastq\r\n",
      "15S-WB-08O-2.unassembled.forward.fastq\r\n",
      "15S-WB-08O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-09M-1-R.assembled.fastq\r\n",
      "15S-WB-09M-1-R.discarded.fastq\r\n",
      "15S-WB-09M-1-R.unassembled.forward.fastq\r\n",
      "15S-WB-09M-1-R.unassembled.reverse.fastq\r\n",
      "15S-WB-09M-2-R.assembled.fastq\r\n",
      "15S-WB-09M-2-R.discarded.fastq\r\n",
      "15S-WB-09M-2-R.unassembled.forward.fastq\r\n",
      "15S-WB-09M-2-R.unassembled.reverse.fastq\r\n",
      "15S-WB-09O-1-R.assembled.fastq\r\n",
      "15S-WB-09O-1-R.discarded.fastq\r\n",
      "15S-WB-09O-1-R.unassembled.forward.fastq\r\n",
      "15S-WB-09O-1-R.unassembled.reverse.fastq\r\n",
      "15S-WB-09O-2-R.assembled.fastq\r\n",
      "15S-WB-09O-2-R.discarded.fastq\r\n",
      "15S-WB-09O-2-R.unassembled.forward.fastq\r\n",
      "15S-WB-09O-2-R.unassembled.reverse.fastq\r\n",
      "15S-WB-10M-1.assembled.fastq\r\n",
      "15S-WB-10M-1.discarded.fastq\r\n",
      "15S-WB-10M-1.unassembled.forward.fastq\r\n",
      "15S-WB-10M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-10M-2.assembled.fastq\r\n",
      "15S-WB-10M-2.discarded.fastq\r\n",
      "15S-WB-10M-2.unassembled.forward.fastq\r\n",
      "15S-WB-10M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-10O-1.assembled.fastq\r\n",
      "15S-WB-10O-1.discarded.fastq\r\n",
      "15S-WB-10O-1.unassembled.forward.fastq\r\n",
      "15S-WB-10O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-10O-2.assembled.fastq\r\n",
      "15S-WB-10O-2.discarded.fastq\r\n",
      "15S-WB-10O-2.unassembled.forward.fastq\r\n",
      "15S-WB-10O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-11M-1.assembled.fastq\r\n",
      "15S-WB-11M-1.discarded.fastq\r\n",
      "15S-WB-11M-1.unassembled.forward.fastq\r\n",
      "15S-WB-11M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-11M-2.assembled.fastq\r\n",
      "15S-WB-11M-2.discarded.fastq\r\n",
      "15S-WB-11M-2.unassembled.forward.fastq\r\n",
      "15S-WB-11M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-11O-1.assembled.fastq\r\n",
      "15S-WB-11O-1.discarded.fastq\r\n",
      "15S-WB-11O-1.unassembled.forward.fastq\r\n",
      "15S-WB-11O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-11O-2.assembled.fastq\r\n",
      "15S-WB-11O-2.discarded.fastq\r\n",
      "15S-WB-11O-2.unassembled.forward.fastq\r\n",
      "15S-WB-11O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-12M-1.assembled.fastq\r\n",
      "15S-WB-12M-1.discarded.fastq\r\n",
      "15S-WB-12M-1.unassembled.forward.fastq\r\n",
      "15S-WB-12M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-12M-2.assembled.fastq\r\n",
      "15S-WB-12M-2.discarded.fastq\r\n",
      "15S-WB-12M-2.unassembled.forward.fastq\r\n",
      "15S-WB-12M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-12O-1.assembled.fastq\r\n",
      "15S-WB-12O-1.discarded.fastq\r\n",
      "15S-WB-12O-1.unassembled.forward.fastq\r\n",
      "15S-WB-12O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-12O-2.assembled.fastq\r\n",
      "15S-WB-12O-2.discarded.fastq\r\n",
      "15S-WB-12O-2.unassembled.forward.fastq\r\n",
      "15S-WB-12O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-13O-1.assembled.fastq\r\n",
      "15S-WB-13O-1.discarded.fastq\r\n",
      "15S-WB-13O-1.unassembled.forward.fastq\r\n",
      "15S-WB-13O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-13O-2.assembled.fastq\r\n",
      "15S-WB-13O-2.discarded.fastq\r\n",
      "15S-WB-13O-2.unassembled.forward.fastq\r\n",
      "15S-WB-13O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-14M-1.assembled.fastq\r\n",
      "15S-WB-14M-1.discarded.fastq\r\n",
      "15S-WB-14M-1.unassembled.forward.fastq\r\n",
      "15S-WB-14M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-14M-2.assembled.fastq\r\n",
      "15S-WB-14M-2.discarded.fastq\r\n",
      "15S-WB-14M-2.unassembled.forward.fastq\r\n",
      "15S-WB-14M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-14O-1.assembled.fastq\r\n",
      "15S-WB-14O-1.discarded.fastq\r\n",
      "15S-WB-14O-1.unassembled.forward.fastq\r\n",
      "15S-WB-14O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-14O-2.assembled.fastq\r\n",
      "15S-WB-14O-2.discarded.fastq\r\n",
      "15S-WB-14O-2.unassembled.forward.fastq\r\n",
      "15S-WB-14O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-15M-1.assembled.fastq\r\n",
      "15S-WB-15M-1.discarded.fastq\r\n",
      "15S-WB-15M-1.unassembled.forward.fastq\r\n",
      "15S-WB-15M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-15M-2.assembled.fastq\r\n",
      "15S-WB-15M-2.discarded.fastq\r\n",
      "15S-WB-15M-2.unassembled.forward.fastq\r\n",
      "15S-WB-15M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-15O-1.assembled.fastq\r\n",
      "15S-WB-15O-1.discarded.fastq\r\n",
      "15S-WB-15O-1.unassembled.forward.fastq\r\n",
      "15S-WB-15O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-15O-2.assembled.fastq\r\n",
      "15S-WB-15O-2.discarded.fastq\r\n",
      "15S-WB-15O-2.unassembled.forward.fastq\r\n",
      "15S-WB-15O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-16M-1.assembled.fastq\r\n",
      "15S-WB-16M-1.discarded.fastq\r\n",
      "15S-WB-16M-1.unassembled.forward.fastq\r\n",
      "15S-WB-16M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-16M-2.assembled.fastq\r\n",
      "15S-WB-16M-2.discarded.fastq\r\n",
      "15S-WB-16M-2.unassembled.forward.fastq\r\n",
      "15S-WB-16M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-16O-1.assembled.fastq\r\n",
      "15S-WB-16O-1.discarded.fastq\r\n",
      "15S-WB-16O-1.unassembled.forward.fastq\r\n",
      "15S-WB-16O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-16O-2.assembled.fastq\r\n",
      "15S-WB-16O-2.discarded.fastq\r\n",
      "15S-WB-16O-2.unassembled.forward.fastq\r\n",
      "15S-WB-16O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-17M-1.assembled.fastq\r\n",
      "15S-WB-17M-1.discarded.fastq\r\n",
      "15S-WB-17M-1.unassembled.forward.fastq\r\n",
      "15S-WB-17M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-17M-2.assembled.fastq\r\n",
      "15S-WB-17M-2.discarded.fastq\r\n",
      "15S-WB-17M-2.unassembled.forward.fastq\r\n",
      "15S-WB-17M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-17O-1.assembled.fastq\r\n",
      "15S-WB-17O-1.discarded.fastq\r\n",
      "15S-WB-17O-1.unassembled.forward.fastq\r\n",
      "15S-WB-17O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-17O-2.assembled.fastq\r\n",
      "15S-WB-17O-2.discarded.fastq\r\n",
      "15S-WB-17O-2.unassembled.forward.fastq\r\n",
      "15S-WB-17O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-18M-1.assembled.fastq\r\n",
      "15S-WB-18M-1.discarded.fastq\r\n",
      "15S-WB-18M-1.unassembled.forward.fastq\r\n",
      "15S-WB-18M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-18M-2.assembled.fastq\r\n",
      "15S-WB-18M-2.discarded.fastq\r\n",
      "15S-WB-18M-2.unassembled.forward.fastq\r\n",
      "15S-WB-18M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-U01M-1.assembled.fastq\r\n",
      "15S-WB-U01M-1.discarded.fastq\r\n",
      "15S-WB-U01M-1.unassembled.forward.fastq\r\n",
      "15S-WB-U01M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-U01M-2.assembled.fastq\r\n",
      "15S-WB-U01M-2.discarded.fastq\r\n",
      "15S-WB-U01M-2.unassembled.forward.fastq\r\n",
      "15S-WB-U01M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-U01O-1.assembled.fastq\r\n",
      "15S-WB-U01O-1.discarded.fastq\r\n",
      "15S-WB-U01O-1.unassembled.forward.fastq\r\n",
      "15S-WB-U01O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-U01O-2.assembled.fastq\r\n",
      "15S-WB-U01O-2.discarded.fastq\r\n",
      "15S-WB-U01O-2.unassembled.forward.fastq\r\n",
      "15S-WB-U01O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-U02O-1.assembled.fastq\r\n",
      "15S-WB-U02O-1.discarded.fastq\r\n",
      "15S-WB-U02O-1.unassembled.forward.fastq\r\n",
      "15S-WB-U02O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-U02O-2.assembled.fastq\r\n",
      "15S-WB-U02O-2.discarded.fastq\r\n",
      "15S-WB-U02O-2.unassembled.forward.fastq\r\n",
      "15S-WB-U02O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-U03M-1.assembled.fastq\r\n",
      "15S-WB-U03M-1.discarded.fastq\r\n",
      "15S-WB-U03M-1.unassembled.forward.fastq\r\n",
      "15S-WB-U03M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-U03M-2.assembled.fastq\r\n",
      "15S-WB-U03M-2.discarded.fastq\r\n",
      "15S-WB-U03M-2.unassembled.forward.fastq\r\n",
      "15S-WB-U03M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-U03O-1.assembled.fastq\r\n",
      "15S-WB-U03O-1.discarded.fastq\r\n",
      "15S-WB-U03O-1.unassembled.forward.fastq\r\n",
      "15S-WB-U03O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-U03O-2.assembled.fastq\r\n",
      "15S-WB-U03O-2.discarded.fastq\r\n",
      "15S-WB-U03O-2.unassembled.forward.fastq\r\n",
      "15S-WB-U03O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-U04M-1.assembled.fastq\r\n",
      "15S-WB-U04M-1.discarded.fastq\r\n",
      "15S-WB-U04M-1.unassembled.forward.fastq\r\n",
      "15S-WB-U04M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-U04M-2.assembled.fastq\r\n",
      "15S-WB-U04M-2.discarded.fastq\r\n",
      "15S-WB-U04M-2.unassembled.forward.fastq\r\n",
      "15S-WB-U04M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-U04O-1.assembled.fastq\r\n",
      "15S-WB-U04O-1.discarded.fastq\r\n",
      "15S-WB-U04O-1.unassembled.forward.fastq\r\n",
      "15S-WB-U04O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-U04O-2.assembled.fastq\r\n",
      "15S-WB-U04O-2.discarded.fastq\r\n",
      "15S-WB-U04O-2.unassembled.forward.fastq\r\n",
      "15S-WB-U04O-2.unassembled.reverse.fastq\r\n",
      "15S-WB-U05M-1.assembled.fastq\r\n",
      "15S-WB-U05M-1.discarded.fastq\r\n",
      "15S-WB-U05M-1.unassembled.forward.fastq\r\n",
      "15S-WB-U05M-1.unassembled.reverse.fastq\r\n",
      "15S-WB-U05M-2.assembled.fastq\r\n",
      "15S-WB-U05M-2.discarded.fastq\r\n",
      "15S-WB-U05M-2.unassembled.forward.fastq\r\n",
      "15S-WB-U05M-2.unassembled.reverse.fastq\r\n",
      "15S-WB-U05O-1.assembled.fastq\r\n",
      "15S-WB-U05O-1.discarded.fastq\r\n",
      "15S-WB-U05O-1.unassembled.forward.fastq\r\n",
      "15S-WB-U05O-1.unassembled.reverse.fastq\r\n",
      "15S-WB-U05O-2.assembled.fastq\r\n",
      "15S-WB-U05O-2.discarded.fastq\r\n",
      "15S-WB-U05O-2.unassembled.forward.fastq\r\n",
      "15S-WB-U05O-2.unassembled.reverse.fastq\r\n",
      "Blank-1.assembled.fastq\r\n",
      "Blank-1.discarded.fastq\r\n",
      "Blank-1.unassembled.forward.fastq\r\n",
      "Blank-1.unassembled.reverse.fastq\r\n",
      "Blank-10.assembled.fastq\r\n",
      "Blank-10.discarded.fastq\r\n",
      "Blank-10.unassembled.forward.fastq\r\n",
      "Blank-10.unassembled.reverse.fastq\r\n",
      "Blank-11.assembled.fastq\r\n",
      "Blank-11.discarded.fastq\r\n",
      "Blank-11.unassembled.forward.fastq\r\n",
      "Blank-11.unassembled.reverse.fastq\r\n",
      "Blank-13.assembled.fastq\r\n",
      "Blank-13.discarded.fastq\r\n",
      "Blank-13.unassembled.forward.fastq\r\n",
      "Blank-13.unassembled.reverse.fastq\r\n",
      "Blank-16.assembled.fastq\r\n",
      "Blank-16.discarded.fastq\r\n",
      "Blank-16.unassembled.forward.fastq\r\n",
      "Blank-16.unassembled.reverse.fastq\r\n",
      "Blank-17.assembled.fastq\r\n",
      "Blank-17.discarded.fastq\r\n",
      "Blank-17.unassembled.forward.fastq\r\n",
      "Blank-17.unassembled.reverse.fastq\r\n",
      "Blank-22.assembled.fastq\r\n",
      "Blank-22.discarded.fastq\r\n",
      "Blank-22.unassembled.forward.fastq\r\n",
      "Blank-22.unassembled.reverse.fastq\r\n",
      "Blank-4.assembled.fastq\r\n",
      "Blank-4.discarded.fastq\r\n",
      "Blank-4.unassembled.forward.fastq\r\n",
      "Blank-4.unassembled.reverse.fastq\r\n",
      "Blank-6.assembled.fastq\r\n",
      "Blank-6.discarded.fastq\r\n",
      "Blank-6.unassembled.forward.fastq\r\n",
      "Blank-6.unassembled.reverse.fastq\r\n",
      "Blank-8.assembled.fastq\r\n",
      "Blank-8.discarded.fastq\r\n",
      "Blank-8.unassembled.forward.fastq\r\n",
      "Blank-8.unassembled.reverse.fastq\r\n",
      "DOE-BS-1.assembled.fastq\r\n",
      "DOE-BS-1.discarded.fastq\r\n",
      "DOE-BS-1.unassembled.forward.fastq\r\n",
      "DOE-BS-1.unassembled.reverse.fastq\r\n",
      "DOE-BS-2.assembled.fastq\r\n",
      "DOE-BS-2.discarded.fastq\r\n",
      "DOE-BS-2.unassembled.forward.fastq\r\n",
      "DOE-BS-2.unassembled.reverse.fastq\r\n",
      "DOE-BS-3.assembled.fastq\r\n",
      "DOE-BS-3.discarded.fastq\r\n",
      "DOE-BS-3.unassembled.forward.fastq\r\n",
      "DOE-BS-3.unassembled.reverse.fastq\r\n",
      "DOE-BS-4.assembled.fastq\r\n",
      "DOE-BS-4.discarded.fastq\r\n",
      "DOE-BS-4.unassembled.forward.fastq\r\n",
      "DOE-BS-4.unassembled.reverse.fastq\r\n",
      "DOE-Blank.assembled.fastq\r\n",
      "DOE-Blank.discarded.fastq\r\n",
      "DOE-Blank.unassembled.forward.fastq\r\n",
      "DOE-Blank.unassembled.reverse.fastq\r\n",
      "DOE-UBS-1.assembled.fastq\r\n",
      "DOE-UBS-1.discarded.fastq\r\n",
      "DOE-UBS-1.unassembled.forward.fastq\r\n",
      "DOE-UBS-1.unassembled.reverse.fastq\r\n",
      "DOE-UBS-2.assembled.fastq\r\n",
      "DOE-UBS-2.discarded.fastq\r\n",
      "DOE-UBS-2.unassembled.forward.fastq\r\n",
      "DOE-UBS-2.unassembled.reverse.fastq\r\n",
      "DOE-UBS-3.assembled.fastq\r\n",
      "DOE-UBS-3.discarded.fastq\r\n",
      "DOE-UBS-3.unassembled.forward.fastq\r\n",
      "DOE-UBS-3.unassembled.reverse.fastq\r\n",
      "DOE-UBS-4.assembled.fastq\r\n",
      "DOE-UBS-4.discarded.fastq\r\n",
      "DOE-UBS-4.unassembled.forward.fastq\r\n",
      "DOE-UBS-4.unassembled.reverse.fastq\r\n",
      "JP-CFB-09M.assembled.fastq\r\n",
      "JP-CFB-09M.discarded.fastq\r\n",
      "JP-CFB-09M.unassembled.forward.fastq\r\n",
      "JP-CFB-09M.unassembled.reverse.fastq\r\n",
      "JP-CFB-09T.assembled.fastq\r\n",
      "JP-CFB-09T.discarded.fastq\r\n",
      "JP-CFB-09T.unassembled.forward.fastq\r\n",
      "JP-CFB-09T.unassembled.reverse.fastq\r\n",
      "JP-CFB-11M.assembled.fastq\r\n",
      "JP-CFB-11M.discarded.fastq\r\n",
      "JP-CFB-11M.unassembled.forward.fastq\r\n",
      "JP-CFB-11M.unassembled.reverse.fastq\r\n",
      "JP-CFB-11T.assembled.fastq\r\n",
      "JP-CFB-11T.discarded.fastq\r\n",
      "JP-CFB-11T.unassembled.forward.fastq\r\n",
      "JP-CFB-11T.unassembled.reverse.fastq\r\n",
      "JP-CFB-17M.assembled.fastq\r\n",
      "JP-CFB-17M.discarded.fastq\r\n",
      "JP-CFB-17M.unassembled.forward.fastq\r\n",
      "JP-CFB-17M.unassembled.reverse.fastq\r\n",
      "JP-CFB-17T.assembled.fastq\r\n",
      "JP-CFB-17T.discarded.fastq\r\n",
      "JP-CFB-17T.unassembled.forward.fastq\r\n",
      "JP-CFB-17T.unassembled.reverse.fastq\r\n",
      "JP-CFB-23M.assembled.fastq\r\n",
      "JP-CFB-23M.discarded.fastq\r\n",
      "JP-CFB-23M.unassembled.forward.fastq\r\n",
      "JP-CFB-23M.unassembled.reverse.fastq\r\n",
      "JP-CFB-23T.assembled.fastq\r\n",
      "JP-CFB-23T.discarded.fastq\r\n",
      "JP-CFB-23T.unassembled.forward.fastq\r\n",
      "JP-CFB-23T.unassembled.reverse.fastq\r\n",
      "JP-CFB-30M.assembled.fastq\r\n",
      "JP-CFB-30M.discarded.fastq\r\n",
      "JP-CFB-30M.unassembled.forward.fastq\r\n",
      "JP-CFB-30M.unassembled.reverse.fastq\r\n",
      "JP-CFB-30T.assembled.fastq\r\n",
      "JP-CFB-30T.discarded.fastq\r\n",
      "JP-CFB-30T.unassembled.forward.fastq\r\n",
      "JP-CFB-30T.unassembled.reverse.fastq\r\n",
      "JP-CFB-53M.assembled.fastq\r\n",
      "JP-CFB-53M.discarded.fastq\r\n",
      "JP-CFB-53M.unassembled.forward.fastq\r\n",
      "JP-CFB-53M.unassembled.reverse.fastq\r\n",
      "JP-CFB-53T.assembled.fastq\r\n",
      "JP-CFB-53T.discarded.fastq\r\n",
      "JP-CFB-53T.unassembled.forward.fastq\r\n",
      "JP-CFB-53T.unassembled.reverse.fastq\r\n",
      "JP-CFB-Blank1.assembled.fastq\r\n",
      "JP-CFB-Blank1.discarded.fastq\r\n",
      "JP-CFB-Blank1.unassembled.forward.fastq\r\n",
      "JP-CFB-Blank1.unassembled.reverse.fastq\r\n"
     ]
    }
   ],
   "source": [
    "!ls ../data/Seq_data/MergedSeqs/\n",
    "# Check to see the files"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Making sure primers aren't still attached"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!head -40000 ../data/Seq_data/MergedSeqs/15S-NT-19O-1.assembled.fastq > test.assembled.fastq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 1.8.1 with Python 2.7.9\n",
      "Command line parameters: -g TTACCGCGGCKGCTGRCAC -O 12 -o test.assembled.fastq.cut test.assembled.fastq\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.16 s (16 us/read; 3.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  10,000\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "Reads written (passing filters):        10,000 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,530,368 bp\n",
      "Total written (filtered):      2,530,368 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: TTACCGCGGCKGCTGRCAC; Type: regular 5'; Length: 19; Trimmed: 0 times.\n",
      "\n",
      "This is cutadapt 1.8.1 with Python 2.7.9\n",
      "Command line parameters: -g GTGYCAGCMGCCGCGGTAA -O 12 -o test.assembled.fastq.cut test.assembled.fastq\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.17 s (17 us/read; 3.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  10,000\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "Reads written (passing filters):        10,000 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,530,368 bp\n",
      "Total written (filtered):      2,530,368 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: GTGYCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 0 times.\n",
      "\n",
      "This is cutadapt 1.8.1 with Python 2.7.9\n",
      "Command line parameters: -a GGACTACNVGGGTWTCTAAT -O 12 -o test.assembled.fastq.cut test.assembled.fastq\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.19 s (19 us/read; 3.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  10,000\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "Reads written (passing filters):        10,000 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,530,368 bp\n",
      "Total written (filtered):      2,530,368 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: GGACTACNVGGGTWTCTAAT; Type: regular 3'; Length: 20; Trimmed: 0 times.\n",
      "\n",
      "This is cutadapt 1.8.1 with Python 2.7.9\n",
      "Command line parameters: -a ATTAGAWACCCBNGTAGTCC -O 12 -o test.assembled.fastq.cut test.assembled.fastq\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.17 s (17 us/read; 3.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  10,000\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "Reads written (passing filters):        10,000 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,530,368 bp\n",
      "Total written (filtered):      2,530,368 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: ATTAGAWACCCBNGTAGTCC; Type: regular 3'; Length: 20; Trimmed: 0 times.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!cutadapt -g TTACCGCGGCKGCTGRCAC -O 12 -o test.assembled.fastq.cut test.assembled.fastq\n",
    "!cutadapt -g GTGYCAGCMGCCGCGGTAA -O 12 -o test.assembled.fastq.cut test.assembled.fastq\n",
    "!cutadapt -a GGACTACNVGGGTWTCTAAT -O 12 -o test.assembled.fastq.cut test.assembled.fastq\n",
    "!cutadapt -a ATTAGAWACCCBNGTAGTCC -O 12 -o test.assembled.fastq.cut test.assembled.fastq\n",
    "# Just making sure that the primers aren't still attached"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Spot-checking quality stats again after assembly"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!fastx_quality_stats -Q 33 -i ../data/Seq_data/MergedSeqs/Blank-1.assembled.fastq -o test.quality"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/Thea/anaconda3/lib/python3.6/site-packages/rpy2/rinterface/__init__.py:186: RRuntimeWarning: `geom_smooth()` using method = 'loess'\n",
      "\n",
      "  warnings.warn(x, RRuntimeWarning)\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAeAAAAHgCAYAAAB91L6VAAAEDWlDQ1BJQ0MgUHJvZmlsZQAAOI2N\nVV1oHFUUPrtzZyMkzlNsNIV0qD8NJQ2TVjShtLp/3d02bpZJNtoi6GT27s6Yyc44M7v9oU9FUHwx\n6psUxL+3gCAo9Q/bPrQvlQol2tQgKD60+INQ6Ium65k7M5lpurHeZe58853vnnvuuWfvBei5qliW\nkRQBFpquLRcy4nOHj4g9K5CEh6AXBqFXUR0rXalMAjZPC3e1W99Dwntf2dXd/p+tt0YdFSBxH2Kz\n5qgLiI8B8KdVy3YBevqRHz/qWh72Yui3MUDEL3q44WPXw3M+fo1pZuQs4tOIBVVTaoiXEI/MxfhG\nDPsxsNZfoE1q66ro5aJim3XdoLFw72H+n23BaIXzbcOnz5mfPoTvYVz7KzUl5+FRxEuqkp9G/Aji\na219thzg25abkRE/BpDc3pqvphHvRFys2weqvp+krbWKIX7nhDbzLOItiM8358pTwdirqpPFnMF2\nxLc1WvLyOwTAibpbmvHHcvttU57y5+XqNZrLe3lE/Pq8eUj2fXKfOe3pfOjzhJYtB/yll5SDFcSD\niH+hRkH25+L+sdxKEAMZahrlSX8ukqMOWy/jXW2m6M9LDBc31B9LFuv6gVKg/0Szi3KAr1kGq1GM\njU/aLbnq6/lRxc4XfJ98hTargX++DbMJBSiYMIe9Ck1YAxFkKEAG3xbYaKmDDgYyFK0UGYpfoWYX\nG+fAPPI6tJnNwb7ClP7IyF+D+bjOtCpkhz6CFrIa/I6sFtNl8auFXGMTP34sNwI/JhkgEtmDz14y\nSfaRcTIBInmKPE32kxyyE2Tv+thKbEVePDfW/byMM1Kmm0XdObS7oGD/MypMXFPXrCwOtoYjyyn7\nBV29/MZfsVzpLDdRtuIZnbpXzvlf+ev8MvYr/Gqk4H/kV/G3csdazLuyTMPsbFhzd1UabQbjFvDR\nmcWJxR3zcfHkVw9GfpbJmeev9F08WW8uDkaslwX6avlWGU6NRKz0g/SHtCy9J30o/ca9zX3Kfc19\nzn3BXQKRO8ud477hLnAfc1/G9mrzGlrfexZ5GLdn6ZZrrEohI2wVHhZywjbhUWEy8icMCGNCUdiB\nlq3r+xafL549HQ5jH+an+1y+LlYBifuxAvRN/lVVVOlwlCkdVm9NOL5BE4wkQ2SMlDZU97hX86Ei\nlU/lUmkQUztTE6mx1EEPh7OmdqBtAvv8HdWpbrJS6tJj3n0CWdM6busNzRV3S9KTYhqvNiqWmuro\niKgYhshMjmhTh9ptWhsF7970j/SbMrsPE1suR5z7DMC+P/Hs+y7ijrQAlhyAgccjbhjPygfeBTjz\nhNqy28EdkUh8C+DU9+z2v/oyeH791OncxHOs5y2AtTc7nb/f73TWPkD/qwBnjX8BoJ98VVBg/m8A\nAEAASURBVHgB7d0HnFTV+f/xZ9mlw9K7IuhfKYrGjiUWYonG2DtGTLBrDIotarAAlqgYW8QaBY0I\n/JREJSJWFFFRwIJdBAUp0nvd+Z/vGWfcNjuzZe7emfmc12t2Z24599z3Lc8t556bF3HJSAgggAAC\nCCAQqECdQKfGxBBAAAEEEEDACxCAWREQQAABBBCoBQECcC2gM0kEEEAAAQQIwKwDCCCAAAII1IJA\nQS1Ms1qTfPjhh2316tXVykP1zvSpUyd8xx+xOnF5eXnVmsd0jLxlyxZvFsayFRUVhXJ5ajnILT8/\nPx2LpNp5htVN5VIK6zYaxm1AXmyjUqhc0j5X61s6t9F99tnHevfuXaZgGReA58+fb9dee22ZGalM\nhzVr1tjGjRutRYsWlRktkGHXrVtnBQUFVrdu3UCmV5mJLF682Jo0aWINGjSozGiBDLty5UorLCwM\nZFqVmcjmzZtNbu3bt6/MaIEMq53O2rVr/TINZIKVmMiqVav8TrFZs2aVGCuYQbX/0DaQzh12Vedk\n0aJFfr8Wxv1HWLdRxYIVK1ZYmzZtqspe4Xg6KLr//vuzIwDriLi6K77Gj30qlKuFnrFyVXce01H0\nTChbOua7Onnq6DrmVp180jGuzuLCWjaVK1a+dMx7dfKMmel/2FImlA2zXwTCdw32l7LxDQEEEEAA\ngawVIABn7aJlxhBAAAEEwixAAA7z0qFsCCCAAAJZK0AAztpFy4whgAACCIRZgAAc5qVD2RBAAAEE\nslaAAJy1i5YZQwABBBAIswABOMxLh7IhgAACCGStAAE4axctM4YAAgggEGYBAnCYlw5lQwABBBDI\nWgECcNYuWmYMAQQQQCDMAgTgMC8dyoYAAgggkLUCBOCsXbTMGAIIIIBAmAUIwGFeOpQNAQQQQCBr\nBQjAWbtomTEEEEAAgTALEIDDvHQoGwIIIIBA1goQgLN20TJjCCCAAAJhFigIc+EoW+4JrFlnNuuH\n6Oe7uWYrVputXe8+rrv6rXPfC9x70Bs2cJ/6Zg3c/0bu07RhPevcyaxda/dpZdbe/S9sknt+zDEC\nCGSOQOABuKioyObOnWudO3cuo/Tjjz9a27ZtraAg8GKVKQsd0i8QiZh9Pstsygyz9z82+2q22fyf\nqjpdF4VLpeZNzbq4oNx1q1/+/79tzLbpYJbvgjgJgaoKLF8VXVcXuPX1p2Xus9Rs8c//ddC4fkP0\nYHGd+6/vRW5db1DPrP7Pnwbu4LF5odlW7dynvdnW7qP/Wl8bN6xqqRgv0wQCj3TPPPOMjRw50saP\nHx+3+uijj2zo0KH+94oVK+zGG2+03r17x/vzJXsEflxkNukDF3Snm73ngq52ZKWTzmi33ToaOFs1\nj+6QGrmdknZM6rdpc8kdnM6Mf1y4yZaurGsLFpstWmK2cVM07xlfmOlTPNV1a73y38EF4+27mCko\nb++OB7UDzMsrPiTfc1VAB4dal+bMM5v9o/vvPt/PN5u7QJ9G7mpM5VeUlSlg1nE3Bf+fWxd37ma2\ni/vo//Zu/VR3UvYJBBqAv/nmGx98izNu2LDBbrnlFjvvvPPsN7/5jb3++us2ePBge/bZZ61+/frF\nB+V7Bgps3mI27TOzN993n6lm33xfciYUDHftYdb7V2a9djDbzgXGDm0qHwhXrlxnhYV145kvWR7d\nec7SDnSu2Xfuvy5tf+92pArgX34X/cRHcF90VqLpKyDrTERnytvof0ezJo2KD8n3bBDQgdtCF2Tn\nLYwGVwXYH9xH//XRQVz5KRp8dUCodbVNS7PWLcza/vy/hTuz1a0R3SKJ3SrRgd2GjdEDx/Xuv9vt\n+TNnTdsH9Z//63bLV7Ojn7ETolPXdHbtabb7jmZ7uI+Css6kSZkvEFgAVqAdMmSInX322fbII4/E\n5RSUN23a5IOvOh588MH26KOP2gcffGD77bdffDi+ZI6A7tO+9aHZxHfM3nCBd9WakmXv1tXs17ub\n7eOCrnYqCnw1nXTmrM9uLv/iSTtV3Vv+es7Pn9nR/3PdDlCXCmd+E/0UH0fftVPVjja2k9UOt2Uz\ns3ou5utT9+f/uj9dPG0pyrNVK+tbc1eWsCWd5W3YkO+DRW2WzRWjTFq/rsB0u6pR4zK9EnaIFEUv\n9brRTPO20R1orXKXg1e69S/2f+mKaNDVrY7VaxNmFe+h5a0DsNiBmC4Zt2q2zrbrXM9atSi1sONj\nVf2LzrQ/+tJ93FWbj93/L2ZF6z687bYnfZR00KogrO1HH31XN1LmCQS22IYPH+4D6o47ltwjLliw\nwNq1c2t1saTfixYtine599577Y033vC/t912W1u82B22ViNt2bLFbaCRMvmo++rVq939wXx/H7qB\nDmPLScuXL7emTZv64crpXa1O2unkucNlfcKWNm/ebCtXrvRGpcumI/e3p9W3N6bWs3c/rufOHn4p\nf8MGRbbnjptsn102Wu+dN7lA5vaQP6fV7hK0PomSDs7WrdPZrYuAFSQtu40b3alFCqlVU7cT3cmd\ndbtPLOnsZPaP+S44F/j/cxfmuzMT93H/12/Ms2UrzX90dlK5pJ20i9bFUtEWd+rlokWdgkpEl2Lj\n19xXXdd0p1ehTMGUKz8/Ym1bFFnHtlvc/dgt1qltse/ut85iSyeta0Wb67j9xy/reOlhqvq7sTuz\n3bdX9KM8dMD4+awC+/irui4w17VPvi5wl7/r2IczzX/ue0r3liP2q+6bbM+dNrrtTFePlody/1GZ\nbbSqflUZT/tclW327NnuQLpujV95Vd76lJcCCcDvvfeeffLJJ6Yg/O2335Yox5IlS6xJkyYlujVu\n3NhWrfplr3z44Yfbrrvu6oeZOHGiD34lRqjkj/Xr15uCSenp6v7z119/7c5WmluLFi2sTRt3famc\n9P7779tee+1V7XKUk7U7I9kQPwAor39tdpNPw4YNrV49t5dwSZdyJ0+rY+Mn1XGXl+v4QBUrX8tm\nETt47yLrs/cW27tXxJ8hunNF1zs6bmy4ZP9/+uknv2EkuxqyZs0a03pT1eRisrVu5S7xuZ3fL0kb\nzRZfwWbeojx3yTDPlriKNouX57kKN3m2wq2i2kHqYEMW+rjVqkTS2Z02voJitb5WL//eirZssEId\nBdRiUtkibudTJ4Q3GLVTVEq5bC4W1tHHHVPo2FXfVZezaeOI+0T/F7r/zZpGfC35dq0irqZ8xF/F\n+GX2dUCiT8W7Re0/tKPWgXoQaX93Fr7/HppSxF0V2GRfzclzlRbruDoUefbhZ3XckwF5/qBXB77u\nZok7kChyV5gitv/uW2wvt+3pCk0YUnW30XTNgw7y165da7FKwK1bu0coajBp+0+0Hle8ptVAIXRG\nqXu8uvT8xRdf2Pfff+93SJ9++qltv/32/sxGC6Z40u/iCDvssIPpo6Qz4ereG1bw1QZeOh/9VnDR\nR99L94+VMVn/2HBV+a9yqRa4NvCwJa1EKtdXc+rb/71s9r9J0ceEYuVs6wLYEb82O8zdOdi1R55b\n6bSDqt5OKrZMEi2L2LR14JJsmNiwlf2/VQdXQct9qpK0ri1evMTat28fH/3rr+u4A606ttNO2tnX\nXtK6ttZduih9IFp7JfplyqvcfQuVr1kzd50/ZEnLVOtaUAG49Ozv0t1V0HKfc052B3zuGPFjd7n6\nnRnRyo2ffBVxV23q2NOujuvT4/N9pcUD9nTb5L5m+l+bdRnSuY2WNqrMb11t1FW2dO3XFYATXdFM\newDWWVPLli19pSqh6OhRn2HDhvl7wrrcrMvQxdP8+fOtQ4cq7vGKZ8T3GhPQ2d7YiQ1s/Fv1fCWR\nWMZ61vbw/c2OOsjcJbDoGUisH/8RQCC9AqpzoHoO+lzc11Xe+naxzZzVwiZPL7C3PogeIL/0lpk+\nuk+8327Rg+RDXEDmcaf0LptUck97AO7UqZM99thj8bJ8+eWXdumll8a7KQDrSPK///2v/f73v7cJ\nEyb4M+SePV21P1KtC3zyldmI/5hNeFuXWl20dSnfnbgd6I6mTzjMVaZyl8bCcomr1rEoAAK1LNDc\nXWI/6qCIHXeobn2YTf3E7OV3zF6ZEn08T5Ui9Wlwn7nbQ2a/P9hd3t6dSly1tdjSHoCTzZiC71//\n+lcbNGiQrx2tyjbXXHNN2i4nJisP/aOXtV52AVeBt/gztKqocsKhRXbSEXV9bWCsEEAgvAK6Ra3H\n+/T52wXR2tU6kH7xzWgwHu9uIemjBmsUiE/6rXs2vkt45ycbSxZ4AO7WrVuJRjiEqgpWOgNetmyZ\nv1ydjdCZME96LOPpF82e/K97VGNJtMSq0NKnt9kffu+eke20zFU8a+IeWwnf/elM8KWMCNSWgLbj\nX7n7xvpc8SfX8pw7M37+dXd2PDnaYM1It83ro8Y/FIiPPDDa6E1tlTdXpht4AE4Eq5vUuldMCl5A\nj9jobFeBN/bMrmqOnni4WV8XePXso1I1n/6KZsJfBBCoVQHV+u69S/Rz/UVmr71nNuZ/0Ypc/hlk\n9/zxLQ+ZHXuIO/A+OtooTa0WOIsnHpoAnMXGoZ21RUvN/vWs2Sh31qs2a5U6tjXrf4L5e0hq9pGE\nAALZK6D6G791lSj1UWM0/zfBfSZGL1E/9byZPqrv0e9Y92xy9EnQ7MWohTkjANcCem1PUjWaHx4T\nveSkBiiUunaKPtZwdJ/o24aiXfmLAAK5IqArXX85M1qb+nV3VvzEOFeJ69NoE7JqRlZtVJ99onvi\nwd0vLt3iW64Y1fR8EoBrWjTE+ampRd3nUfBduTpaUFW6uODU6KNEvzRIEOKZoGgIIJBWAVXe0mNK\n+nz+bfT21AtvRNtxv3qY2b1PRg/Wj3c1rXkConqLggBcPb+MGNu1Z+AbztCGo8vOSnr92SV/iD6/\nqwoaJAQQQKC0QI/t3P3gy8wudxW3VEdEB/DzXCvBN7jHmP75b3e7yp0Rn3JEetpzL12WbPxNAM7G\npVpsnqZ/bjbkgV9eMKCXCFx4WrSmIw24F4PiKwIIJBTQi010efpPrn6I7gs/7i5P62BelbUeHRu9\nbH28axeAS9MJCcvtQQAulyXzO+oF4bc/Zvbf16LzojcOnXuS2VnH83hB5i9d5gCB2hHQ0xHnu1tW\nZ7pKWc+45i4fccFXgXjQvWaPPWs2wAVptYzHVbXUlo+rkE7KJgG1fvP4c65W4zm/BF/VcPyfO1K9\n8HSCbzYta+YFgdoS0BMSf3QH8xPdQb5uZalZy9nzXAC+xV1dG2D2wae1VbLMmi5nwJm1vCos7Vez\nza79h5maj1Tafhv3+/zo837RLvxFAAEEak5AgVi3tE77ndmDz0QvT3/6tdkZV7rGPA4wu/Jsc2+d\nqrnpZVtOnAFnwRLVK/FUwer4P0eDry43X+VW/OdcRQk9cE9CAAEE0inQotDsanfV7eVHoxU7NS01\nc6krcf982iz2uGM6y5CJeROAM3GpFSuzznYVeO93NRL1arK9XcB9/p/Ry0NUiCgGxVcEEEi7QIc2\nZne4s9+nbjfrsa17+5179PGeke5s+Dz37nD3hiZSSQECcEmPjPmlR4seGGV26mXR5/P0ns/Bl7iH\n5909mK07ZMxsUFAEEMhCgd13dI8+3mN2ozs5aO7OjuctNLv0tkZ22a2uSdtlWTjDVZwlAnAV4Wpz\ntB/dc3h/uMrs7hHulWMuEB+wh3vDyYPRR4tqs1xMGwEEEIgJqGEfPSM84eHYviniL0vrbHjsBLNI\nJDZk7v4nAGfYsterxI6+0OzDmdFWaK5zlaweusmsXasMmxGKiwACOSHQrGn06tzw69da162irfBd\nd7drX/pqs+/n5wRBwpkkACekCVcP3Uu55i6zgbeZ6bWBO3SJXuI54+hwlZPSIIAAAuUJ7NZji/3n\n/mitaTUCpFciHuNOJvQK1Fw9GyYAl7emhKzbnB/dpRx3r/fZidGCnXmMu4TjjiD1mBEJAQQQyBQB\ntR2t54b1hMZO20ffwnajC8r9rzOb/1OmzEXNlZMAXHOWacnplXfMTnCVq778zkyt0Pzzencm7O6h\n0Ah6WrjJFAEEAhDQm5VGDYsGYz2t8c50s99f4ALzKwFMPESTIACHaGEUL4patFJTkhcPiV5yVpX+\nZ12twj57Fx+K7wgggEBmCijwqhGPMa7xIN1S0621v7qgPODmX97WlplzlnqpCcCpWwU2pN7X+yd3\nSUaNnCudcGj0aJHHi6Ie/EUAgewR0BuXdEvtnJOibUi/9LbZsRebTfsse+Yx0ZwQgBPJ1FL37+bm\n2WmXF9h7H0UvMw9x7aoOvdSsfr1aKhCTRQABBNIsoFtqA/9o9i939tvWPdGhRy3VnKVeeag2D7I1\nEYBDtGTf+sC9ZeSv9e2HBXmm1waOdDWeTzwsRAWkKAgggEAaBdR0rmpK61abAu89T0YfV4q9xzyN\nk66VrAnAtcJedqJPjDM77wbdB8lzTbhF/CWZXbqXHY4uCCCAQDYLqF1pVTZVGwc6M576qdlx7pL0\nu+6qYLYlAnAtL1FVthrkKlfpxdY64jt0ny32xK2beYNILS8XJo8AArUroDYOVEFrm45mS5a7ejHX\nRt+4lE3PDBOAa3EdW7ve1QK8yWz0S9FCXHS62W0DN1rD+rVYKCaNAAIIhESgW9dog0OH7hs9Qbnr\nCbfPvNFMFVWzIRGAa2kp6ojuTNee85tTzVQd/7aBZn8+I1oLsJaKxGQRQACB0AnoRTP3uqdCruxv\nlu8i1uvvuzfAXRJtGyF0ha1kgQjAlQSricFnz4u2bKUXVzduGG3L+Zjf1ETO5IEAAghkp8CfTjAb\n4Sqmtm0ZfbuS3gT30luZPa8E4ICX30dfuFcIurPduQuiK5Lem7nvrgEXgskhgAACGSgQe83hrj2i\nzVgOuMXsH+6ydKY+qkQADnAlnDLD7Ky/mi1faaam2J5xL1fovm2ABWBSCCCAQIYLtHFnwE/c6h7R\nPDw6I8OfidalUUtamZYIwAEtMbXpfO6g6FGbjt7+fYdZhzYBTZzJIIAAAlkkoMeThvzFPUHi3qak\n+8JvuPvCJw3IvNcbEoADWCnHuQbGL3EtvGzabLafu9z82FCzwiYBTJhJIIAAAlkscPpRrvUsdxla\nzw5/N9fs5Euj70rPlFkmAKd5SY38r9nVw6L3KA7bz+yBG8waNkjzRMkeAQQQyBGBvXq554Xvjt7W\n0+093eb772uZMfME4DQup+GjXDvOw6MTOP5Qs7uu5jWCaeQmawQQyFGBrdqZPX1n9AqjrjRe6W7x\n3T3SLOyNdhCA07TCqhHxf4yIZn7mMS4Qu/sT+flpmhjZIoAAAjkuoPelP3iTe8rkyCjEA0+7Fzy4\nx5Y2bgovDAE4DcvmXteAuBoRVzr7RLNrzqOBjagGfxFAAIH0CahRoxsudrf9zonuc8dPMuvvmrBc\nuTp906xOzgTg6uiVM66eSbvfnf0qnX+K2eV/in7nLwIIIIBAMAJnHWd2n2s9S69x1cscTr/cbMHi\nYKZdmakQgCujlWTYO/9lpmfSlC48zWxAv+h3/iKAAAIIBCvwm32izws3b2r2zfeu9UFXQ/rL74It\nQ7KpEYCTCaXYX8H34THRgdWm8yV/SHFEBkMAAQQQSIvAr7pHK2d1cpW0Fi4x63uF2Xsfp2VSVcqU\nAFwltpIj6ZJzLPj+5UwzvdWIhAACCCBQ+wJdt3KtDrpHQXtup/etu3o57tL0y5Nrv1wqAQG4msvh\nkbHuTR0/V7jSZecLTq1mhoyOAAIIIFCjAq1bmI38+y+PKakN6bETanQSVcqMAFwltuhIT7pGNu54\nLPq9v3tTB5edq4HJqAgggEAaBfTmueHuXcK//XW0YaTrXOMdOoGqzUQArqL+cxNdW6Q/N7LR9/dm\nV/SvYkaMhgACCCAQiEDdArNh7j3spxwRnZxOoP4xovYaaHDFIVVWQG81GukCcB233PRGjuvOr2wO\nDI8AAgggUBsCddxp541/Nmvu2o9+0D218tiz+bZoSRM7+eDgS5NxAXjTpk22cqVr8LMaSXls2bKl\nTD7Kd82aNVZQUGB169Yt01+TnDy9wB7/bxvLb1hgv91/o13eb72tWlWNwpQadfPmzZaXl+dazaq9\no7JSRYr/lNnatWtt48aN8W7p/rLK4WqZJFvmKlOyYdJd1vLyL3IvKo249vCKl02/tYyLdytv3HR3\nUzm0LaiMYUtanqXdwlJGmelTR3vykCUty9WrV4dy/xG2bbS/e1a4Qd16dveT9e2FNxva3Dmb7doL\nVlmzZtWLL6VXCe03E21jGReAFRgLC92hSzWSduhaGUrnow2+cePG1qRJE2vatGmZ/h/OdC2suHf4\n1m2yi/XZ2+xOdykjP9896V2Dad26dfEDgBrMtkayklmjRo2sQYPg3iaxYcMGv0xKL6vSM6RglmyY\n0uME8VsHVOvXry9RtrCUUzsFHVBpfQ9b0oGXyhcWq+I+2n9oGwjjQbLWNS1P7SfDlsK4jV7gnlhp\n2Xyz3XB/vr37UYHd8URTe/iWQtPrDmsqKQAnOlgL3yFcTc11DeejB7jPv8Fsgzv523Mn92KFv9K2\ncw0Tkx0CCCAQuMBxhxTZ385bZXXyzN7+0D1GOji6nw+iIATgFJS/n2/W3z07tmqNWY9to68UVBNn\nJAQQQACBzBc4dJ8NNvCP7uqmuyb81gdm5w4yW7s+/fNFAE5ivGip2R+vMVu8zKxLJ1dtfYhZk0ZJ\nRqI3AggggEBGCey9i3uy5S/RV8aqtawgnhPOuHvAQS5RtZqiI6F5C83atjJ71AXfVs2DLAHTQgAB\nBBAISkBB+KGbzF6dYvaHo9M/VQJwAmO91PmSoWZfzDLTeyYVfNWeKAkBBBBAIHsFersgrE8QiQCc\nQFkPaL/jnvfVPYF/urPg7bdJMCCdEUAAAQQQqIIA94DLQVMTk6+8E+1xu3t7xp69yhmITggggAAC\nCFRDgABcCu+1d82eda1cKf313Gi7odFf/EUAAQQQQKDmBAjAxSzfdNXPR7qzXyU1Mdnv2Oh3/iKA\nAAIIIFDTAgTgn0U/+9a16XyXe0tGxGzfXV2jG7xWsKbXNfJDAAEEECgmQAB2GAsXu4B7vdm6Da6y\nVWezv/zBXFu9xZT4igACCCCAQA0L5Hwt6DXrzM67wUwNbugxo0tOdjWfa7Ad0BpeXmSHAAIIIJAl\nAjl9BqyXwAy87Zdnfe927TsXNsmSJctsIIAAAgiEWiCnA/AtD5m98b5ZgXvz373XuqYmtwr1sqJw\nCCCAAAJZJJCzAdi9Jc70kgWlG9zLmXv/KvqdvwgggAACCAQhkLP3gAt+buFqknv06OC9g6BmGggg\ngAACCPwikLNnwCLId5eeCb6/rAx8QwABBBAITiCnA3BwzEwJAQQQQACBkgIE4JIe/EIAAQQQQCAQ\nAQJwIMxMBAEEEEAAgZICBOCSHvxCAAEEEEAgEAECcCDMTAQBBBBAAIGSAgTgkh78QgABBBBAIBAB\nAnAgzEwEAQQQQACBkgIE4JIe/EIAAQQQQCAQAQJwIMxMBAEEEEAAgZICBOCSHvxCAAEEEEAgEAEC\ncCDMTAQBBBBAAIGSAgTgkh78QgABBBBAIBABAnAgzEwEAQQQQACBkgIE4JIe/EIAAQQQQCAQAQJw\nIMxMBAEEEEAAgZICBOCSHvxCAAEEEEAgEAECcCDMTAQBBBBAAIGSAgTgkh78QgABBBBAIBABAnAg\nzEwEAQQQQACBkgIE4JIe/EIAAQQQQCAQAQJwIMxMBAEEEEAAgZICBOCSHvxCAAEEEEAgEAECcCDM\nTAQBBBBAAIGSAgTgkh78QgABBBBAIBABAnAgzEwEAQQQQACBkgIE4JIe/EIAAQQQQCAQgUAD8Lp1\n62zx4sUJZ2zRokUJ+9EDAQQQQACBbBIoCGpmbr31Vps6darVqVPH2rRpYzfffLM1b97cT37KlCn2\nz3/+0/crKiqyc845xw444ICgisZ0EEAAAQQQCFwgkDPgN99802bOnGmjRo2y0aNH+8A7YsSI+Mw+\n8MAD1r9/f3viiSfs4osvtvvuu88UiEkIIIAAAghkq0AgAbhr1652/fXXW926dS0vL89at25ty5Yt\n86YbNmywuXPnWseOHf3vLl262Pz5823lypXZas58IYAAAgggYIFcgu7cubOnnjNnjo0fP94mT57s\nL0GrY/369e3ss8/2vw888EDfr2/fvvHL0xpm8ODBNmHCBH213r1728KFC/33qv6JRCKmT+l8FPSX\nLl1qW7Zs8Z+GDRtWdRJVHk/lUtKBStiSrkosX7480LKpzoCWSellVdpGbqpjELaUaF0LSzlVvjVr\n1oSlOPFyxK6ArV+/Pt4tLF9ktnr16rAUp0Q55LZkyZJAt9ESBajgR9i3UbnpFmm9evUqmIvK94rF\nk/LGDCQAxyY8a9Ysf3abn5/vd6jdunXzgXDTpk1+J7BgwQK/g9eZsgqt4ZTOPfdcO+200/z3J598\n0lq2bOm/V/WPdtSaZmFhYYksND1100f3p6s7nRKZp/hDVwRUjoKCQBdNSqVT8G3UqFGNr6AVTVzr\ngZZHsmWhINK4ceOKsqqVfiq/3JKVvzYKF9shapmGLWl5qnxNmjQJW9H8gZ5OHLSzDlvSwaq2lzDu\nP8K6jSoW6ICqWbNmadnvax+QaF0JdC9/8MEHmz6TJk3yZ7z77ruvffHFFzZy5EgbN26c39gUgI4/\n/njr2bOn7bPPPn79bteunemjpBVLAbo6aePGjT7Al85Hv2P563vp/tWZZqrjbt68OV6GVMcJajid\nlevgIEiX4sukovnUCh5kuSoqS/F+sSsZYSybzpaCXp7FbSr6rnKpfGF00/5D+wmVMWxJ61tsHxa2\nsoV1G9WBXnG3ml7nNN+x/UDpZRLIIdzrr79u06ZNi09bwVVHQzrj/fLLL61Hjx7xI10dWe600072\n+eefx4fnCwIIIIAAAtkmEEgA1in4XXfdZWvXrjWd4ak2tCpd6aNgqyD8zTffeFvdJ/7ggw9s5513\nzjZr5gcBBBBAAIG4QCCXoPv06WOffvqp9evXz3SJWfdXhw4d6q+L6z7wRRddZJdcconvrso2f/rT\nn2yPPfaIF5IvCCCAAAIIZJtAIAFY18AHDBjgn/FVTePSFVKOOeYYO+qoo3wrWa1atfL3MLINmvlB\nAAEEEECguEAgATg2QVUOKB18Y/1UoSFW0SrWjf8IIIAAAghkq0Ag94CzFY/5QgABBBBAoKoCBOCq\nyjEeAggggAAC1RAgAFcDj1ERQAABBBCoqgABuKpyjIcAAggggEA1BAjA1cBjVAQQQAABBKoqQACu\nqhzjIYAAAgggUA0BAnA18BgVAQQQQACBqgoQgKsqx3gIIIAAAghUQ4AAXA08RkUAAQQQQKCqAgTg\nqsoxHgIIIIAAAtUQIABXA49REUAAAQQQqKoAAbiqcoyHAAIIIIBANQQIwNXAY1QEEEAAAQSqKkAA\nrqoc4yGAAAIIIFANAQJwNfAYFQEEEEAAgaoKVCoAL1q0yD766CM/rQ0bNlR1moyHAAIIIIBAzguk\nFIAnTpxoPXr0sHbt2tkxxxzj0fbaay8bM2ZMzgMCgAACCCCAQFUEkgbgGTNm2FFHHWU77rijDRw4\nMD6N3r1728knn2yvvPJKvBtfEEAAAQQQQCA1gaQB+O6777aTTjrJxo4da4ceemg81wcffNDOOecc\ne/755+Pd+IIAAggggAACqQkkDcBfffWV7bfffuXmtueee9rkyZPL7UdHBBBAAAEEEEgskDQAb7vt\ntvbcc8+Vm8OoUaNsm222KbcfHRFAAAEEEEAgsUBB4l7RPv3797dDDjnE+vXr5ytiFRUV+fu+Tzzx\nhL322mtcgk4GSH8EEEAAAQTKEUgagA866CDT/d4rrrjCRowY4bPQveBGjRrZ7bff7itolZMvnRBA\nAAEEEECgAoGkAVjj6iz4xBNPtGnTptmcOXOsQ4cOtuuuu1rbtm0ryJpeCCCAAAIIIJBIIGkAfvfd\nd/24euzo4IMPTpQP3RFAAAEEEECgEgJJA/Ctt95qagHrnXfeqUS2DIoAAggggAACFQkkrQXdunVr\nW716tW3ZsqWifOiHAAIIIIAAApUQSHoGfPbZZ9tbb71lu+++u6lC1tZbb215eXnxSeywww5UxIpr\n8AUBBBBAAIHUBJIG4Mcee8zmzp3rc/v666/L5Hr88ccTgMuo0AEBBBBAAIGKBZIG4Iceesj0ISGA\nAAIIIIBAzQkkDcCxSW3atMl0Brxw4ULfIEf79u1jvfiPAAIIIIAAApUUSFoJS/mNGzfOunTp4t+I\n1KdPH/8ccMeOHe2ll16q5OQYHAEEEEAAAQQkkDQAf/jhh/5tSN26dTO9GWn69Omm9wOrecpjjz2W\nx5NYjxBAAAEEEKiCQNJL0Pfdd5+ppvOrr75aovazAvDnn39u//73v23fffetwqQZBQEEEEAAgdwV\nSHoGrNcR6kUMxR89inGdddZZ9t5778V+8h8BBBBAAAEEUhRIGoB1r/f5558vNzt179SpU7n96IgA\nAggggAACiQWSBmCd5b799tt200032bx583yLWPPnz7fbbrvNJkyYYH379k2cO30QQAABBBBAoFyB\npPeAf/e739kdd9xh1157rV1//fX+UnQkErF69erZoEGDfAWtcnOmIwIIIIAAAggkFEgagDXmwIED\n7dRTTzXViJ49e7a/7Lznnnta586dE2ZMDwQQQAABBBBILJD0ErRGXbFihY0dO9aOPvpou+SSS/wz\nwY8//ritWrUqcc70QQABBBBAAIGEAkkDsFq/2n777e0f//hHPJOioiIbPny4bbvttjZz5sx4d74g\ngAACCCCAQGoCSS9BjxkzxreA9eKLL8Zz1OVnXYo+4ogjbOjQof5Z4HjPNH9Rk5grV66s1lSUh16v\nWDof/V6zZo0VFBRY3bp1y/Sv1kRTHHnz5s3+Pnt+fn6KYwQ3mMzWrl1rGzduDGyiusqiZVJ6WZUu\ngMqUbJjS4wTxWwerqjMRxrKpXNoWVMawJS3PsLrJTJ86dZKevwTOqmWp18eGcf8R1m1U+zV95Nao\nUaMa31aVd6JtLGkAVgMcavFKBSueVAnr5JNP9mfCxbun+7sCY2FhYbUmox26VobS+WiDb9y4sTVp\n0sSaNm1apn+1JpriyOvWrYsfAKQ4SmCDyUzrQYMGDQKb5oYNG/wyKb2sShdAAS7ZMKXHCeK3DqjW\nr18fyrJpp6ADKq3vYUs68FL5wrhMtf/QNhDGIKd1TctT+8mwpbBuo9qvaTtN135fATjRwVrSQzg1\nQan7v+WlF154gYpY5cHQDQEEEEAAgSQCSQOw3vc7ZcoUu/LKK23GjBmmZ4A//vhju/HGG02XpfWc\nMAkBBBBAAAEEKieQ9BK02nzWGfAZZ5xht99+ezx3na7r93HHHRfvxhcEEEAAAQQQSE0gaQBWNroH\nvHz5cv/yhS+//NLatWtnO+20k7Vo0SK1qTAUAggggAACCJQQSHoJOja0Khz06tXLTjzxRGvevLl/\nCYNu+JMQQAABBBBAoPICSQOwaoidcsopNmDAAJ+7nv/deeed/SNIO+64oy1durTyU2UMBBBAAAEE\nclwgaQC+66677JVXXrH99tvPPyul9p+PPPJI++CDD/yZ8JAhQ3KckNlHAAEEEECg8gJJA7DehHTh\nhRf6Z37fffdd++mnn+zSSy+13Xff3b8neOrUqZWfKmMggAACCCCQ4wJJA7AuMbdp08YzvfTSS74h\nhl//+tf+txpJCOOD8jm+TJl9BBBAAIEMEEgagHfZZRcbPXq0TZs2zUaMGGGHH3641a9f39eI1gsZ\nDjvssAyYTYqIAAIIIIBAuASSBmA1wPHDDz/4S85qHu7mm2/2c7DXXnv5JgJPOOGEcM0RpUEAAQQQ\nQCADBJI+B9ylSxfTs7+ffPKJ9ezZ0wddzdfzzz9vBx54oH9xQAbMJ0VEAAEEEEAgVAJJA7BKq4bH\n9Qak4umggw4q/pPvCCCAAAIIIFAJgaSXoCuRF4MigAACCCCAQIoCBOAUoRgMAQQQQACBmhQgANek\nJnkhgAACCCCQokDSe8C33Xabvwd8+umnx58HTjFvBkMAAQQQQACBBAJJz4CbNm3q3/3bqVMn/1ak\ncePG2aZNmxJkR2cEEEAAAQQQSEUgaQBWM5Tz58+3p59+2iKRiG+SsmPHjv7lDNOnT09lGgyDAAII\nIIAAAqUEkgZgDa+Wr9Tgxn/+8x/78ccfbfDgwb4lrN12283UUtY999xjK1asKJU1PxFAAAEEEEAg\nkUBKATg2clFRkW+QQ2e+H330kQ/MnTt3Nt0n3m677eyNN96IDcp/BBBAAAEEEKhAIKUArFawrrrq\nKttmm22sT58+9uGHH9rf/vY3fzasFrHmzJnj24Tu379/BZOiFwIIIIAAAgjEBJLWgr7ssstM7wTW\nG5HOOOMM++Mf/2i9evWKje//FxQU+ACsYKz7xHl5eSX68wMBBBBAAAEESgokDcBbb721PfXUU3bS\nSSdZ3bp1S4ytylmLFi3y94GPO+44O/HEEwm+JYT4gQACCCCAQPkCSQPwjBkzbMcddywTfJWd3oz0\n+eef2yuvvGLNmjUrfwp0RQABBBBAAIEyAuUG4ClTpviazhr6448/tvfee69MBSs9C6zhTj755DKZ\n0gEBBBBAAAEEKhYoNwDvvffettVWW/nLy3Xq1LH8/HyrV69eiZwaNWpkZ555pg0aNKhEd34ggAAC\nCCCAQHKBcgOwgu5DDz3kx77zzjtNjxrpHjAJAQQQQAABBGpGoNwAXDzrgQMHFv/JdwQQQAABBBCo\nAYFyA/CIESN8e896rvfFF1/0jW8kmlb37t19G9GJ+tMdAQQQQAABBMoKJAzAa9asMQXgF154wUaN\nGlV2zJ+7HHvssQTghDr0QAABBBBAoHyBcgOwHiuKpQceeMD0ISGAAAIIIIBAzQmk1BRlzU2OnBBA\nAAEEEEBAAuWeAY8ePdoWLFiQktD2229vRxxxRErDMhACCCCAAAIIRAXKDcB6veDUqVNTMtJrCgnA\nKVExEAIIIIAAAnGBcgPw22+/HR8g2Zdly5YlG4T+CCCAAAIIIFBKoNwAXHwYvQP4jjvu8O09q2a0\n3nakz+bNm23p0qW2++67my5ZkxBAAAEEEEAgdYGklbAefPBB/y5gvfVIAXf58uVWWFho3333nX9W\neMCAAalPjSERQAABBBBAwAskDcATJ060Y445xvRWpMsvv9y22247mzBhgn366afWpEkT27hxI5QI\nIIAAAgggUEmBpAFY7/w95JBDfLa9evWyd955x1+Cbt++vZ1//vk8I1xJcAZHAAEEEEBAAkkDcNeu\nXW327NleS81O6jL0F1984X8rCM+aNct/5w8CCCCAAAIIpC6QNAAfdthh9sgjj9j999/v7/326NHD\nbr/9dt8+9PDhw22//fZLfWoMiQACCCCAAAJeIGkAPuuss+y0006zRx991I9wyy232MiRI23nnXe2\n6dOnW9++faFEAAEEEEAAgUoKJH0MSfmpLei1a9f6rFUhSxWwPv74YzvyyCOtcePGKU9y3bp1pkeZ\nWrduXWYcPdqk+81t27a1goKUilUmDzoggAACCCCQKQIpR7pGjRrF56lbt26mT2XSrbfe6lvXqlOn\njrVp08Zuvvlma968uc/iyy+/tOuuu840DT3mdMopp9jpp59emewZFgEEEEAAgYwSSBqAFThfeuml\nhDN10EEH2Q033JCwv3q8+eabNnPmTP9aQ53dXnvttaZ3Dl9yySW+RvXgwYPtjDPO8I87LVy40HTZ\n+9BDD/WBusKM6YkAAggggECGCiQNwM2aNbMOHTrEZ0+XivWihk8++cRfLu7Zs2e8X6Ivqkl9/fXX\nW926df0gugQda8JSzxdv2LDBB1/1bNeunT3xxBO+wlcsv1jrW8V/x75X57/yLZ5i0yn+v3j/IL7X\n5rRTnb/SbqmOV5XhUvWIDVeVaaRznJhV7H86p1XZvGNmYS2b5iesZYvZVdY8iOHDWrYwlyu2rqWj\njMozUUoagC+44ALTp3T66aef7OCDD7b8/PzSvcr87ty5s+82Z84cGz9+vE2ePNlfglbHefPmmfo/\n9thj9vLLL/vL0Oeee6717t07ns9VV11l//nPf/zv3/3udym/qSmeQYIvpd/4tH79en9PW5fJdVBQ\nun+CbHKqc+zAKaiZVt2D+vXrp7QsYvUUgipbZaYT5nVp9erVlZmVQIcN6zJdtWpVoA6VmdiSJUsq\nM3igw4Z1eQpBbnl5efETxZqC2bJli+lTXkoagMsbSd10H7dfv372+OOPm96IlErSM8OqaKWgrUvN\nuo+smdZZcMeOHU2Xu6dMmWI33nijjRs3zu94la8uUQ8aNMhPYtiwYf4sOZXpJRpGK8GmTZtMZ/el\n0zbbbFO6U6C/dRAgn9jVgkAnnmRiegZcle4UEINMuoKSLGmH2LRp02SDBd4/1ma6KheGLamdd1WM\nrExFyqDmQQcFOnMI4zLV/kPbQConH0F5xaajEyPVrQnj/iOs26hac1y5cqW1bNnSxzVdha3JpOCb\naF2pcgBWAXVUX5mjLZ0x6zNp0iR/Brzvvvv6M16tLBdffLFfqbt06eKbutRZcp8+fbyDVvbYTl9n\np/pUJ+koR6m6+VSnDInGVdlqYh4T5V/d7rHyVTefmh4/rOWKrWOx/zU939XNL6xuKpcCcBjdYmaU\nrXJrX8ytcmOlf2gtx1jZ9L2ml2u1LkGrLejPP/+8hIIuz3799df+svE111xTol95P15//XV/trnb\nbrv53rpvrMeRFMB1tFH6rEptTKs2NAkBBBBAAIFsFUh6Bjxq1Ch78sknS8y/jhB0maN///6+RnOJ\nnuX80Cn4XXfdZXqzUr169XxtaF1y1keXstVfQVpnx3ok6bPPPrMrrriinJzohAACCCCAQHYIJA3A\nagEr1gpWVWdZl5LVeIfuGevsWcF76NCh/lRfl5Zvuukm/1HTljozvvLKK62278VWdV4ZDwEEEEAA\ngVQEkgbgVDJJNozOmPXeYN3njd3sLj6OmrUcO3asv5/cokWLGr8GX3xafEcAAQQQQCAMAkkDcLKG\nOIrPhB4nUgMbiZIa4VBNs0SpVatWiXrRHQEEEEAAgawSSFqdePvtt7fZ7nWE06ZN8w1yqCKVHl9Q\nLeV3333X9EpCNdShT3ltPGeVFjODAAIIIIBADQkkPQPWPdnCwkL78MMPrfgZ6ooVK2yXXXaxffbZ\nx/7yl7/UUHHIBgEEEEAAgdwQSHoGrFrQl156aYngKxo1YqHuY8aMyQ0p5hIBBBBAAIEaFEgagHX2\n+9prr5U7SbVqVV5rUuUOTEcEEEAAAQQQiAskvQR97LHH+jcV6d6vmpzUc7tqQOPf//633Xnnnf6Z\n3nhufEEAAQQQQACBlASSBuBTTz3VPx6kx4guu+yyeKZqo1UBONV2oOMj8gUBBBBAAAEELGkAltFF\nF11kffv2tenTp9s333xjW2+9te2xxx7UemYFQgABBBBAoIoCKQVg5a3Wq2IvU6jitBgNAQQQQAAB\nBH4WSFoJCykEEEAAAQQQqHkBAnDNm5IjAggggAACSQUIwEmJGAABBBBAAIGaFyAA17wpOSKAAAII\nIJBUgACclIgBEEAAAQQQqHmBlGpBP/DAA/bcc8+Z2n/esmVLiVIccsghpjcmkRBAAAEEEEAgdYGk\nAfjhhx+2Cy+80Pbcc0/r3r271atXr0TuXbp0KfGbHwgggAACCCCQXCBpANaZr1q7Gjt2bPLcGAIB\nBBBAAAEEUhJI6R5wr169UsqMgRBAAAEEEEAgNYGkAfjII4/0rxwsKipKLUeGQgABBBBAAIGkAkkv\nQffu3dtUCUv3gFXhqlWrViUy1X3ho48+ukQ3fiCAAAIIIIBAxQJJA/Ajjzxic+fO9bnoRQylk15X\nSAAurcJvBBBAAAEEKhZIGoCHDx9u+iRKy5YtS9SL7ggggAACCCCQQCBpANa93zvuuMNeeeUVW7Nm\njUUiEf/ZvHmzLV261HbffXcbPXp0guzpjAACCCCAAALlCSSthPXggw/aVVddZYsWLfIBd/ny5VZY\nWGjfffedbdq0yQYMGFBevnRDAAEEEEAAgQoEkgbgiRMn2jHHHGMzZsywyy+/3LbbbjubMGGCffrp\np9akSRPbuHFjBdnTCwEEEEAAAQTKE0gagOfPn+9rP2tkPQ/8zjvv+EvQ7du3t/PPP9/XkC4vY7oh\ngAACCCCAQGKBpAG4a9euNnv2bJ+DHjnSfd8vvvjC/1YQnjVrVuLc6YMAAggggAAC5QokDcCHHXaY\n6VGk+++/39/77dGjh91+++32ySef+NrR++23X7kZ0xEBBBBAAAEEEgskDcBnnXWWnXbaafboo4/6\nXG655RYbOXKk7bzzzjZ9+nTr27dv4tzpgwACCCCAAALlCiR9DEljqSWstWvX+gxUIUsVsD7++GNT\nM5WNGzcuN2M6IoAAAggggEBigZQCsEZv1KhRPJdu3bqZPiQEEEAAAQQQqJpAuQF4xIgR/hnf/v37\n24svvujv9ybKXhWz1BwlCQEEEEAAAQRSF0gYgNXqlQLwCy+8YKNGjUqYo4IvATghDz0QQAABBBAo\nV6DcAKxmJ2NJ93/1ISGAAAIIIIBAzQkkrQVdc5MiJwQQQAABBBCICZR7BnzRRRfZ559/Hhumwv8H\nHXSQDRo0qMJh6IkAAggggAACJQXKPQNu0aKFtWvXLv6ZNm2aTZ061Vq1amU9e/a0devW2aRJk/yj\nSJ07dy6ZI78QQAABBBBAIKlAuWfAQ4YMiY/41FNP2UsvvWTffvuttW3bNt5dTVLuuuuutnLlyng3\nviCAAAIIIIBAagLlngEXH1UBWJekiwdf9W/ZsqVdcsklNmbMmOKD8x0BBBBAAAEEUhBIGoCbNWtm\nn332WblZTZkyxfRCBhICCCCAAAIIVE4gaQBW05Pjxo2zYcOG2bx583wDHXPnzrW///3v9uyzz9rJ\nJ59cuSkyNAIIIIAAAghYufeAi7uceuqptmTJErv00ktt4MCB8V5169a1hx9+2E466aR4N74ggAAC\nCCCAQGoCSQOwstE9YAXiGTNm2DfffGM77LCD7bbbbqbL00GnTZs2Vbvil/LYsmVLtfNJx7xv3rzZ\n8vLyLD8/Px3ZVytPmemlHBs3bqxWPukYWWUKY4XAoqIii0QioSybyqVtQWUMW9LyDKubzPSpUyfp\nBcTAWbUsV69eHcr9R1i3Ue3X9JGb3nlQ0/sR5Z1oG0spAGst0iNIvXr1statW9suu+xiGzZsCHzl\n0gR15l1YWFitaauZTa0M1c2nWoVIMLIe8SooKPDzmWCQWussM62gDRo0qLUyJJqwNpowLk8dUK1f\nvz6UZdNOQQdUTZo0ScRaa91XrVrld1phXKbaf2gbCONBstY1LU/tJ8OWwrqNar+m7VRuTZs2rfFt\nVQE40cFaSodwEydOtB49evjngnVPWGmvvfaiBnTY1nDKgwACCCCQMQJJA7AuOx911FG24447lrgH\n3Lt3b18Bq3i70Rkz1xQUAQQQQACBWhZIGoDvvvtuX9Fq7Nixduihh8aL++CDD9o555xjzz//fLwb\nXxBAAAEEEEAgNYGkAfirr76y/fbbr9zc9txzT5s8eXK5/eiIAAIIIIAAAokFkgbgbbfd1p577rly\nc9B7grfZZpty+9ERAQQQQAABBBILJK0F3b9/fzvkkEOsX79+viKWak7qvu8TTzxhr732GpegE9vS\nBwEEEEAAgYQCSQOwXjeo+71XXHGFjRgxwmeke8F6HOX222/3FbQS5k4PBBBAAAEEEChXIGkA1lg6\nCz7xxBNNryWcM2eOdejQwb8JqfQLGsqdAh0RQAABBBBAoIxASgFYYy1atMj0ALoa4lArMO+//77P\nrGPHjr5VrDI50wEBBBBAAAEEEgokDcBffPGFnXDCCQnfiKS2oEePHp1wAvRAAAEEEEAAgbICSQPw\nY4895sd65513bOutty7TpFbDhg3L5koXBBBAAAEEEKhQIGkA1n3f008/3fbZZ58KM6InAggggAAC\nCKQukPQ5YD2CNH78+NRzZEgEEEAAAQQQSCqQ9AxYjx/p8rNeP3j00UfbVlttVSJTNdTRp0+fEt34\ngQACCCCAAAIVCyQNwFOmTLG3337bli1bZtOnTy+TmyphEYDLsNABAQQQQACBCgWSBuAHHnjA9KjR\nyy+/bDvvvHOZSliJ3nNY4VTpiQACCCCAQI4LJA3AP/30k5166qm2xx575DgVs48AAggggEDNCSSt\nhHXkkUf6tp/V+AYJAQQQQAABBGpGIOkZ8P7772+6DK1XDx5wwAH+cnTxy87du3f3lbNqpjjkggAC\nCCCAQG4IJA3AjzzyiC1YsMBrfPfdd2VUjj32WAJwGRU6IIAAAgggULFA0gA8fPhw04eEAAIIIIAA\nAjUnkPQecM1NipwQQAABBBBAICZAAI5J8B8BBBBAAIEABQjAAWIzKQQQQAABBGICBOCYBP8RQAAB\nBBAIUIAAHCA2k0IAAQQQQCAmQACOSfAfAQQQQACBAAUIwAFiMykEEEAAAQRiAgTgmAT/EUAAAQQQ\nCFCAABwgNpNCAAEEEEAgJkAAjknwHwEEEEAAgQAFCMABYjMpBBBAAAEEYgIE4JgE/xFAAAEEEAhQ\ngAAcIDaTQgABBBBAICZAAI5J8B8BBBBAAIEABQjAAWIzKQQQQAABBGICBOCYBP8RQAABBBAIUIAA\nHCA2k0IAAQQQQCAmQACOSfAfAQQQQACBAAUIwAFiMykEEEAAAQRiAgTgmAT/EUAAAQQQCFCAABwg\nNpNCAAEEEEAgJhBoAF63bp0tXrw4Nu1y/y9cuNDWr19fbj86IoAAAgggkC0CgQXgW2+91c444wy7\n4IIL7MILL7Tly5eXMZw9e7adfvrp9umnn5bpRwcEEEAAAQSySSCQAPzmm2/azJkzbdSoUTZ69Ghr\n3ry5jRgxooTj5s2bbciQISW68QMBBBBAAIFsFSgIYsa6du1q119/vdWtW9dPrnXr1rZs2bISk37k\nkUdst912s1WrVpXozg8EEEAAAQSyUSCQANy5c2dvN2fOHBs/frxNnjzZbr755rjnjBkz7L333rMH\nH3zQdLZcOilY6/6x0pYtW/yn9DCV+V1UVGSRSKTa+VRmmqkOq7JpHuvUCeTiRKrF8sPJLFa+So0Y\nwMBhLZeWpVLsfwAUKU9CZmF1U7nCvo2mDB3ggDGzMO4/wrquaduMuel7TW+rFeUXSACOrX+zZs2y\n+fPnW35+vqmyVbdu3Wz16tV222232eDBg61evXqxQUv8v/POO+3FF1/03fr06WM//fRTif6V/SFs\npeL5fP/999ayZUtr0qRJZbOr0eFjZcvLy6vRfGsiM5VtxYoVFtayhbHyXmx5Fl/XamJZ1FQeKt/a\ntWtrKrsayyfmFtZlumbNmhqb15rMSG46YWEbTV01tq41btzYVwDWCWKzZs18nNLV2uqmioJ6oAH4\n4IMPNn0mTZrkz4D33Xdfu/vuu61Hjx5+xlX5auPGjabKWDprbtu2rZ933RuO3R/W//bt21fLRBuP\nptOiRYt4Pjo4aNq0qbVr1y7erTa+6Ey/oKAgfrm+NsqQaJqqwa4DlAYNGiQapNa6r1y50goLC2tt\n+okmrLoNcqvuOpso/+p01xmJgm9tH3SWNw+6FaXyaUcYtqT9h7YBnUiELS1atMjv12K3+8JUvrBu\no4oFOrHYaaed/JVWxR+djNWvX79GtlsF4ETrSiAB+PXXX/cbku7xKvXs2dO0Ei9YsMDP8I8//mjD\nhg3z/VQ7euzYsT4AHXPMMb4bfxBAAAEEEMg2gUACsI4A7rrrLn+PV5eZVRu6Y8eO/hM7s43BnnLK\nKXb55ZfbHnvsEevEfwQQQAABBLJOIJAArPu2urzcr18w6SFqAAAYwElEQVQ/27Bhg38MaejQoaGs\naJR1S5gZQgABBBAIpUAgAVg18gYMGGAXX3yx6T6Arq8nSs8880yiXnRHAAEEEEAgawQCfdZFlYsq\nCr5Zo8qMIIAAAgggkEQg0ACcpCz0RgABBBBAIGcECMA5s6iZUQQQQACBMAkQgMO0NCgLAggggEDO\nCBCAc2ZRM6MIIIAAAmESIACHaWlQFgQQQACBnBEgAOfMomZGEUAAAQTCJEAADtPSoCwIIIAAAjkj\nQADOmUXNjCKAAAIIhEmAABympUFZEEAAAQRyRoAAnDOLmhlFAAEEEAiTAAE4TEuDsiCAAAII5IwA\nAThnFjUzigACCCAQJgECcJiWBmVBAAEEEMgZAQJwzixqZhQBBBBAIEwCBOAwLQ3KggACCCCQMwIE\n4JxZ1MwoAggggECYBAjAYVoalAUBBBBAIGcECMA5s6iZUQQQQACBMAkQgMO0NCgLAggggEDOCBCA\nc2ZRM6MIIIAAAmESIACHaWlQFgQQQACBnBEgAOfMomZGEUAAAQTCJEAADtPSoCwIIIAAAjkjQADO\nmUXNjCKAAAIIhEmAABympUFZEEAAAQRyRoAAnDOLmhlFAAEEEAiTAAE4TEuDsiCAAAII5IwAAThn\nFjUzigACCCAQJgECcJiWBmVBAAEEEMgZAQJwzixqZhQBBBBAIEwCBOAwLQ3KggACCCCQMwIE4JxZ\n1MwoAggggECYBAjAYVoalAUBBBBAIGcECMA5s6iZUQQQQACBMAkQgMO0NCgLAggggEDOCBCAc2ZR\nM6MIIIAAAmESIACHaWlQFgQQQACBnBEgAOfMomZGEUAAAQTCJEAADtPSoCwIIIAAAjkjQADOmUXN\njCKAAAIIhEmAABympUFZEEAAAQRyRoAAnDOLmhlFAAEEEAiTQEGYCpNKWTZt2mQrV65MZdCEwyiP\nLVu22Pvvv28tWrSw1atXW4cOHSwvL6/aeSecaIo9Nm/e7MuRn5+f4hjBDSaztWvX2saNG4ObaIpT\nUpmqu16kOKlKDVZUVGSRSCSUZVO5tC2ojGFLWp5hdZOZPnXqhO/8RctS+7Mw7j/Cuo1qv6aP9h/r\n1q2zNWvWeEMt45rYpyjvRNtYxgXgunXrWmFhYbX2FwLWyqAVVQFY6Lvuumu18qypkVWWgoIC03yG\nLcmsUaNG1qBBg7AVzW8o1V0v0jFTOqBav359tdfZdJRNOwUdUDVp0iQd2Vcrz1WrVvmdVhiXqfYf\n2gbCGOS0rml5hnH/oWAWxuWp/Zq2U5VNbo0bN/aG9evXr5HyKgAnOlgL3yFctTZbRkYAAQQQQCAz\nBAjAmbGcKCUCCCCAQJYJEICzbIEyOwgggAACmSFAAM6M5UQpEUAAAQSyTIAAnGULlNlBAAEEEMgM\nAQJwZiwnSokAAgggkGUCBOAsW6DMDgIIIIBAZggQgDNjOVFKBBBAAIEsEyAAZ9kCZXYQQAABBDJD\ngACcGcuJUiKAAAIIZJkAATjLFiizgwACCCCQGQIE4MxYTpQSAQQQQCDLBAjAWbZAmR0EEEAAgcwQ\nIABnxnKilAgggAACWSZAAM6yBcrsIIAAAghkhgABODOWE6VEAAEEEMgyAQJwli1QZgcBBBBAIDME\nCMCZsZwoJQIIIIBAlgkQgLNsgTI7CCCAAAKZIUAAzozlRCkRQAABBLJMgACcZQuU2UEAAQQQyAwB\nAnBmLCdKiQACCCCQZQIE4CxboMwOAggggEBmCBCAM2M5UUoEEEAAgSwTIABn2QJldhBAAAEEMkOA\nAJwZy4lSIoAAAghkmQABOMsWKLODAAIIIJAZAgTgzFhOlBIBBBBAIMsECMBZtkCZHQQQQACBzBAg\nAGfGcqKUCCCAAAJZJkAAzrIFyuwggAACCGSGAAE4M5YTpUQAAQQQyDIBAnCWLVBmBwEEEEAgMwQI\nwJmxnCglAggggECWCRCAs2yBMjsIIIAAApkhQADOjOVEKRFAAAEEskyAAJxlC5TZQQABBBDIDAEC\ncGYsJ0qJAAIIIJBlAgTgLFugzA4CCCCAQGYIEIAzYzlRSgQQQACBLBMgAGfZAmV2EEAAAQQyQ4AA\nnBnLiVIigAACCGSZQKABeN26dbZ48eKEhAsXLrRIJJKwPz0QQAABBBDIFoGCoGbk1ltvtalTp1qd\nOnWsTZs2dvPNN1vz5s395F966SUbMWKEFRQU2MaNG+2EE06wk046KaiiMR0EEEAAAQQCFwjkDPjN\nN9+0mTNn2qhRo2z06NE+8CrgKq1fv97uvfdeGzRokA/Cw4YNs4ceesiWLFkSOAYTRAABBBBAICiB\nQAJw165d7frrr7e6detaXl6etW7d2pYtW+bnUZek+/bta927d/e/O3bsaO3atbMPPvggKAOmgwAC\nCCCAQOACgVyC7ty5s5+xOXPm2Pjx423y5Mn+ErQ6brXVVnb66afHZ/yzzz6zefPm2U477RTvpvGW\nLl3qf2/atMlfpo73rMKXLVu2WFFRkcXyiv2vQlY1PsrmzZv9ffAw3gtXmVQ+3SYIW9IyDWu5ZBXG\nsmkbCLOb1rcwumkb0D5DdmFLMlPZwrj/COu6FvPSuqaPfuujk8WaWP8034mWRyABOLaSzpo1y+bP\nn2/5+fmmClfdunWL9fL/586da9ddd52dd9551qlTp3i/MWPG2Kuvvup/KzAvX7483q8qX7TjUdKB\nQYMGDaxDhw7VzrMq5ShvHC0oLfgwJq1Ia9assbVr14aueLEDqrAVTMtTn+qus+maL5Vtw4YN6cq+\nyvnGtlHtCMOWwmomJ7mtWrUqlPuQMG+jKpu2Ue3jtt12W2vYsKE3rIntVnnG1ufS63KeW5kCr3Y8\nadIkfwb8wgsv+IpXKpTuEV999dV25plnVlgBa8iQIT5Il56RyvxWENGRTYsWLSozWiDDqqa4KqPp\ncn3Ykm4XNGnSxB+0hK1sK1eutMLCwrAVy18xkFv79u1DVzbtFHQwpWUatqQgovI1a9YsbEXzB6E6\ncNeJRNjSokWL/H4tjPuPsG6jigUrVqzwlYPTsTwVgO+55x679NJLy2QfyD3g119/3aZNmxafeM+e\nPf1KvGDBAt/tww8/tCuuuMIuvvjiCoNvPAO+IIAAAgggkOECgQRgHQHcdddd/mhb909UG1qVrfTR\n2cE111xjF110ke29997+MoBO+8N4WSzDlzXFRwABBBAIkUAg94D79Oljn376qfXr188HVj3/O3To\nUP9M8HPPPecDs54TLp50NnzKKacU78R3BBBAAAEEskYgkACsxjcGDBjgLzHrPkDLli3jgOecc47p\nQ0IAAQQQQCCXBAK5BB0DVeWi4sE31p3/CCCAAAII5JpAoAE413CZXwQQQAABBBIJEIATydAdAQQQ\nQACBNAoQgNOIS9YIIIAAAggkEiAAJ5KhOwIIIIAAAmkUIACnEZesEUAAAQQQSCRAAE4kQ3cEEEAA\nAQTSKEAATiMuWSOAAAIIIJBIgACcSIbuCCCAAAIIpFGAAJxGXLJGAAEEEEAgkQABOJEM3RFAAAEE\nEEijAAE4jbhkjQACCCCAQCIBAnAiGbojgAACCCCQRoFA3oZUk+X//vvv7cknn6xWlnrXsN5R3KhR\no2rlk46RN27caPn5+f6Tjvyrk+fq1autfv36Vrdu3epkk5Zx169fbw0aNEhL3tXJVOvZmjVrrLCw\nsDrZpGXcoqIi27Rpk1+maZlANTLV8oxEItawYcNq5JKeUbX/0Dagt7yFLa1atcrv17QPCVsK6zaq\nd9SvW7fOmjZtmhYybWeJUp5bySOJeoaxu1awimYolTK/+OKLNnPmTLvyyitTGZxhfhYYOHCgnXba\nabbHHnuEzkSrcV5eXujK9eOPP9pll11mo0aNCl3ZwlwgHWSvXbvWzj333DAXM3Rl+9Of/mTXXnut\nbbfddqErW1i30c8++8zuu+8+++c//5k2Mx1I1qtXr0z+GXcGXBNHKVoRdMTTrFmzMiB0SCygdznr\nlZK4JTYq3Wf58uW2dOlSzErDJPmtKwc6O2ddSwJVqveyZcv8FQ3cSsFU8FNXM7Sd1oZZxgXgChxT\n7qXLgW3btk15eAaMCnTs2DGUl+3DvHy0cXfu3DnMRQxl2Vq0aBHKS+OhxCpWqK233hq3Yh6pfNWt\nq06dOqUyaI0Pk3GXoGtcgAwRQAABBBCoBYHw1SKoBQQmiQACCCCAQNACORmAdZ9ENXpJiQVU07P4\nRzUFiyfV1l64cGHxTjn9fcWKFab1qnSqyKmifqXzydbfc+fOtdLrVvH1Tt9L98/l7VcWS5YsSbg6\nqNJfaa/YwBX1iw2Trf+1faoWdumkegal17fiw6R7G82pS9CqeHX11VfbDz/8YFogRx99tF1yySWh\nrD1bfCUI+rse9erbt2+Jyfbr18/OPvts3+1f//qX/d///Z+vkNWqVSu7/fbbrWXLliWGz6UfWq8u\nuOAC+93vfmcnnXRSfNYrcqqoXzyDLP+iJxH+/Oc/2+jRo61169Z+bita93J5+1XF0b///e/23nvv\n+cftdI/8wgsvtB133NG7ffTRRzZ06FD/Xfu2G2+80Xr37p20nx8gi//MmTPHhg0bZgsWLPAHJrvs\nsotdccUV8cfbzjrrLPv222/jAqqv8dRTT/nfgWyjbsHmTLrnnnsiN9xwQ8Qd8UTmzZsXOfnkkyPv\nvPNOzsx/qjP66quvRlxAibgjxvjHHSn60d2GHjn++OMjboX2/YYMGRLRJ1fTF198EXEHJ5EDDzww\n4gJJnKEip4r6xTPI4i/uMcLIuHHjIkceeWRk//33j/z000/xua1o3cvl7fftt9+OnHnmmRH3TLm3\nevrppyPu5MF/13Z6yimnRF555RX/+7XXXvO2se03Ub84ehZ/GTx4cOSBBx6IaJ3Tfv/iiy+OPPfc\nc36O3dmt327nz58f38/JTCmobTSnLkG/8cYbduKJJ/rnsVSj97DDDjO3wcePfvgSFfj666+te/fu\nvjalHuhX4xt6/Ehp0qRJdvDBB1u7du18dz0X7DZ432hCdOzc+usO6OyYY46Jn23E5r4ip4r6xcbP\n5v8ffvih/fe//7XbbrutzGxWtO7l8vartg/OP//8+FMIe++9t02bNs1fPv3mm2/8I1u/+c1vvKe2\nT50hf/DBB1ZRvzL4WdbBxVFTrXB3wuCvcuo53F133dXeffddP6ffffedd2rfvr1v+Ej7OX2UgtpG\ncyYA676IO9I2YceSvi9atCj2k/8/C2gn+Pnnn/tGNw499FB/OUuNIii5o8UyhrpPosteuZgef/xx\nO+6448rcxqjIqaJ+uWDYq1cve/TRR/1BXun5TbTu5fr2++tf/9r22WefONf48eOtZ8+ePmDo8qoO\niIsn/da+raJ+xYfPxu9qmEe3zmKPnOp+78SJE+1Xv/qVn12ta2rNTJfytZ/T5egvv/zS9wtqG82Z\nAKyKGzoiaty4cXxdU1OUalmLVFJAD6Rrg3/iiSds+PDh/iha/5XUqERxw1hTgbnqGDtiLilYsVOu\nGyYyk2GidY/t95c17H//+5+pNb+rrrrKd1SlrCZNmvwygPumbVTbZEX9SoyQ5T8UfHVfXHVVdBU0\nlnQwqHvnzz77rD8gvOaaa0wnFEFtoznTEEesLV6dycXaDFYbvbHKH7EFwn+zv/3tb3GGbt26+cpq\nY8aM8d3kGDsbVofYdxzjZEmdMCxpVfxXonXvoosu8oPl+varJjq1Ld5999227bbbxtc17cuKp9i+\nTWd4ifoVHz6bv2v+//rXv/r2u++888747TRVmtQnlvr372+6sqBL00FtozlzBqyj7ubNm/tLqDFw\nXWbo0KFD7Cf/nYAu9andYjXNFku6lKN75kq6tCW3WNJ3ucbOhGPdc/1/RU4V9ctlt4rWPbZf820V\nP//883b//ffb9ttvH19VtD7pUnPxFNu3VdSv+PDZ+l23xv7yl7+Ynta49dZb4ydfmt+33nrLZsyY\nEZ917ed0wKJL1kFtozkTgKV8+OGH+yrmOiKaPXu2vfzyy9anT5/4AuCL+aPD6dOn+zdO6ZL94sWL\n7YUXXog7ydDVtrRZs2b5s18dkccqf+D3i0BFThX1+yWH3Pumin7J1j09IpKL26/OzCZMmOCDiC43\n6wBZH1XO2nnnnX0lIlVs0zb70ksv+be96R5xRf1yYQ1ztaB9QNXjplpvZBa7XabfeoRSV1V0idrV\nLPeVtFSBLahtNKeeA9bR0HXXXeeDh2r3nnDCCf4mfS6siJWZR12C0ZtB9Ayd7oWogsKll14af5vH\nww8/7FdWvRhDR+I33XRTvHZmZaaTTcPqEtduu+1W4jngipwq6pdNLhXNi854VWPXPRYSvxVU0bqX\ny9vv6aef7tsvKO2p5/F1xqYDl0GDBvnKgLp8qmdd9cyrUkX9SueXTb9VA/yPf/xjmVnSs9Oq06IX\nfuhSvnsU1dcmb9Omjd+XbbXVVn6cILbRnArAsSWhoyAdRcYerYl1539JAbUWppcJlFdpRq3HqLJC\nTbydquRUs+tXRU4V9csuhcrPTUXrHttv+Z46+1VltfIaxamoX/m55U5XHQhqfdOttNIp3dtoTgbg\n0sj8RgABBBBAIGiBnLoHHDQu00MAAQQQQCCRAAE4kQzdEUAAAQQQSKMAATiNuGSNAAIIIIBAIgEC\ncCIZuiOQQwLnnnuu7b777vE51uNnagktlkr3j3XnPwIIVF0gZ1rCqjoRYyKQ/QJqZ7h4ozTnnXee\n6fV/aktXqXT/7BdhDhFIvwC1oNNvzBQQyDgBvWBCj5mpzWESAgikR4BL0OlxJVcEakRgypQptsce\ne9jUqVP9Waja3Hbvd/Wvois+ATWrd8ABB/iXGahxFDXUr2cYY0mNWKhRAr0BTM+JqgGM2GvZNIwa\npD/jjDP84H/+859Nr/5z76D101azhsX7ayA1YnDXXXdZjx49/LPge+21l6lRiFhSudVNb5xRq0J6\nxlINQ6jRexICCEQFCMCsCQiEWGDlypWm9+ceddRRPvAqyKnRAL2DWG+6UVL/2DtgR4wYYQqgDz74\noH8PamzW1O311183vb/4oYce8m3eHnjggfE81OqZXkGpdOyxx/qG/rt06eLb0VXLSsX7a5jrr7/e\nB3kdDKgJP7UCprfM/Otf/1JvU7n1Ptrf/va3vh1xBWu9fUzDqBlTEgIIOAHXQgoJAQRCKuDa9Y1o\nMx0yZEi8hO7ebMQ1Fh+5/PLLfTf36siIO0uOuHaB48O4ZvT8eO79p77bNttsE3HNZcb7//jjjxH3\n9pfIzJkzfbdzzjkn4oJovL8LwpEjjzwy/rt4/++//z7iWkiLuMbt4/31xTVZGnHNIkbcmXckVu6/\n//3v8WF++OEHXyZ3cBDvxhcEclmAM2AOwxDIAIHTTjstXkq9TlNnlmrj1+28/Jmm2grW21xiSf2V\n1M6t0t577216FZtePq5G/fV2mEceecS/1N0PUIk/eoOMGq/v27dvibGOOOII/xL4b7/9Nt79oIMO\nin9XG7squ86OSQggYEYAZi1AIAMEYg3Ex4qqBvjV2PzChQt9beXY6yJj/TW83hfrznR9JzUsr8u/\njz76qA/eanhe94l1L7eySS9MULAvXmtaeeiStlJsmvpe+j3R9erV82/wUT8SArkuQADO9TWA+c8I\nAVWiKp50/7dr166+QpXeYaq3VpVOule83Xbb+c66j6tX+Wk83UdWxSh3edhXpCo9XrLfCt4681bD\n/8VT7DVvsWmqX/Gz8uLD8h0BBDgDZh1AICMEVMs5lvQO2DfffNN+9atf+VdEqtaz3tFcPH300Uf+\ncrDeB7t+/XpfaWv06NH+LWDHH3+86bsCpS5jl5cUOBOdHet1bkqlp6nfCvRdXOUtEgIIJBfgDDi5\nEUMgUOsCer+rajvrrHPAgAH+v2o2K11zzTX+nbrDhg0znSl/8sknvgEN1Uzef//9/X1XnbVee+21\n9uqrr/pxdTas2sixy8alZ1CvmXQVtPz9Yr24vHhSUFet7IEDB/rHldT/mWee8WfTevE5CQEEUhOg\nJazUnBgKgVoV0KNBCqY6m+3Zs6eNGTPG3+NVof7whz/4oPq3v/3NB0VVdFJgHTlypDVu3NiXW48N\nKXDrPrDep6tHgq6++mo7++yzy52vM88808aNG+fvF+vxpdLp8ccft4suusgOOeQQf6ase9IKvoMH\nDy49KL8RQCCBAC1hJYChMwJhEFCNZdVoVsUnVXpSG82dOnUqt2i6L6vndTVc/fr1yx1Gl5XdY0Tm\nHkvyzwKXO9DPHWP3ect7wXtsPB0QqKEO3Y8mIYBA5QQ4A66cF0MjUGsCCqqJgq8Kpfu2ye6/5ufn\npxwslV9FwVfT1Nk2wVcSJAQqL8A94MqbMQYCCCCAAALVFuASdLUJyQCB9AnoUSJdfu7evbu51qfS\nNyFyRgCBwAUIwIGTM0EEEEAAAQR4Dph1AAEEEEAAgVoR4B5wrbAzUQQQQACBXBf4/whAR5INn77m\nAAAAAElFTkSuQmCC\n"
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "%%R\n",
    "library(\"ggplot2\")\n",
    "testqual = read.csv(\"test.quality\", sep='\\t')\n",
    "ggplot(testqual, aes(x=column, y=med, ymin=Q1, ymax=Q3)) + geom_smooth(se=FALSE) + geom_linerange(alpha=0.3) + labs(x='position', y='median quality score') + theme_bw() + theme( text = element_text(size=16) ) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\r\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\r\n",
      "http://drive5.com/usearch\r\n",
      "\r\n",
      "License: twhitman@wisc.edu\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!usearch9"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n",
      "usearch v9.2.64_i86osx32, 4.0Gb RAM (17.2Gb total), 4 cores\n",
      "(C) Copyright 2013-16 Robert C. Edgar, all rights reserved.\n",
      "http://drive5.com/usearch\n",
      "\n",
      "License: twhitman@wisc.edu\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 20Mb     75.7% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     30583  Reads (30.6k)                   \n",
      "        27  Discarded reads with expected errs > 1.00\n",
      "     30556  Filtered reads (30.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb     29.2% Filtering, 99.9% passed\r",
      "00:02 35Mb     68.0% Filtering, 99.9% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     78961  Reads (79.0k)                   \n",
      "        85  Discarded reads with expected errs > 1.00\n",
      "     78876  Filtered reads (78.9k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 7.9Mb    57.0% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     10230  Reads (10.2k)                   \n",
      "        10  Discarded reads with expected errs > 1.00\n",
      "     10220  Filtered reads (10.2k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     22527  Reads (22.5k)                   \n",
      "        17  Discarded reads with expected errs > 1.00\n",
      "     22510  Filtered reads (22.5k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 20Mb      4.2% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     31175  Reads (31.2k)                   \n",
      "        26  Discarded reads with expected errs > 1.00\n",
      "     31149  Filtered reads (31.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 18Mb      8.5% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "     27241  Reads (27.2k)                    \n",
      "        12  Discarded reads with expected errs > 1.00\n",
      "     27229  Filtered reads (27.2k, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 19Mb     24.9% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "     29062  Reads (29.1k)                    \n",
      "         8  Discarded reads with expected errs > 1.00\n",
      "     29054  Filtered reads (29.1k, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 11Mb     40.6% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     15281  Reads (15.3k)                   \n",
      "        20  Discarded reads with expected errs > 1.00\n",
      "     15261  Filtered reads (15.3k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     14035  Reads (14.0k)                   \n",
      "         8  Discarded reads with expected errs > 1.00\n",
      "     14027  Filtered reads (14.0k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 14Mb     27.7% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     20106  Reads (20.1k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     20092  Filtered reads (20.1k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 12Mb     95.4% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     17432  Reads (17.4k)                   \n",
      "        11  Discarded reads with expected errs > 1.00\n",
      "     17421  Filtered reads (17.4k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "      9666  Reads (9666)                    \n",
      "         9  Discarded reads with expected errs > 1.00\n",
      "      9657  Filtered reads (9657, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 17Mb     74.8% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     26087  Reads (26.1k)                   \n",
      "        19  Discarded reads with expected errs > 1.00\n",
      "     26068  Filtered reads (26.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     18823  Reads (18.8k)                   \n",
      "        18  Discarded reads with expected errs > 1.00\n",
      "     18805  Filtered reads (18.8k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 12Mb      8.8% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     16841  Reads (16.8k)                   \n",
      "        13  Discarded reads with expected errs > 1.00\n",
      "     16828  Filtered reads (16.8k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 12Mb     78.5% Filtering, 100.0% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed \r\n",
      "     17988  Reads (18.0k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     17974  Filtered reads (18.0k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 29Mb     55.0% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     46370  Reads (46.4k)                   \n",
      "        36  Discarded reads with expected errs > 1.00\n",
      "     46334  Filtered reads (46.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 22Mb     26.3% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     34495  Reads (34.5k)                   \n",
      "        31  Discarded reads with expected errs > 1.00\n",
      "     34464  Filtered reads (34.5k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 20Mb     14.8% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     31060  Reads (31.1k)                   \n",
      "        24  Discarded reads with expected errs > 1.00\n",
      "     31036  Filtered reads (31.0k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 19Mb     10.7% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     28481  Reads (28.5k)                   \n",
      "        28  Discarded reads with expected errs > 1.00\n",
      "     28453  Filtered reads (28.5k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     13175  Reads (13.2k)                   \n",
      "         7  Discarded reads with expected errs > 1.00\n",
      "     13168  Filtered reads (13.2k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 22Mb     44.1% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     35130  Reads (35.1k)                   \n",
      "        30  Discarded reads with expected errs > 1.00\n",
      "     35100  Filtered reads (35.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb     15.7% Filtering, 99.9% passed\r",
      "00:02 35Mb     75.3% Filtering, 99.9% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.8% passed\r\n",
      "     56269  Reads (56.3k)                   \n",
      "        91  Discarded reads with expected errs > 1.00\n",
      "     56178  Filtered reads (56.2k, 99.8%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 19Mb     82.4% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     29715  Reads (29.7k)                   \n",
      "        21  Discarded reads with expected errs > 1.00\n",
      "     29694  Filtered reads (29.7k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     24518  Reads (24.5k)                   \n",
      "        28  Discarded reads with expected errs > 1.00\n",
      "     24490  Filtered reads (24.5k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 11Mb     26.9% Filtering, 100.0% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed \r\n",
      "     15737  Reads (15.7k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     15723  Filtered reads (15.7k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     22090  Reads (22.1k)                   \n",
      "        23  Discarded reads with expected errs > 1.00\n",
      "     22067  Filtered reads (22.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 23Mb      8.7% Filtering, 99.8% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     36503  Reads (36.5k)                   \n",
      "        25  Discarded reads with expected errs > 1.00\n",
      "     36478  Filtered reads (36.5k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 16Mb     33.5% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     24400  Reads (24.4k)                   \n",
      "        20  Discarded reads with expected errs > 1.00\n",
      "     24380  Filtered reads (24.4k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 22Mb     62.3% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     33823  Reads (33.8k)                   \n",
      "        31  Discarded reads with expected errs > 1.00\n",
      "     33792  Filtered reads (33.8k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 23Mb     68.9% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     36967  Reads (37.0k)                   \n",
      "        33  Discarded reads with expected errs > 1.00\n",
      "     36934  Filtered reads (36.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     10977  Reads (11.0k)                   \n",
      "        13  Discarded reads with expected errs > 1.00\n",
      "     10964  Filtered reads (11.0k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 17Mb     56.4% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     25941  Reads (25.9k)                   \n",
      "        25  Discarded reads with expected errs > 1.00\n",
      "     25916  Filtered reads (25.9k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     17568  Reads (17.6k)                   \n",
      "        15  Discarded reads with expected errs > 1.00\n",
      "     17553  Filtered reads (17.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 14Mb     23.7% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     20525  Reads (20.5k)                   \n",
      "        24  Discarded reads with expected errs > 1.00\n",
      "     20501  Filtered reads (20.5k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     15393  Reads (15.4k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     15379  Filtered reads (15.4k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 22Mb     20.3% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     34276  Reads (34.3k)                   \n",
      "        35  Discarded reads with expected errs > 1.00\n",
      "     34241  Filtered reads (34.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 25Mb     29.4% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     39780  Reads (39.8k)                   \n",
      "        23  Discarded reads with expected errs > 1.00\n",
      "     39757  Filtered reads (39.8k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 10Mb     90.5% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     13974  Reads (14.0k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     13960  Filtered reads (14.0k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "     22551  Reads (22.6k)                    \n",
      "         8  Discarded reads with expected errs > 1.00\n",
      "     22543  Filtered reads (22.5k, 100.0%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 12Mb     84.5% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     17232  Reads (17.2k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     17218  Filtered reads (17.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     12998  Reads (13.0k)                   \n",
      "        12  Discarded reads with expected errs > 1.00\n",
      "     12986  Filtered reads (13.0k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     21345  Reads (21.3k)                   \n",
      "        26  Discarded reads with expected errs > 1.00\n",
      "     21319  Filtered reads (21.3k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.8% passed\r\n",
      "     26143  Reads (26.1k)                   \n",
      "        63  Discarded reads with expected errs > 1.00\n",
      "     26080  Filtered reads (26.1k, 99.8%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 9.3Mb    57.0% Filtering, 100.0% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed \r\n",
      "     12565  Reads (12.6k)                   \n",
      "        17  Discarded reads with expected errs > 1.00\n",
      "     12548  Filtered reads (12.5k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     13381  Reads (13.4k)                   \n",
      "        16  Discarded reads with expected errs > 1.00\n",
      "     13365  Filtered reads (13.4k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     10721  Reads (10.7k)                   \n",
      "         9  Discarded reads with expected errs > 1.00\n",
      "     10712  Filtered reads (10.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 8.9Mb    59.7% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     11884  Reads (11.9k)                   \n",
      "         6  Discarded reads with expected errs > 1.00\n",
      "     11878  Filtered reads (11.9k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 35Mb     36.8% Filtering, 99.9% passed\r",
      "00:02 35Mb     79.3% Filtering, 99.9% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     83877  Reads (83.9k)                   \n",
      "       102  Discarded reads with expected errs > 1.00\n",
      "     83775  Filtered reads (83.8k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     15092  Reads (15.1k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     15078  Filtered reads (15.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 9.7Mb    61.7% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     13267  Reads (13.3k)                   \n",
      "        11  Discarded reads with expected errs > 1.00\n",
      "     13256  Filtered reads (13.3k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     16830  Reads (16.8k)                   \n",
      "        11  Discarded reads with expected errs > 1.00\n",
      "     16819  Filtered reads (16.8k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "      8731  Reads (8731)                    \n",
      "         6  Discarded reads with expected errs > 1.00\n",
      "      8725  Filtered reads (8725, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     14190  Reads (14.2k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     14176  Filtered reads (14.2k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 23Mb     40.6% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     35915  Reads (35.9k)                   \n",
      "        28  Discarded reads with expected errs > 1.00\n",
      "     35887  Filtered reads (35.9k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 35Mb     16.6% Filtering, 99.9% passed\r",
      "00:02 35Mb     64.1% Filtering, 99.9% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     72495  Reads (72.5k)                   \n",
      "        62  Discarded reads with expected errs > 1.00\n",
      "     72433  Filtered reads (72.4k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 11Mb     54.2% Filtering, 100.0% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed \r\n",
      "     15351  Reads (15.4k)                   \n",
      "        12  Discarded reads with expected errs > 1.00\n",
      "     15339  Filtered reads (15.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     16901  Reads (16.9k)                   \n",
      "        16  Discarded reads with expected errs > 1.00\n",
      "     16885  Filtered reads (16.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     13329  Reads (13.3k)                   \n",
      "         9  Discarded reads with expected errs > 1.00\n",
      "     13320  Filtered reads (13.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 27Mb      0.2% Filtering, 100.0% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     42896  Reads (42.9k)                   \n",
      "        40  Discarded reads with expected errs > 1.00\n",
      "     42856  Filtered reads (42.9k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     19067  Reads (19.1k)                   \n",
      "        13  Discarded reads with expected errs > 1.00\n",
      "     19054  Filtered reads (19.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     21366  Reads (21.4k)                   \n",
      "        30  Discarded reads with expected errs > 1.00\n",
      "     21336  Filtered reads (21.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 32Mb     69.9% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     51612  Reads (51.6k)                   \n",
      "        50  Discarded reads with expected errs > 1.00\n",
      "     51562  Filtered reads (51.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb     18.3% Filtering, 100.0% passed\r",
      "00:02 35Mb     47.7% Filtering, 100.0% passed\r",
      "00:03 35Mb     82.6% Filtering, 99.9% passed \r",
      "00:03 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "    127263  Reads (127.3k)                  \n",
      "       111  Discarded reads with expected errs > 1.00\n",
      "    127152  Filtered reads (127.2k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 15Mb     95.8% Filtering, 99.8% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.8% passed\r\n",
      "     22091  Reads (22.1k)                   \n",
      "        35  Discarded reads with expected errs > 1.00\n",
      "     22056  Filtered reads (22.1k, 99.8%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     21311  Reads (21.3k)                   \n",
      "        17  Discarded reads with expected errs > 1.00\n",
      "     21294  Filtered reads (21.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.8% passed\r\n",
      "      7203  Reads (7203)                    \n",
      "        11  Discarded reads with expected errs > 1.00\n",
      "      7192  Filtered reads (7192, 99.8%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 14Mb      9.7% Filtering, 99.8% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     20636  Reads (20.6k)                   \n",
      "        19  Discarded reads with expected errs > 1.00\n",
      "     20617  Filtered reads (20.6k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 10Mb     91.0% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     14122  Reads (14.1k)                   \n",
      "        11  Discarded reads with expected errs > 1.00\n",
      "     14111  Filtered reads (14.1k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     25160  Reads (25.2k)                   \n",
      "        30  Discarded reads with expected errs > 1.00\n",
      "     25130  Filtered reads (25.1k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 17Mb     11.9% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "     25427  Reads (25.4k)                    \n",
      "        12  Discarded reads with expected errs > 1.00\n",
      "     25415  Filtered reads (25.4k, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb      5.3% Filtering, 100.0% passed\r",
      "00:02 35Mb     28.2% Filtering, 100.0% passed\r",
      "00:03 35Mb     51.7% Filtering, 100.0% passed\r",
      "00:04 35Mb     74.8% Filtering, 100.0% passed\r",
      "00:05 35Mb     99.7% Filtering, 99.9% passed \r",
      "00:05 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "    130273  Reads (130.3k)                  \n",
      "        67  Discarded reads with expected errs > 1.00\n",
      "    130206  Filtered reads (130.2k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 36Mb     20.5% Filtering, 99.9% passed\r",
      "00:02 36Mb     41.6% Filtering, 99.9% passed\r",
      "00:03 36Mb     63.8% Filtering, 99.9% passed\r",
      "00:04 36Mb     85.4% Filtering, 99.9% passed\r",
      "00:04 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "    139687  Reads (139.7k)                  \n",
      "       130  Discarded reads with expected errs > 1.00\n",
      "    139557  Filtered reads (139.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb      9.0% Filtering, 99.9% passed\r",
      "00:02 35Mb     44.4% Filtering, 100.0% passed\r",
      "00:03 35Mb     79.5% Filtering, 99.9% passed \r",
      "00:03 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     79675  Reads (79.7k)                   \n",
      "        60  Discarded reads with expected errs > 1.00\n",
      "     79615  Filtered reads (79.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 28Mb     25.7% Filtering, 99.9% passed\r",
      "00:02 28Mb     72.5% Filtering, 100.0% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     44459  Reads (44.5k)                   \n",
      "        23  Discarded reads with expected errs > 1.00\n",
      "     44436  Filtered reads (44.4k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 20Mb     50.8% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     31123  Reads (31.1k)                   \n",
      "        28  Discarded reads with expected errs > 1.00\n",
      "     31095  Filtered reads (31.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb     13.4% Filtering, 99.9% passed\r",
      "00:02 35Mb     54.1% Filtering, 99.9% passed\r",
      "00:03 35Mb     95.2% Filtering, 99.9% passed\r",
      "00:03 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     89311  Reads (89.3k)                   \n",
      "        87  Discarded reads with expected errs > 1.00\n",
      "     89224  Filtered reads (89.2k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "     25613  Reads (25.6k)                    \n",
      "        12  Discarded reads with expected errs > 1.00\n",
      "     25601  Filtered reads (25.6k, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     10564  Reads (10.6k)                   \n",
      "         8  Discarded reads with expected errs > 1.00\n",
      "     10556  Filtered reads (10.6k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "      9163  Reads (9163)                    \n",
      "         8  Discarded reads with expected errs > 1.00\n",
      "      9155  Filtered reads (9155, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     11262  Reads (11.3k)                   \n",
      "         9  Discarded reads with expected errs > 1.00\n",
      "     11253  Filtered reads (11.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 11Mb     10.9% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     16087  Reads (16.1k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     16073  Filtered reads (16.1k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 100.0% passed\r\n",
      "      6821  Reads (6821)                     \n",
      "         1  Discarded reads with expected errs > 1.00\n",
      "      6820  Filtered reads (6820, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     12563  Reads (12.6k)                   \n",
      "        12  Discarded reads with expected errs > 1.00\n",
      "     12551  Filtered reads (12.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 8.3Mb    47.3% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.8% passed\r\n",
      "     10900  Reads (10.9k)                   \n",
      "        21  Discarded reads with expected errs > 1.00\n",
      "     10879  Filtered reads (10.9k, 99.8%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 21Mb     82.5% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     32228  Reads (32.2k)                   \n",
      "        30  Discarded reads with expected errs > 1.00\n",
      "     32198  Filtered reads (32.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     19565  Reads (19.6k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     19551  Filtered reads (19.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 21Mb     36.6% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.8% passed\r\n",
      "     33330  Reads (33.3k)                   \n",
      "        50  Discarded reads with expected errs > 1.00\n",
      "     33280  Filtered reads (33.3k, 99.8%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 24Mb     58.3% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     37421  Reads (37.4k)                   \n",
      "        37  Discarded reads with expected errs > 1.00\n",
      "     37384  Filtered reads (37.4k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     18087  Reads (18.1k)                   \n",
      "        19  Discarded reads with expected errs > 1.00\n",
      "     18068  Filtered reads (18.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 20Mb     18.7% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     30148  Reads (30.1k)                   \n",
      "        44  Discarded reads with expected errs > 1.00\n",
      "     30104  Filtered reads (30.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     12946  Reads (12.9k)                   \n",
      "         7  Discarded reads with expected errs > 1.00\n",
      "     12939  Filtered reads (12.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "     17899  Reads (17.9k)                    \n",
      "         5  Discarded reads with expected errs > 1.00\n",
      "     17894  Filtered reads (17.9k, 100.0%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "     12632  Reads (12.6k)                    \n",
      "         6  Discarded reads with expected errs > 1.00\n",
      "     12626  Filtered reads (12.6k, 100.0%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 15Mb     10.8% Filtering, 100.0% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 100.0% passed\r\n",
      "     22173  Reads (22.2k)                    \n",
      "         7  Discarded reads with expected errs > 1.00\n",
      "     22166  Filtered reads (22.2k, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 15Mb     79.2% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     22908  Reads (22.9k)                   \n",
      "        16  Discarded reads with expected errs > 1.00\n",
      "     22892  Filtered reads (22.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     20915  Reads (20.9k)                   \n",
      "        20  Discarded reads with expected errs > 1.00\n",
      "     20895  Filtered reads (20.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 9.7Mb    60.3% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     13371  Reads (13.4k)                   \n",
      "         9  Discarded reads with expected errs > 1.00\n",
      "     13362  Filtered reads (13.4k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "      8583  Reads (8583)                    \n",
      "         6  Discarded reads with expected errs > 1.00\n",
      "      8577  Filtered reads (8577, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     14567  Reads (14.6k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     14553  Filtered reads (14.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     17038  Reads (17.0k)                   \n",
      "        17  Discarded reads with expected errs > 1.00\n",
      "     17021  Filtered reads (17.0k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 36Mb     11.1% Filtering, 100.0% passed\r",
      "00:02 36Mb     58.1% Filtering, 100.0% passed\r",
      "00:02 2.0Mb   100.0% Filtering, 99.9% passed \r\n",
      "     58661  Reads (58.7k)                   \n",
      "        47  Discarded reads with expected errs > 1.00\n",
      "     58614  Filtered reads (58.6k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 7.5Mb    75.5% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "      9447  Reads (9447)                    \n",
      "        10  Discarded reads with expected errs > 1.00\n",
      "      9437  Filtered reads (9437, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.8% passed\r\n",
      "      4944  Reads (4944)                    \n",
      "         8  Discarded reads with expected errs > 1.00\n",
      "      4936  Filtered reads (4936, 99.8%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     14182  Reads (14.2k)                   \n",
      "        13  Discarded reads with expected errs > 1.00\n",
      "     14169  Filtered reads (14.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 7.5Mb    89.7% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "      9567  Reads (9567)                    \n",
      "         5  Discarded reads with expected errs > 1.00\n",
      "      9562  Filtered reads (9562, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     10707  Reads (10.7k)                   \n",
      "         7  Discarded reads with expected errs > 1.00\n",
      "     10700  Filtered reads (10.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "     11307  Reads (11.3k)                    \n",
      "         5  Discarded reads with expected errs > 1.00\n",
      "     11302  Filtered reads (11.3k, 100.0%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 7.5Mb    49.5% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "      9437  Reads (9437)                    \n",
      "        10  Discarded reads with expected errs > 1.00\n",
      "      9427  Filtered reads (9427, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     22871  Reads (22.9k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     22857  Filtered reads (22.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 5.3Mb    12.7% Filtering, 99.6% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.8% passed\r\n",
      "      5779  Reads (5779)                    \n",
      "        10  Discarded reads with expected errs > 1.00\n",
      "      5769  Filtered reads (5769, 99.8%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     16313  Reads (16.3k)                   \n",
      "        15  Discarded reads with expected errs > 1.00\n",
      "     16298  Filtered reads (16.3k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 12Mb     47.1% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     16475  Reads (16.5k)                   \n",
      "        16  Discarded reads with expected errs > 1.00\n",
      "     16459  Filtered reads (16.5k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     14213  Reads (14.2k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     14199  Filtered reads (14.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 12Mb     37.8% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     17028  Reads (17.0k)                   \n",
      "        15  Discarded reads with expected errs > 1.00\n",
      "     17013  Filtered reads (17.0k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 14Mb     95.8% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     21351  Reads (21.4k)                   \n",
      "        28  Discarded reads with expected errs > 1.00\n",
      "     21323  Filtered reads (21.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "      9784  Reads (9784)                    \n",
      "         8  Discarded reads with expected errs > 1.00\n",
      "      9776  Filtered reads (9776, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 16Mb     76.2% Filtering, 100.0% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed \r\n",
      "     23368  Reads (23.4k)                   \n",
      "        15  Discarded reads with expected errs > 1.00\n",
      "     23353  Filtered reads (23.4k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "      8851  Reads (8851)                     \n",
      "         2  Discarded reads with expected errs > 1.00\n",
      "      8849  Filtered reads (8849, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 11Mb     92.0% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     15898  Reads (15.9k)                   \n",
      "         9  Discarded reads with expected errs > 1.00\n",
      "     15889  Filtered reads (15.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     18799  Reads (18.8k)                   \n",
      "        20  Discarded reads with expected errs > 1.00\n",
      "     18779  Filtered reads (18.8k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 16Mb     39.6% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     24434  Reads (24.4k)                   \n",
      "        17  Discarded reads with expected errs > 1.00\n",
      "     24417  Filtered reads (24.4k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     15585  Reads (15.6k)                   \n",
      "        15  Discarded reads with expected errs > 1.00\n",
      "     15570  Filtered reads (15.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     16878  Reads (16.9k)                   \n",
      "        14  Discarded reads with expected errs > 1.00\n",
      "     16864  Filtered reads (16.9k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 22Mb     39.3% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     34173  Reads (34.2k)                   \n",
      "        20  Discarded reads with expected errs > 1.00\n",
      "     34153  Filtered reads (34.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 22Mb     32.7% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     35067  Reads (35.1k)                   \n",
      "        26  Discarded reads with expected errs > 1.00\n",
      "     35041  Filtered reads (35.0k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 21Mb     39.3% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     31923  Reads (31.9k)                   \n",
      "        16  Discarded reads with expected errs > 1.00\n",
      "     31907  Filtered reads (31.9k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     18171  Reads (18.2k)                   \n",
      "        27  Discarded reads with expected errs > 1.00\n",
      "     18144  Filtered reads (18.1k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 15Mb      0.2% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     21617  Reads (21.6k)                   \n",
      "        11  Discarded reads with expected errs > 1.00\n",
      "     21606  Filtered reads (21.6k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     13760  Reads (13.8k)                   \n",
      "        18  Discarded reads with expected errs > 1.00\n",
      "     13742  Filtered reads (13.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 21Mb      5.7% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     33294  Reads (33.3k)                   \n",
      "        22  Discarded reads with expected errs > 1.00\n",
      "     33272  Filtered reads (33.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 16Mb     29.3% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     23353  Reads (23.4k)                   \n",
      "        13  Discarded reads with expected errs > 1.00\n",
      "     23340  Filtered reads (23.3k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 18Mb     80.0% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     27179  Reads (27.2k)                   \n",
      "        26  Discarded reads with expected errs > 1.00\n",
      "     27153  Filtered reads (27.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     22345  Reads (22.3k)                   \n",
      "        18  Discarded reads with expected errs > 1.00\n",
      "     22327  Filtered reads (22.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb     14.5% Filtering, 100.0% passed\r",
      "00:02 35Mb     63.2% Filtering, 100.0% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     72970  Reads (73.0k)                   \n",
      "        41  Discarded reads with expected errs > 1.00\n",
      "     72929  Filtered reads (72.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 26Mb     14.4% Filtering, 100.0% passed\r",
      "00:02 26Mb     99.6% Filtering, 99.9% passed \r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     41496  Reads (41.5k)                   \n",
      "        36  Discarded reads with expected errs > 1.00\n",
      "     41460  Filtered reads (41.5k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 24Mb     94.2% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     37164  Reads (37.2k)                   \n",
      "        24  Discarded reads with expected errs > 1.00\n",
      "     37140  Filtered reads (37.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 22Mb     97.9% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     34321  Reads (34.3k)                   \n",
      "        32  Discarded reads with expected errs > 1.00\n",
      "     34289  Filtered reads (34.3k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 36Mb     45.7% Filtering, 100.0% passed\r",
      "00:02 36Mb     80.7% Filtering, 99.9% passed \r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     83279  Reads (83.3k)                   \n",
      "        54  Discarded reads with expected errs > 1.00\n",
      "     83225  Filtered reads (83.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 96.5% passed\r\n",
      "        57  Reads (57)                      \n",
      "         2  Discarded reads with expected errs > 1.00\n",
      "        55  Filtered reads (55, 96.5%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb     26.0% Filtering, 99.9% passed\r",
      "00:02 35Mb     76.2% Filtering, 99.9% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     62670  Reads (62.7k)                   \n",
      "        63  Discarded reads with expected errs > 1.00\n",
      "     62607  Filtered reads (62.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "        22  Reads (22)                       \n",
      "         0  Discarded reads with expected errs > 1.00\n",
      "        22  Filtered reads (22, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 33Mb     32.3% Filtering, 100.0% passed\r",
      "00:02 33Mb     88.6% Filtering, 99.9% passed \r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     53761  Reads (53.8k)                   \n",
      "        31  Discarded reads with expected errs > 1.00\n",
      "     53730  Filtered reads (53.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 28Mb     67.6% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     45110  Reads (45.1k)                   \n",
      "        49  Discarded reads with expected errs > 1.00\n",
      "     45061  Filtered reads (45.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 24Mb     49.5% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     38385  Reads (38.4k)                   \n",
      "        27  Discarded reads with expected errs > 1.00\n",
      "     38358  Filtered reads (38.4k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 19Mb     37.5% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     29712  Reads (29.7k)                   \n",
      "        30  Discarded reads with expected errs > 1.00\n",
      "     29682  Filtered reads (29.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 15Mb     82.9% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     22343  Reads (22.3k)                   \n",
      "        23  Discarded reads with expected errs > 1.00\n",
      "     22320  Filtered reads (22.3k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 32Mb     56.3% Filtering, 100.0% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed \r\n",
      "     50782  Reads (50.8k)                   \n",
      "        34  Discarded reads with expected errs > 1.00\n",
      "     50748  Filtered reads (50.7k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "        29  Reads (29)                       \n",
      "         0  Discarded reads with expected errs > 1.00\n",
      "        29  Filtered reads (29, 100.0%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 36Mb      1.2% Filtering, 99.9% passed\r",
      "00:02 36Mb     24.8% Filtering, 99.9% passed\r",
      "00:03 36Mb     44.3% Filtering, 99.9% passed\r",
      "00:04 36Mb     65.3% Filtering, 99.9% passed\r",
      "00:05 36Mb     82.7% Filtering, 99.9% passed\r",
      "00:06 36Mb     98.2% Filtering, 99.9% passed\r",
      "00:06 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "    141460  Reads (141.5k)                  \n",
      "       130  Discarded reads with expected errs > 1.00\n",
      "    141330  Filtered reads (141.3k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "        20  Reads (20)                       \n",
      "         0  Discarded reads with expected errs > 1.00\n",
      "        20  Filtered reads (20, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 27Mb     65.3% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     43586  Reads (43.6k)                   \n",
      "        36  Discarded reads with expected errs > 1.00\n",
      "     43550  Filtered reads (43.5k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 22Mb     37.3% Filtering, 100.0% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed \r\n",
      "     33589  Reads (33.6k)                   \n",
      "        21  Discarded reads with expected errs > 1.00\n",
      "     33568  Filtered reads (33.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb     24.5% Filtering, 99.9% passed\r",
      "00:02 35Mb     68.1% Filtering, 99.9% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     69833  Reads (69.8k)                   \n",
      "        93  Discarded reads with expected errs > 1.00\n",
      "     69740  Filtered reads (69.7k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 6.7Mb    95.7% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "      8080  Reads (8080)                    \n",
      "         9  Discarded reads with expected errs > 1.00\n",
      "      8071  Filtered reads (8071, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     35211  Reads (35.2k)                   \n",
      "        52  Discarded reads with expected errs > 1.00\n",
      "     35159  Filtered reads (35.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     27731  Reads (27.7k)                   \n",
      "        31  Discarded reads with expected errs > 1.00\n",
      "     27700  Filtered reads (27.7k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 14Mb     52.8% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     20027  Reads (20.0k)                   \n",
      "        11  Discarded reads with expected errs > 1.00\n",
      "     20016  Filtered reads (20.0k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 24Mb     78.9% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     37729  Reads (37.7k)                   \n",
      "        45  Discarded reads with expected errs > 1.00\n",
      "     37684  Filtered reads (37.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 30Mb     59.4% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     47808  Reads (47.8k)                   \n",
      "        42  Discarded reads with expected errs > 1.00\n",
      "     47766  Filtered reads (47.8k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 13Mb     65.5% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "     18341  Reads (18.3k)                    \n",
      "         8  Discarded reads with expected errs > 1.00\n",
      "     18333  Filtered reads (18.3k, 100.0%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     21062  Reads (21.1k)                   \n",
      "        18  Discarded reads with expected errs > 1.00\n",
      "     21044  Filtered reads (21.0k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb      7.9% Filtering, 100.0% passed\r",
      "00:02 35Mb     57.5% Filtering, 100.0% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     71865  Reads (71.9k)                   \n",
      "        55  Discarded reads with expected errs > 1.00\n",
      "     71810  Filtered reads (71.8k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb     64.9% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     58350  Reads (58.4k)                   \n",
      "        75  Discarded reads with expected errs > 1.00\n",
      "     58275  Filtered reads (58.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 17Mb     44.2% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     25084  Reads (25.1k)                   \n",
      "        17  Discarded reads with expected errs > 1.00\n",
      "     25067  Filtered reads (25.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     26223  Reads (26.2k)                   \n",
      "        19  Discarded reads with expected errs > 1.00\n",
      "     26204  Filtered reads (26.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb     35.7% Filtering, 99.9% passed\r",
      "00:02 35Mb     69.0% Filtering, 99.9% passed\r",
      "00:03 35Mb     98.1% Filtering, 99.9% passed\r",
      "00:03 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     91341  Reads (91.3k)                   \n",
      "        98  Discarded reads with expected errs > 1.00\n",
      "     91243  Filtered reads (91.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 24Mb     83.0% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     38305  Reads (38.3k)                   \n",
      "        38  Discarded reads with expected errs > 1.00\n",
      "     38267  Filtered reads (38.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb     25.9% Filtering, 100.0% passed\r",
      "00:02 35Mb     50.8% Filtering, 99.9% passed \r",
      "00:03 35Mb     77.9% Filtering, 99.9% passed\r",
      "00:03 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "    110065  Reads (110.1k)                  \n",
      "       127  Discarded reads with expected errs > 1.00\n",
      "    109938  Filtered reads (109.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 16Mb     18.8% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     23531  Reads (23.5k)                   \n",
      "        23  Discarded reads with expected errs > 1.00\n",
      "     23508  Filtered reads (23.5k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 16Mb     73.5% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     24624  Reads (24.6k)                   \n",
      "        22  Discarded reads with expected errs > 1.00\n",
      "     24602  Filtered reads (24.6k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     19867  Reads (19.9k)                   \n",
      "        17  Discarded reads with expected errs > 1.00\n",
      "     19850  Filtered reads (19.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 21Mb     26.4% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     32872  Reads (32.9k)                   \n",
      "        27  Discarded reads with expected errs > 1.00\n",
      "     32845  Filtered reads (32.8k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 15Mb      4.4% Filtering, 100.0% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed \r\n",
      "     22860  Reads (22.9k)                   \n",
      "        30  Discarded reads with expected errs > 1.00\n",
      "     22830  Filtered reads (22.8k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 14Mb     64.2% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     21709  Reads (21.7k)                   \n",
      "        17  Discarded reads with expected errs > 1.00\n",
      "     21692  Filtered reads (21.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 24Mb     74.8% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     37847  Reads (37.8k)                   \n",
      "        24  Discarded reads with expected errs > 1.00\n",
      "     37823  Filtered reads (37.8k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 31Mb     46.1% Filtering, 100.0% passed\r",
      "00:02 31Mb     99.3% Filtering, 99.9% passed \r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     50303  Reads (50.3k)                   \n",
      "        36  Discarded reads with expected errs > 1.00\n",
      "     50267  Filtered reads (50.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     27508  Reads (27.5k)                   \n",
      "        20  Discarded reads with expected errs > 1.00\n",
      "     27488  Filtered reads (27.5k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 22Mb      8.5% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "     34815  Reads (34.8k)                    \n",
      "        17  Discarded reads with expected errs > 1.00\n",
      "     34798  Filtered reads (34.8k, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 26Mb      9.6% Filtering, 100.0% passed\r",
      "00:02 26Mb     94.1% Filtering, 99.9% passed \r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     40607  Reads (40.6k)                   \n",
      "        32  Discarded reads with expected errs > 1.00\n",
      "     40575  Filtered reads (40.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 36Mb     25.2% Filtering, 100.0% passed\r",
      "00:02 36Mb     46.2% Filtering, 100.0% passed\r",
      "00:03 36Mb     66.0% Filtering, 99.9% passed \r",
      "00:04 36Mb     85.1% Filtering, 99.9% passed\r",
      "00:04 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "    123682  Reads (123.7k)                  \n",
      "        90  Discarded reads with expected errs > 1.00\n",
      "    123592  Filtered reads (123.6k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 32Mb     15.4% Filtering, 100.0% passed\r",
      "00:02 32Mb     73.4% Filtering, 99.9% passed \r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     51247  Reads (51.2k)                   \n",
      "        30  Discarded reads with expected errs > 1.00\n",
      "     51217  Filtered reads (51.2k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 25Mb     34.7% Filtering, 99.9% passed\r",
      "00:02 25Mb     98.5% Filtering, 99.9% passed\r",
      "00:02 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     38786  Reads (38.8k)                   \n",
      "        50  Discarded reads with expected errs > 1.00\n",
      "     38736  Filtered reads (38.7k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     18925  Reads (18.9k)                   \n",
      "        20  Discarded reads with expected errs > 1.00\n",
      "     18905  Filtered reads (18.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 35Mb      9.7% Filtering, 100.0% passed\r",
      "00:02 35Mb     43.5% Filtering, 100.0% passed\r",
      "00:03 35Mb     72.4% Filtering, 99.9% passed \r",
      "00:03 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     80556  Reads (80.6k)                   \n",
      "        57  Discarded reads with expected errs > 1.00\n",
      "     80499  Filtered reads (80.5k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 26Mb     61.1% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "     41750  Reads (41.8k)                    \n",
      "        18  Discarded reads with expected errs > 1.00\n",
      "     41732  Filtered reads (41.7k, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 24Mb     28.4% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     37716  Reads (37.7k)                   \n",
      "        33  Discarded reads with expected errs > 1.00\n",
      "     37683  Filtered reads (37.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 23Mb     90.2% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     35267  Reads (35.3k)                   \n",
      "        30  Discarded reads with expected errs > 1.00\n",
      "     35237  Filtered reads (35.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 31Mb     57.0% Filtering, 99.9% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     50219  Reads (50.2k)                   \n",
      "        59  Discarded reads with expected errs > 1.00\n",
      "     50160  Filtered reads (50.2k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 36Mb     48.1% Filtering, 99.9% passed\r",
      "00:02 36Mb     85.3% Filtering, 99.9% passed\r",
      "00:02 2.0Mb   100.0% Filtering, 99.8% passed\r\n",
      "     65308  Reads (65.3k)                   \n",
      "       103  Discarded reads with expected errs > 1.00\n",
      "     65205  Filtered reads (65.2k, 99.8%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     19130  Reads (19.1k)                   \n",
      "        28  Discarded reads with expected errs > 1.00\n",
      "     19102  Filtered reads (19.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 9.1Mb     0.4% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     12288  Reads (12.3k)                   \n",
      "        13  Discarded reads with expected errs > 1.00\n",
      "     12275  Filtered reads (12.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     17823  Reads (17.8k)                   \n",
      "        15  Discarded reads with expected errs > 1.00\n",
      "     17808  Filtered reads (17.8k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     10718  Reads (10.7k)                   \n",
      "        11  Discarded reads with expected errs > 1.00\n",
      "     10707  Filtered reads (10.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     12070  Reads (12.1k)                   \n",
      "         9  Discarded reads with expected errs > 1.00\n",
      "     12061  Filtered reads (12.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 8.9Mb    70.3% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     11956  Reads (12.0k)                   \n",
      "        10  Discarded reads with expected errs > 1.00\n",
      "     11946  Filtered reads (11.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     11800  Reads (11.8k)                   \n",
      "         7  Discarded reads with expected errs > 1.00\n",
      "     11793  Filtered reads (11.8k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 98.4% passed\r\n",
      "        61  Reads (61)                      \n",
      "         1  Discarded reads with expected errs > 1.00\n",
      "        60  Filtered reads (60, 98.4%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "      4680  Reads (4680)                     \n",
      "         2  Discarded reads with expected errs > 1.00\n",
      "      4678  Filtered reads (4678, 100.0%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     11926  Reads (11.9k)                   \n",
      "         7  Discarded reads with expected errs > 1.00\n",
      "     11919  Filtered reads (11.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "      9018  Reads (9018)                    \n",
      "         9  Discarded reads with expected errs > 1.00\n",
      "      9009  Filtered reads (9009, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "      8576  Reads (8576)                     \n",
      "         3  Discarded reads with expected errs > 1.00\n",
      "      8573  Filtered reads (8573, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     13311  Reads (13.3k)                   \n",
      "        12  Discarded reads with expected errs > 1.00\n",
      "     13299  Filtered reads (13.3k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 8.3Mb    42.6% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     10825  Reads (10.8k)                   \n",
      "         8  Discarded reads with expected errs > 1.00\n",
      "     10817  Filtered reads (10.8k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     31212  Reads (31.2k)                   \n",
      "        31  Discarded reads with expected errs > 1.00\n",
      "     31181  Filtered reads (31.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     10705  Reads (10.7k)                   \n",
      "         7  Discarded reads with expected errs > 1.00\n",
      "     10698  Filtered reads (10.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     13507  Reads (13.5k)                   \n",
      "        13  Discarded reads with expected errs > 1.00\n",
      "     13494  Filtered reads (13.5k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "      6182  Reads (6182)                     \n",
      "         2  Discarded reads with expected errs > 1.00\n",
      "      6180  Filtered reads (6180, 100.0%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "        24  Reads (24)                       \n",
      "         0  Discarded reads with expected errs > 1.00\n",
      "        24  Filtered reads (24, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 2.1Mb    35.7% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.7% passed \r\n",
      "       286  Reads (286)                     \n",
      "         1  Discarded reads with expected errs > 1.00\n",
      "       285  Filtered reads (285, 99.7%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.8% passed\r\n",
      "       534  Reads (534)                     \n",
      "         1  Discarded reads with expected errs > 1.00\n",
      "       533  Filtered reads (533, 99.8%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.6% passed\r\n",
      "       244  Reads (244)                     \n",
      "         1  Discarded reads with expected errs > 1.00\n",
      "       243  Filtered reads (243, 99.6%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.8% passed\r\n",
      "      1063  Reads (1063)                    \n",
      "         2  Discarded reads with expected errs > 1.00\n",
      "      1061  Filtered reads (1061, 99.8%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 100.0% passed\r\n",
      "       190  Reads (190)                      \n",
      "         0  Discarded reads with expected errs > 1.00\n",
      "       190  Filtered reads (190, 100.0%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 98.3% passed\r\n",
      "       116  Reads (116)                     \n",
      "         2  Discarded reads with expected errs > 1.00\n",
      "       114  Filtered reads (114, 98.3%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 100.0% passed\r\n",
      "       358  Reads (358)                      \n",
      "         0  Discarded reads with expected errs > 1.00\n",
      "       358  Filtered reads (358, 100.0%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     26952  Reads (27.0k)                   \n",
      "        22  Discarded reads with expected errs > 1.00\n",
      "     26930  Filtered reads (26.9k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.7% passed\r\n",
      "      1070  Reads (1070)                    \n",
      "         3  Discarded reads with expected errs > 1.00\n",
      "      1067  Filtered reads (1067, 99.7%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 25Mb     10.1% Filtering, 99.9% passed\r",
      "00:02 25Mb     96.2% Filtering, 99.9% passed\r",
      "00:02 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     39683  Reads (39.7k)                   \n",
      "        37  Discarded reads with expected errs > 1.00\n",
      "     39646  Filtered reads (39.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     18401  Reads (18.4k)                   \n",
      "        15  Discarded reads with expected errs > 1.00\n",
      "     18386  Filtered reads (18.4k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 17Mb     56.2% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     25228  Reads (25.2k)                   \n",
      "        25  Discarded reads with expected errs > 1.00\n",
      "     25203  Filtered reads (25.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     20945  Reads (20.9k)                   \n",
      "        22  Discarded reads with expected errs > 1.00\n",
      "     20923  Filtered reads (20.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 8.4Mb    53.7% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     11059  Reads (11.1k)                   \n",
      "        11  Discarded reads with expected errs > 1.00\n",
      "     11048  Filtered reads (11.0k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     30505  Reads (30.5k)                   \n",
      "        31  Discarded reads with expected errs > 1.00\n",
      "     30474  Filtered reads (30.5k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     26134  Reads (26.1k)                   \n",
      "        29  Discarded reads with expected errs > 1.00\n",
      "     26105  Filtered reads (26.1k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 14Mb     61.1% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     19920  Reads (19.9k)                   \n",
      "        19  Discarded reads with expected errs > 1.00\n",
      "     19901  Filtered reads (19.9k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 26Mb     72.1% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     41029  Reads (41.0k)                   \n",
      "        31  Discarded reads with expected errs > 1.00\n",
      "     40998  Filtered reads (41.0k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     16465  Reads (16.5k)                   \n",
      "        13  Discarded reads with expected errs > 1.00\n",
      "     16452  Filtered reads (16.5k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 13Mb     21.1% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     18243  Reads (18.2k)                   \n",
      "        19  Discarded reads with expected errs > 1.00\n",
      "     18224  Filtered reads (18.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     13716  Reads (13.7k)                   \n",
      "        13  Discarded reads with expected errs > 1.00\n",
      "     13703  Filtered reads (13.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 10Mb     49.2% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     14638  Reads (14.6k)                   \n",
      "        19  Discarded reads with expected errs > 1.00\n",
      "     14619  Filtered reads (14.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     22558  Reads (22.6k)                   \n",
      "        20  Discarded reads with expected errs > 1.00\n",
      "     22538  Filtered reads (22.5k, 99.9%)\n",
      "00:00 1.9Mb     0.1% Filtering\r",
      "00:01 9.5Mb    82.2% Filtering, 99.9% passed\r",
      "00:01 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     12970  Reads (13.0k)                   \n",
      "        12  Discarded reads with expected errs > 1.00\n",
      "     12958  Filtered reads (13.0k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     22842  Reads (22.8k)                   \n",
      "        15  Discarded reads with expected errs > 1.00\n",
      "     22827  Filtered reads (22.8k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 24Mb     25.3% Filtering, 99.9% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     37216  Reads (37.2k)                   \n",
      "        34  Discarded reads with expected errs > 1.00\n",
      "     37182  Filtered reads (37.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 9.4Mb    26.1% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.8% passed \r\n",
      "     12869  Reads (12.9k)                   \n",
      "        22  Discarded reads with expected errs > 1.00\n",
      "     12847  Filtered reads (12.8k, 99.8%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 2.0Mb   100.0% Filtering, 99.9% passed\r\n",
      "     25698  Reads (25.7k)                   \n",
      "        17  Discarded reads with expected errs > 1.00\n",
      "     25681  Filtered reads (25.7k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:01 8.4Mb     0.2% Filtering, 100.0% passed\r",
      "00:01 1.9Mb   100.0% Filtering, 99.9% passed \r\n",
      "     11196  Reads (11.2k)                   \n",
      "        13  Discarded reads with expected errs > 1.00\n",
      "     11183  Filtered reads (11.2k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "     12622  Reads (12.6k)                   \n",
      "         7  Discarded reads with expected errs > 1.00\n",
      "     12615  Filtered reads (12.6k, 99.9%)\n",
      "00:00 1.8Mb     0.1% Filtering\r",
      "00:00 1.9Mb   100.0% Filtering, 99.9% passed\r\n",
      "      3485  Reads (3485)                    \n",
      "         3  Discarded reads with expected errs > 1.00\n",
      "      3482  Filtered reads (3482, 99.9%)\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "# Using \"bash magic\" - to run bash code in iPython notebook\n",
    "\n",
    "Seqs=../data/Seq_data/MergedSeqs/*.assembled.fastq\n",
    "Seqs=($Seqs)\n",
    "# Setting array of merged sequence files\n",
    "\n",
    "# For loop to run pear merge fwd and rev reads on all datasets.\n",
    "# Most of this code just pulls out the sample's name for each pair of seqeunce files.\n",
    "for ((i = 0; i < ${#Seqs[@]}; ++i)); do\n",
    "    filepath=${Seqs[i]}\n",
    "    # Take the full filepath for each sample (using read 1 arbitrarily)\n",
    "    splitfilepath=(${filepath//\\// })\n",
    "    # Split it up by /s\n",
    "    filename=${splitfilepath[${#splitfilepath[@]} - 1]}\n",
    "    # Take the last element from that filepath (i.e., the filename for read 1)\n",
    "    splitfilename=(${filename//_/ })\n",
    "    # Split the filename up using the underscore\n",
    "    outputname=${splitfilename[$1]}\n",
    "    # Take the first element of the split filename - i.e., everything before the first underscore - i.e., the sampleID\n",
    "    usearch9 -fastq_filter \"${Seqs[i]}\" -fastaout ../data/Seq_data/MergedSeqs/$outputname.maxee1 -fastq_maxee 1\n",
    "    # Removing sequences with more than 1 expected error using usearch\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of sequence pre-filter: 30556\n",
      "Number of sequences post-filter: 30556\n",
      "Number of sequence pre-filter: 30556\n",
      "Number of sequences post-filter: 30556\n",
      "Number of sequence pre-filter: 78876\n",
      "Number of sequences post-filter: 78876\n",
      "Number of sequence pre-filter: 78876\n",
      "Number of sequences post-filter: 78876\n",
      "Number of sequence pre-filter: 10220\n",
      "Number of sequences post-filter: 10220\n",
      "Number of sequence pre-filter: 10220\n",
      "Number of sequences post-filter: 10220\n",
      "Number of sequence pre-filter: 22510\n",
      "Number of sequences post-filter: 22510\n",
      "Number of sequence pre-filter: 22510\n",
      "Number of sequences post-filter: 22510\n",
      "Number of sequence pre-filter: 31149\n",
      "Number of sequences post-filter: 31149\n",
      "Number of sequence pre-filter: 31149\n",
      "Number of sequences post-filter: 31149\n",
      "Number of sequence pre-filter: 27229\n",
      "Number of sequences post-filter: 27229\n",
      "Number of sequence pre-filter: 27229\n",
      "Number of sequences post-filter: 27229\n",
      "Number of sequence pre-filter: 29054\n",
      "Number of sequences post-filter: 29054\n",
      "Number of sequence pre-filter: 29054\n",
      "Number of sequences post-filter: 29054\n",
      "Number of sequence pre-filter: 15261\n",
      "Number of sequences post-filter: 15261\n",
      "Number of sequence pre-filter: 15261\n",
      "Number of sequences post-filter: 15261\n",
      "Number of sequence pre-filter: 14027\n",
      "Number of sequences post-filter: 14027\n",
      "Number of sequence pre-filter: 14027\n",
      "Number of sequences post-filter: 14027\n",
      "Number of sequence pre-filter: 20092\n",
      "Number of sequences post-filter: 20092\n",
      "Number of sequence pre-filter: 20092\n",
      "Number of sequences post-filter: 20092\n",
      "Number of sequence pre-filter: 17421\n",
      "Number of sequences post-filter: 17421\n",
      "Number of sequence pre-filter: 17421\n",
      "Number of sequences post-filter: 17421\n",
      "Number of sequence pre-filter: 9657\n",
      "Number of sequences post-filter: 9657\n",
      "Number of sequence pre-filter: 9657\n",
      "Number of sequences post-filter: 9657\n",
      "Number of sequence pre-filter: 26068\n",
      "Number of sequences post-filter: 26068\n",
      "Number of sequence pre-filter: 26068\n",
      "Number of sequences post-filter: 26068\n",
      "Number of sequence pre-filter: 18805\n",
      "Number of sequences post-filter: 18805\n",
      "Number of sequence pre-filter: 18805\n",
      "Number of sequences post-filter: 18805\n",
      "Number of sequence pre-filter: 16828\n",
      "Number of sequences post-filter: 16828\n",
      "Number of sequence pre-filter: 16828\n",
      "Number of sequences post-filter: 16828\n",
      "Number of sequence pre-filter: 17974\n",
      "Number of sequences post-filter: 17974\n",
      "Number of sequence pre-filter: 17974\n",
      "Number of sequences post-filter: 17974\n",
      "Number of sequence pre-filter: 46334\n",
      "Number of sequences post-filter: 46334\n",
      "Number of sequence pre-filter: 46334\n",
      "Number of sequences post-filter: 46334\n",
      "Number of sequence pre-filter: 34464\n",
      "Number of sequences post-filter: 34464\n",
      "Number of sequence pre-filter: 34464\n",
      "Number of sequences post-filter: 34464\n",
      "Number of sequence pre-filter: 31036\n",
      "Number of sequences post-filter: 31036\n",
      "Number of sequence pre-filter: 31036\n",
      "Number of sequences post-filter: 31036\n",
      "Number of sequence pre-filter: 28453\n",
      "Number of sequences post-filter: 28453\n",
      "Number of sequence pre-filter: 28453\n",
      "Number of sequences post-filter: 28453\n",
      "Number of sequence pre-filter: 13168\n",
      "Number of sequences post-filter: 13168\n",
      "Number of sequence pre-filter: 13168\n",
      "Number of sequences post-filter: 13168\n",
      "Number of sequence pre-filter: 35100\n",
      "Number of sequences post-filter: 35100\n",
      "Number of sequence pre-filter: 35100\n",
      "Number of sequences post-filter: 35100\n",
      "Number of sequence pre-filter: 56178\n",
      "Number of sequences post-filter: 56178\n",
      "Number of sequence pre-filter: 56178\n",
      "Number of sequences post-filter: 56178\n",
      "Number of sequence pre-filter: 29694\n",
      "Number of sequences post-filter: 29694\n",
      "Number of sequence pre-filter: 29694\n",
      "Number of sequences post-filter: 29694\n",
      "Number of sequence pre-filter: 24490\n",
      "Number of sequences post-filter: 24490\n",
      "Number of sequence pre-filter: 24490\n",
      "Number of sequences post-filter: 24490\n",
      "Number of sequence pre-filter: 15723\n",
      "Number of sequences post-filter: 15723\n",
      "Number of sequence pre-filter: 15723\n",
      "Number of sequences post-filter: 15723\n",
      "Number of sequence pre-filter: 22067\n",
      "Number of sequences post-filter: 22067\n",
      "Number of sequence pre-filter: 22067\n",
      "Number of sequences post-filter: 22067\n",
      "Number of sequence pre-filter: 36478\n",
      "Number of sequences post-filter: 36478\n",
      "Number of sequence pre-filter: 36478\n",
      "Number of sequences post-filter: 36478\n",
      "Number of sequence pre-filter: 24380\n",
      "Number of sequences post-filter: 24380\n",
      "Number of sequence pre-filter: 24380\n",
      "Number of sequences post-filter: 24380\n",
      "Number of sequence pre-filter: 33792\n",
      "Number of sequences post-filter: 33792\n",
      "Number of sequence pre-filter: 33792\n",
      "Number of sequences post-filter: 33792\n",
      "Number of sequence pre-filter: 36934\n",
      "Number of sequences post-filter: 36934\n",
      "Number of sequence pre-filter: 36934\n",
      "Number of sequences post-filter: 36934\n",
      "Number of sequence pre-filter: 10964\n",
      "Number of sequences post-filter: 10964\n",
      "Number of sequence pre-filter: 10964\n",
      "Number of sequences post-filter: 10964\n",
      "Number of sequence pre-filter: 25916\n",
      "Number of sequences post-filter: 25916\n",
      "Number of sequence pre-filter: 25916\n",
      "Number of sequences post-filter: 25916\n",
      "Number of sequence pre-filter: 17553\n",
      "Number of sequences post-filter: 17553\n",
      "Number of sequence pre-filter: 17553\n",
      "Number of sequences post-filter: 17553\n",
      "Number of sequence pre-filter: 20501\n",
      "Number of sequences post-filter: 20501\n",
      "Number of sequence pre-filter: 20501\n",
      "Number of sequences post-filter: 20501\n",
      "Number of sequence pre-filter: 15379\n",
      "Number of sequences post-filter: 15379\n",
      "Number of sequence pre-filter: 15379\n",
      "Number of sequences post-filter: 15379\n",
      "Number of sequence pre-filter: 34241\n",
      "Number of sequences post-filter: 34241\n",
      "Number of sequence pre-filter: 34241\n",
      "Number of sequences post-filter: 34241\n",
      "Number of sequence pre-filter: 39757\n",
      "Number of sequences post-filter: 39757\n",
      "Number of sequence pre-filter: 39757\n",
      "Number of sequences post-filter: 39757\n",
      "Number of sequence pre-filter: 13960\n",
      "Number of sequences post-filter: 13960\n",
      "Number of sequence pre-filter: 13960\n",
      "Number of sequences post-filter: 13960\n",
      "Number of sequence pre-filter: 22543\n",
      "Number of sequences post-filter: 22543\n",
      "Number of sequence pre-filter: 22543\n",
      "Number of sequences post-filter: 22543\n",
      "Number of sequence pre-filter: 17218\n",
      "Number of sequences post-filter: 17218\n",
      "Number of sequence pre-filter: 17218\n",
      "Number of sequences post-filter: 17218\n",
      "Number of sequence pre-filter: 12986\n",
      "Number of sequences post-filter: 12986\n",
      "Number of sequence pre-filter: 12986\n",
      "Number of sequences post-filter: 12986\n",
      "Number of sequence pre-filter: 21319\n",
      "Number of sequences post-filter: 21319\n",
      "Number of sequence pre-filter: 21319\n",
      "Number of sequences post-filter: 21319\n",
      "Number of sequence pre-filter: 26080\n",
      "Number of sequences post-filter: 26080\n",
      "Number of sequence pre-filter: 26080\n",
      "Number of sequences post-filter: 26080\n",
      "Number of sequence pre-filter: 12548\n",
      "Number of sequences post-filter: 12548\n",
      "Number of sequence pre-filter: 12548\n",
      "Number of sequences post-filter: 12548\n",
      "Number of sequence pre-filter: 13365\n",
      "Number of sequences post-filter: 13365\n",
      "Number of sequence pre-filter: 13365\n",
      "Number of sequences post-filter: 13365\n",
      "Number of sequence pre-filter: 10712\n",
      "Number of sequences post-filter: 10712\n",
      "Number of sequence pre-filter: 10712\n",
      "Number of sequences post-filter: 10712\n",
      "Number of sequence pre-filter: 11878\n",
      "Number of sequences post-filter: 11878\n",
      "Number of sequence pre-filter: 11878\n",
      "Number of sequences post-filter: 11878\n",
      "Number of sequence pre-filter: 83775\n",
      "Number of sequences post-filter: 83775\n",
      "Number of sequence pre-filter: 83775\n",
      "Number of sequences post-filter: 83775\n",
      "Number of sequence pre-filter: 15078\n",
      "Number of sequences post-filter: 15078\n",
      "Number of sequence pre-filter: 15078\n",
      "Number of sequences post-filter: 15078\n",
      "Number of sequence pre-filter: 13256\n",
      "Number of sequences post-filter: 13256\n",
      "Number of sequence pre-filter: 13256\n",
      "Number of sequences post-filter: 13256\n",
      "Number of sequence pre-filter: 16819\n",
      "Number of sequences post-filter: 16819\n",
      "Number of sequence pre-filter: 16819\n",
      "Number of sequences post-filter: 16819\n",
      "Number of sequence pre-filter: 8725\n",
      "Number of sequences post-filter: 8725\n",
      "Number of sequence pre-filter: 8725\n",
      "Number of sequences post-filter: 8725\n",
      "Number of sequence pre-filter: 14176\n",
      "Number of sequences post-filter: 14176\n",
      "Number of sequence pre-filter: 14176\n",
      "Number of sequences post-filter: 14176\n",
      "Number of sequence pre-filter: 35887\n",
      "Number of sequences post-filter: 35887\n",
      "Number of sequence pre-filter: 35887\n",
      "Number of sequences post-filter: 35887\n",
      "Number of sequence pre-filter: 72433\n",
      "Number of sequences post-filter: 72433\n",
      "Number of sequence pre-filter: 72433\n",
      "Number of sequences post-filter: 72433\n",
      "Number of sequence pre-filter: 15339\n",
      "Number of sequences post-filter: 15339\n",
      "Number of sequence pre-filter: 15339\n",
      "Number of sequences post-filter: 15339\n",
      "Number of sequence pre-filter: 16885\n",
      "Number of sequences post-filter: 16885\n",
      "Number of sequence pre-filter: 16885\n",
      "Number of sequences post-filter: 16885\n",
      "Number of sequence pre-filter: 13320\n",
      "Number of sequences post-filter: 13320\n",
      "Number of sequence pre-filter: 13320\n",
      "Number of sequences post-filter: 13320\n",
      "Number of sequence pre-filter: 42856\n",
      "Number of sequences post-filter: 42856\n",
      "Number of sequence pre-filter: 42856\n",
      "Number of sequences post-filter: 42856\n",
      "Number of sequence pre-filter: 19054\n",
      "Number of sequences post-filter: 19054\n",
      "Number of sequence pre-filter: 19054\n",
      "Number of sequences post-filter: 19054\n",
      "Number of sequence pre-filter: 21336\n",
      "Number of sequences post-filter: 21336\n",
      "Number of sequence pre-filter: 21336\n",
      "Number of sequences post-filter: 21336\n",
      "Number of sequence pre-filter: 51562\n",
      "Number of sequences post-filter: 51562\n",
      "Number of sequence pre-filter: 51562\n",
      "Number of sequences post-filter: 51562\n",
      "Number of sequence pre-filter: 127152\n",
      "Number of sequences post-filter: 127152\n",
      "Number of sequence pre-filter: 127152\n",
      "Number of sequences post-filter: 127152\n",
      "Number of sequence pre-filter: 22056\n",
      "Number of sequences post-filter: 22056\n",
      "Number of sequence pre-filter: 22056\n",
      "Number of sequences post-filter: 22056\n",
      "Number of sequence pre-filter: 21294\n",
      "Number of sequences post-filter: 21294\n",
      "Number of sequence pre-filter: 21294\n",
      "Number of sequences post-filter: 21294\n",
      "Number of sequence pre-filter: 7192\n",
      "Number of sequences post-filter: 7192\n",
      "Number of sequence pre-filter: 7192\n",
      "Number of sequences post-filter: 7192\n",
      "Number of sequence pre-filter: 20617\n",
      "Number of sequences post-filter: 20617\n",
      "Number of sequence pre-filter: 20617\n",
      "Number of sequences post-filter: 20617\n",
      "Number of sequence pre-filter: 14111\n",
      "Number of sequences post-filter: 14111\n",
      "Number of sequence pre-filter: 14111\n",
      "Number of sequences post-filter: 14111\n",
      "Number of sequence pre-filter: 25130\n",
      "Number of sequences post-filter: 25130\n",
      "Number of sequence pre-filter: 25130\n",
      "Number of sequences post-filter: 25130\n",
      "Number of sequence pre-filter: 25415\n",
      "Number of sequences post-filter: 25415\n",
      "Number of sequence pre-filter: 25415\n",
      "Number of sequences post-filter: 25415\n",
      "Number of sequence pre-filter: 130206\n",
      "Number of sequences post-filter: 130206\n",
      "Number of sequence pre-filter: 130206\n",
      "Number of sequences post-filter: 130206\n",
      "Number of sequence pre-filter: 139557\n",
      "Number of sequences post-filter: 139557\n",
      "Number of sequence pre-filter: 139557\n",
      "Number of sequences post-filter: 139557\n",
      "Number of sequence pre-filter: 79615\n",
      "Number of sequences post-filter: 79615\n",
      "Number of sequence pre-filter: 79615\n",
      "Number of sequences post-filter: 79615\n",
      "Number of sequence pre-filter: 44436\n",
      "Number of sequences post-filter: 44436\n",
      "Number of sequence pre-filter: 44436\n",
      "Number of sequences post-filter: 44436\n",
      "Number of sequence pre-filter: 31095\n",
      "Number of sequences post-filter: 31095\n",
      "Number of sequence pre-filter: 31095\n",
      "Number of sequences post-filter: 31095\n",
      "Number of sequence pre-filter: 89224\n",
      "Number of sequences post-filter: 89224\n",
      "Number of sequence pre-filter: 89224\n",
      "Number of sequences post-filter: 89224\n",
      "Number of sequence pre-filter: 25601\n",
      "Number of sequences post-filter: 25601\n",
      "Number of sequence pre-filter: 25601\n",
      "Number of sequences post-filter: 25601\n",
      "Number of sequence pre-filter: 10556\n",
      "Number of sequences post-filter: 10556\n",
      "Number of sequence pre-filter: 10556\n",
      "Number of sequences post-filter: 10556\n",
      "Number of sequence pre-filter: 9155\n",
      "Number of sequences post-filter: 9155\n",
      "Number of sequence pre-filter: 9155\n",
      "Number of sequences post-filter: 9155\n",
      "Number of sequence pre-filter: 11253\n",
      "Number of sequences post-filter: 11253\n",
      "Number of sequence pre-filter: 11253\n",
      "Number of sequences post-filter: 11253\n",
      "Number of sequence pre-filter: 16073\n",
      "Number of sequences post-filter: 16073\n",
      "Number of sequence pre-filter: 16073\n",
      "Number of sequences post-filter: 16073\n",
      "Number of sequence pre-filter: 6820\n",
      "Number of sequences post-filter: 6820\n",
      "Number of sequence pre-filter: 6820\n",
      "Number of sequences post-filter: 6820\n",
      "Number of sequence pre-filter: 12551\n",
      "Number of sequences post-filter: 12551\n",
      "Number of sequence pre-filter: 12551\n",
      "Number of sequences post-filter: 12551\n",
      "Number of sequence pre-filter: 10879\n",
      "Number of sequences post-filter: 10879\n",
      "Number of sequence pre-filter: 10879\n",
      "Number of sequences post-filter: 10879\n",
      "Number of sequence pre-filter: 32198\n",
      "Number of sequences post-filter: 32198\n",
      "Number of sequence pre-filter: 32198\n",
      "Number of sequences post-filter: 32198\n",
      "Number of sequence pre-filter: 19551\n",
      "Number of sequences post-filter: 19551\n",
      "Number of sequence pre-filter: 19551\n",
      "Number of sequences post-filter: 19551\n",
      "Number of sequence pre-filter: 33280\n",
      "Number of sequences post-filter: 33280\n",
      "Number of sequence pre-filter: 33280\n",
      "Number of sequences post-filter: 33280\n",
      "Number of sequence pre-filter: 37384\n",
      "Number of sequences post-filter: 37384\n",
      "Number of sequence pre-filter: 37384\n",
      "Number of sequences post-filter: 37384\n",
      "Number of sequence pre-filter: 18068\n",
      "Number of sequences post-filter: 18068\n",
      "Number of sequence pre-filter: 18068\n",
      "Number of sequences post-filter: 18068\n",
      "Number of sequence pre-filter: 30104\n",
      "Number of sequences post-filter: 30104\n",
      "Number of sequence pre-filter: 30104\n",
      "Number of sequences post-filter: 30104\n",
      "Number of sequence pre-filter: 12939\n",
      "Number of sequences post-filter: 12939\n",
      "Number of sequence pre-filter: 12939\n",
      "Number of sequences post-filter: 12939\n",
      "Number of sequence pre-filter: 17894\n",
      "Number of sequences post-filter: 17894\n",
      "Number of sequence pre-filter: 17894\n",
      "Number of sequences post-filter: 17894\n",
      "Number of sequence pre-filter: 12626\n",
      "Number of sequences post-filter: 12626\n",
      "Number of sequence pre-filter: 12626\n",
      "Number of sequences post-filter: 12626\n",
      "Number of sequence pre-filter: 22166\n",
      "Number of sequences post-filter: 22166\n",
      "Number of sequence pre-filter: 22166\n",
      "Number of sequences post-filter: 22166\n",
      "Number of sequence pre-filter: 22892\n",
      "Number of sequences post-filter: 22892\n",
      "Number of sequence pre-filter: 22892\n",
      "Number of sequences post-filter: 22892\n",
      "Number of sequence pre-filter: 20895\n",
      "Number of sequences post-filter: 20895\n",
      "Number of sequence pre-filter: 20895\n",
      "Number of sequences post-filter: 20895\n",
      "Number of sequence pre-filter: 13362\n",
      "Number of sequences post-filter: 13362\n",
      "Number of sequence pre-filter: 13362\n",
      "Number of sequences post-filter: 13362\n",
      "Number of sequence pre-filter: 8577\n",
      "Number of sequences post-filter: 8577\n",
      "Number of sequence pre-filter: 8577\n",
      "Number of sequences post-filter: 8577\n",
      "Number of sequence pre-filter: 14553\n",
      "Number of sequences post-filter: 14553\n",
      "Number of sequence pre-filter: 14553\n",
      "Number of sequences post-filter: 14553\n",
      "Number of sequence pre-filter: 17021\n",
      "Number of sequences post-filter: 17021\n",
      "Number of sequence pre-filter: 17021\n",
      "Number of sequences post-filter: 17021\n",
      "Number of sequence pre-filter: 58614\n",
      "Number of sequences post-filter: 58614\n",
      "Number of sequence pre-filter: 58614\n",
      "Number of sequences post-filter: 58614\n",
      "Number of sequence pre-filter: 9437\n",
      "Number of sequences post-filter: 9437\n",
      "Number of sequence pre-filter: 9437\n",
      "Number of sequences post-filter: 9437\n",
      "Number of sequence pre-filter: 4936\n",
      "Number of sequences post-filter: 4936\n",
      "Number of sequence pre-filter: 4936\n",
      "Number of sequences post-filter: 4936\n",
      "Number of sequence pre-filter: 14169\n",
      "Number of sequences post-filter: 14169\n",
      "Number of sequence pre-filter: 14169\n",
      "Number of sequences post-filter: 14169\n",
      "Number of sequence pre-filter: 9562\n",
      "Number of sequences post-filter: 9562\n",
      "Number of sequence pre-filter: 9562\n",
      "Number of sequences post-filter: 9562\n",
      "Number of sequence pre-filter: 10700\n",
      "Number of sequences post-filter: 10700\n",
      "Number of sequence pre-filter: 10700\n",
      "Number of sequences post-filter: 10700\n",
      "Number of sequence pre-filter: 11302\n",
      "Number of sequences post-filter: 11302\n",
      "Number of sequence pre-filter: 11302\n",
      "Number of sequences post-filter: 11302\n",
      "Number of sequence pre-filter: 9427\n",
      "Number of sequences post-filter: 9427\n",
      "Number of sequence pre-filter: 9427\n",
      "Number of sequences post-filter: 9427\n",
      "Number of sequence pre-filter: 22857\n",
      "Number of sequences post-filter: 22857\n",
      "Number of sequence pre-filter: 22857\n",
      "Number of sequences post-filter: 22857\n",
      "Number of sequence pre-filter: 5769\n",
      "Number of sequences post-filter: 5769\n",
      "Number of sequence pre-filter: 5769\n",
      "Number of sequences post-filter: 5769\n",
      "Number of sequence pre-filter: 16298\n",
      "Number of sequences post-filter: 16298\n",
      "Number of sequence pre-filter: 16298\n",
      "Number of sequences post-filter: 16298\n",
      "Number of sequence pre-filter: 16459\n",
      "Number of sequences post-filter: 16459\n",
      "Number of sequence pre-filter: 16459\n",
      "Number of sequences post-filter: 16459\n",
      "Number of sequence pre-filter: 14199\n",
      "Number of sequences post-filter: 14199\n",
      "Number of sequence pre-filter: 14199\n",
      "Number of sequences post-filter: 14199\n",
      "Number of sequence pre-filter: 17013\n",
      "Number of sequences post-filter: 17013\n",
      "Number of sequence pre-filter: 17013\n",
      "Number of sequences post-filter: 17013\n",
      "Number of sequence pre-filter: 21323\n",
      "Number of sequences post-filter: 21323\n",
      "Number of sequence pre-filter: 21323\n",
      "Number of sequences post-filter: 21323\n",
      "Number of sequence pre-filter: 9776\n",
      "Number of sequences post-filter: 9776\n",
      "Number of sequence pre-filter: 9776\n",
      "Number of sequences post-filter: 9776\n",
      "Number of sequence pre-filter: 23353\n",
      "Number of sequences post-filter: 23353\n",
      "Number of sequence pre-filter: 23353\n",
      "Number of sequences post-filter: 23353\n",
      "Number of sequence pre-filter: 8849\n",
      "Number of sequences post-filter: 8849\n",
      "Number of sequence pre-filter: 8849\n",
      "Number of sequences post-filter: 8849\n",
      "Number of sequence pre-filter: 15889\n",
      "Number of sequences post-filter: 15889\n",
      "Number of sequence pre-filter: 15889\n",
      "Number of sequences post-filter: 15889\n",
      "Number of sequence pre-filter: 18779\n",
      "Number of sequences post-filter: 18779\n",
      "Number of sequence pre-filter: 18779\n",
      "Number of sequences post-filter: 18779\n",
      "Number of sequence pre-filter: 24417\n",
      "Number of sequences post-filter: 24417\n",
      "Number of sequence pre-filter: 24417\n",
      "Number of sequences post-filter: 24417\n",
      "Number of sequence pre-filter: 15570\n",
      "Number of sequences post-filter: 15570\n",
      "Number of sequence pre-filter: 15570\n",
      "Number of sequences post-filter: 15570\n",
      "Number of sequence pre-filter: 16864\n",
      "Number of sequences post-filter: 16864\n",
      "Number of sequence pre-filter: 16864\n",
      "Number of sequences post-filter: 16864\n",
      "Number of sequence pre-filter: 34153\n",
      "Number of sequences post-filter: 34153\n",
      "Number of sequence pre-filter: 34153\n",
      "Number of sequences post-filter: 34153\n",
      "Number of sequence pre-filter: 35041\n",
      "Number of sequences post-filter: 35041\n",
      "Number of sequence pre-filter: 35041\n",
      "Number of sequences post-filter: 35041\n",
      "Number of sequence pre-filter: 31907\n",
      "Number of sequences post-filter: 31907\n",
      "Number of sequence pre-filter: 31907\n",
      "Number of sequences post-filter: 31907\n",
      "Number of sequence pre-filter: 18144\n",
      "Number of sequences post-filter: 18144\n",
      "Number of sequence pre-filter: 18144\n",
      "Number of sequences post-filter: 18144\n",
      "Number of sequence pre-filter: 21606\n",
      "Number of sequences post-filter: 21606\n",
      "Number of sequence pre-filter: 21606\n",
      "Number of sequences post-filter: 21606\n",
      "Number of sequence pre-filter: 13742\n",
      "Number of sequences post-filter: 13742\n",
      "Number of sequence pre-filter: 13742\n",
      "Number of sequences post-filter: 13742\n",
      "Number of sequence pre-filter: 33272\n",
      "Number of sequences post-filter: 33272\n",
      "Number of sequence pre-filter: 33272\n",
      "Number of sequences post-filter: 33272\n",
      "Number of sequence pre-filter: 23340\n",
      "Number of sequences post-filter: 23340\n",
      "Number of sequence pre-filter: 23340\n",
      "Number of sequences post-filter: 23340\n",
      "Number of sequence pre-filter: 27153\n",
      "Number of sequences post-filter: 27153\n",
      "Number of sequence pre-filter: 27153\n",
      "Number of sequences post-filter: 27153\n",
      "Number of sequence pre-filter: 22327\n",
      "Number of sequences post-filter: 22327\n",
      "Number of sequence pre-filter: 22327\n",
      "Number of sequences post-filter: 22327\n",
      "Number of sequence pre-filter: 72929\n",
      "Number of sequences post-filter: 72929\n",
      "Number of sequence pre-filter: 72929\n",
      "Number of sequences post-filter: 72929\n",
      "Number of sequence pre-filter: 41460\n",
      "Number of sequences post-filter: 41460\n",
      "Number of sequence pre-filter: 41460\n",
      "Number of sequences post-filter: 41460\n",
      "Number of sequence pre-filter: 37140\n",
      "Number of sequences post-filter: 37140\n",
      "Number of sequence pre-filter: 37140\n",
      "Number of sequences post-filter: 37140\n",
      "Number of sequence pre-filter: 34289\n",
      "Number of sequences post-filter: 34289\n",
      "Number of sequence pre-filter: 34289\n",
      "Number of sequences post-filter: 34289\n",
      "Number of sequence pre-filter: 83225\n",
      "Number of sequences post-filter: 83225\n",
      "Number of sequence pre-filter: 83225\n",
      "Number of sequences post-filter: 83225\n",
      "Number of sequence pre-filter: 55\n",
      "Number of sequences post-filter: 55\n",
      "Number of sequence pre-filter: 55\n",
      "Number of sequences post-filter: 55\n",
      "Number of sequence pre-filter: 62607\n",
      "Number of sequences post-filter: 62607\n",
      "Number of sequence pre-filter: 62607\n",
      "Number of sequences post-filter: 62607\n",
      "Number of sequence pre-filter: 22\n",
      "Number of sequences post-filter: 22\n",
      "Number of sequence pre-filter: 22\n",
      "Number of sequences post-filter: 22\n",
      "Number of sequence pre-filter: 53730\n",
      "Number of sequences post-filter: 53730\n",
      "Number of sequence pre-filter: 53730\n",
      "Number of sequences post-filter: 53730\n",
      "Number of sequence pre-filter: 45061\n",
      "Number of sequences post-filter: 45061\n",
      "Number of sequence pre-filter: 45061\n",
      "Number of sequences post-filter: 45061\n",
      "Number of sequence pre-filter: 38358\n",
      "Number of sequences post-filter: 38358\n",
      "Number of sequence pre-filter: 38358\n",
      "Number of sequences post-filter: 38358\n",
      "Number of sequence pre-filter: 29682\n",
      "Number of sequences post-filter: 29682\n",
      "Number of sequence pre-filter: 29682\n",
      "Number of sequences post-filter: 29682\n",
      "Number of sequence pre-filter: 22320\n",
      "Number of sequences post-filter: 22320\n",
      "Number of sequence pre-filter: 22320\n",
      "Number of sequences post-filter: 22320\n",
      "Number of sequence pre-filter: 50748\n",
      "Number of sequences post-filter: 50748\n",
      "Number of sequence pre-filter: 50748\n",
      "Number of sequences post-filter: 50748\n",
      "Number of sequence pre-filter: 29\n",
      "Number of sequences post-filter: 29\n",
      "Number of sequence pre-filter: 29\n",
      "Number of sequences post-filter: 29\n",
      "Number of sequence pre-filter: 141330\n",
      "Number of sequences post-filter: 141330\n",
      "Number of sequence pre-filter: 141330\n",
      "Number of sequences post-filter: 141330\n",
      "Number of sequence pre-filter: 20\n",
      "Number of sequences post-filter: 20\n",
      "Number of sequence pre-filter: 20\n",
      "Number of sequences post-filter: 20\n",
      "Number of sequence pre-filter: 43550\n",
      "Number of sequences post-filter: 43550\n",
      "Number of sequence pre-filter: 43550\n",
      "Number of sequences post-filter: 43550\n",
      "Number of sequence pre-filter: 33568\n",
      "Number of sequences post-filter: 33568\n",
      "Number of sequence pre-filter: 33568\n",
      "Number of sequences post-filter: 33568\n",
      "Number of sequence pre-filter: 69740\n",
      "Number of sequences post-filter: 69740\n",
      "Number of sequence pre-filter: 69740\n",
      "Number of sequences post-filter: 69740\n",
      "Number of sequence pre-filter: 8071\n",
      "Number of sequences post-filter: 8071\n",
      "Number of sequence pre-filter: 8071\n",
      "Number of sequences post-filter: 8071\n",
      "Number of sequence pre-filter: 35159\n",
      "Number of sequences post-filter: 35159\n",
      "Number of sequence pre-filter: 35159\n",
      "Number of sequences post-filter: 35159\n",
      "Number of sequence pre-filter: 27700\n",
      "Number of sequences post-filter: 27700\n",
      "Number of sequence pre-filter: 27700\n",
      "Number of sequences post-filter: 27700\n",
      "Number of sequence pre-filter: 20016\n",
      "Number of sequences post-filter: 20016\n",
      "Number of sequence pre-filter: 20016\n",
      "Number of sequences post-filter: 20016\n",
      "Number of sequence pre-filter: 37684\n",
      "Number of sequences post-filter: 37684\n",
      "Number of sequence pre-filter: 37684\n",
      "Number of sequences post-filter: 37684\n",
      "Number of sequence pre-filter: 47766\n",
      "Number of sequences post-filter: 47766\n",
      "Number of sequence pre-filter: 47766\n",
      "Number of sequences post-filter: 47766\n",
      "Number of sequence pre-filter: 18333\n",
      "Number of sequences post-filter: 18333\n",
      "Number of sequence pre-filter: 18333\n",
      "Number of sequences post-filter: 18333\n",
      "Number of sequence pre-filter: 21044\n",
      "Number of sequences post-filter: 21044\n",
      "Number of sequence pre-filter: 21044\n",
      "Number of sequences post-filter: 21044\n",
      "Number of sequence pre-filter: 71810\n",
      "Number of sequences post-filter: 71810\n",
      "Number of sequence pre-filter: 71810\n",
      "Number of sequences post-filter: 71810\n",
      "Number of sequence pre-filter: 58275\n",
      "Number of sequences post-filter: 58275\n",
      "Number of sequence pre-filter: 58275\n",
      "Number of sequences post-filter: 58275\n",
      "Number of sequence pre-filter: 25067\n",
      "Number of sequences post-filter: 25067\n",
      "Number of sequence pre-filter: 25067\n",
      "Number of sequences post-filter: 25067\n",
      "Number of sequence pre-filter: 26204\n",
      "Number of sequences post-filter: 26204\n",
      "Number of sequence pre-filter: 26204\n",
      "Number of sequences post-filter: 26204\n",
      "Number of sequence pre-filter: 91243\n",
      "Number of sequences post-filter: 91243\n",
      "Number of sequence pre-filter: 91243\n",
      "Number of sequences post-filter: 91243\n",
      "Number of sequence pre-filter: 38267\n",
      "Number of sequences post-filter: 38267\n",
      "Number of sequence pre-filter: 38267\n",
      "Number of sequences post-filter: 38267\n",
      "Number of sequence pre-filter: 109938\n",
      "Number of sequences post-filter: 109938\n",
      "Number of sequence pre-filter: 109938\n",
      "Number of sequences post-filter: 109938\n",
      "Number of sequence pre-filter: 23508\n",
      "Number of sequences post-filter: 23508\n",
      "Number of sequence pre-filter: 23508\n",
      "Number of sequences post-filter: 23508\n",
      "Number of sequence pre-filter: 24602\n",
      "Number of sequences post-filter: 24602\n",
      "Number of sequence pre-filter: 24602\n",
      "Number of sequences post-filter: 24602\n",
      "Number of sequence pre-filter: 19850\n",
      "Number of sequences post-filter: 19850\n",
      "Number of sequence pre-filter: 19850\n",
      "Number of sequences post-filter: 19850\n",
      "Number of sequence pre-filter: 32845\n",
      "Number of sequences post-filter: 32845\n",
      "Number of sequence pre-filter: 32845\n",
      "Number of sequences post-filter: 32845\n",
      "Number of sequence pre-filter: 22830\n",
      "Number of sequences post-filter: 22830\n",
      "Number of sequence pre-filter: 22830\n",
      "Number of sequences post-filter: 22830\n",
      "Number of sequence pre-filter: 21692\n",
      "Number of sequences post-filter: 21692\n",
      "Number of sequence pre-filter: 21692\n",
      "Number of sequences post-filter: 21692\n",
      "Number of sequence pre-filter: 37823\n",
      "Number of sequences post-filter: 37823\n",
      "Number of sequence pre-filter: 37823\n",
      "Number of sequences post-filter: 37823\n",
      "Number of sequence pre-filter: 50267\n",
      "Number of sequences post-filter: 50267\n",
      "Number of sequence pre-filter: 50267\n",
      "Number of sequences post-filter: 50267\n",
      "Number of sequence pre-filter: 27488\n",
      "Number of sequences post-filter: 27488\n",
      "Number of sequence pre-filter: 27488\n",
      "Number of sequences post-filter: 27488\n",
      "Number of sequence pre-filter: 34798\n",
      "Number of sequences post-filter: 34798\n",
      "Number of sequence pre-filter: 34798\n",
      "Number of sequences post-filter: 34798\n",
      "Number of sequence pre-filter: 40575\n",
      "Number of sequences post-filter: 40575\n",
      "Number of sequence pre-filter: 40575\n",
      "Number of sequences post-filter: 40575\n",
      "Number of sequence pre-filter: 123592\n",
      "Number of sequences post-filter: 123592\n",
      "Number of sequence pre-filter: 123592\n",
      "Number of sequences post-filter: 123592\n",
      "Number of sequence pre-filter: 51217\n",
      "Number of sequences post-filter: 51217\n",
      "Number of sequence pre-filter: 51217\n",
      "Number of sequences post-filter: 51217\n",
      "Number of sequence pre-filter: 38736\n",
      "Number of sequences post-filter: 38736\n",
      "Number of sequence pre-filter: 38736\n",
      "Number of sequences post-filter: 38736\n",
      "Number of sequence pre-filter: 18905\n",
      "Number of sequences post-filter: 18905\n",
      "Number of sequence pre-filter: 18905\n",
      "Number of sequences post-filter: 18905\n",
      "Number of sequence pre-filter: 80499\n",
      "Number of sequences post-filter: 80499\n",
      "Number of sequence pre-filter: 80499\n",
      "Number of sequences post-filter: 80499\n",
      "Number of sequence pre-filter: 41732\n",
      "Number of sequences post-filter: 41732\n",
      "Number of sequence pre-filter: 41732\n",
      "Number of sequences post-filter: 41732\n",
      "Number of sequence pre-filter: 37683\n",
      "Number of sequences post-filter: 37683\n",
      "Number of sequence pre-filter: 37683\n",
      "Number of sequences post-filter: 37683\n",
      "Number of sequence pre-filter: 35237\n",
      "Number of sequences post-filter: 35237\n",
      "Number of sequence pre-filter: 35237\n",
      "Number of sequences post-filter: 35237\n",
      "Number of sequence pre-filter: 50160\n",
      "Number of sequences post-filter: 50160\n",
      "Number of sequence pre-filter: 50160\n",
      "Number of sequences post-filter: 50160\n",
      "Number of sequence pre-filter: 65205\n",
      "Number of sequences post-filter: 65205\n",
      "Number of sequence pre-filter: 65205\n",
      "Number of sequences post-filter: 65205\n",
      "Number of sequence pre-filter: 19102\n",
      "Number of sequences post-filter: 19102\n",
      "Number of sequence pre-filter: 19102\n",
      "Number of sequences post-filter: 19102\n",
      "Number of sequence pre-filter: 12275\n",
      "Number of sequences post-filter: 12275\n",
      "Number of sequence pre-filter: 12275\n",
      "Number of sequences post-filter: 12275\n",
      "Number of sequence pre-filter: 17808\n",
      "Number of sequences post-filter: 17808\n",
      "Number of sequence pre-filter: 17808\n",
      "Number of sequences post-filter: 17808\n",
      "Number of sequence pre-filter: 10707\n",
      "Number of sequences post-filter: 10707\n",
      "Number of sequence pre-filter: 10707\n",
      "Number of sequences post-filter: 10707\n",
      "Number of sequence pre-filter: 12061\n",
      "Number of sequences post-filter: 12061\n",
      "Number of sequence pre-filter: 12061\n",
      "Number of sequences post-filter: 12061\n",
      "Number of sequence pre-filter: 11946\n",
      "Number of sequences post-filter: 11946\n",
      "Number of sequence pre-filter: 11946\n",
      "Number of sequences post-filter: 11946\n",
      "Number of sequence pre-filter: 11793\n",
      "Number of sequences post-filter: 11793\n",
      "Number of sequence pre-filter: 11793\n",
      "Number of sequences post-filter: 11793\n",
      "Number of sequence pre-filter: 60\n",
      "Number of sequences post-filter: 60\n",
      "Number of sequence pre-filter: 60\n",
      "Number of sequences post-filter: 60\n",
      "Number of sequence pre-filter: 4678\n",
      "Number of sequences post-filter: 4678\n",
      "Number of sequence pre-filter: 4678\n",
      "Number of sequences post-filter: 4678\n",
      "Number of sequence pre-filter: 11919\n",
      "Number of sequences post-filter: 11919\n",
      "Number of sequence pre-filter: 11919\n",
      "Number of sequences post-filter: 11919\n",
      "Number of sequence pre-filter: 9009\n",
      "Number of sequences post-filter: 9009\n",
      "Number of sequence pre-filter: 9009\n",
      "Number of sequences post-filter: 9009\n",
      "Number of sequence pre-filter: 8573\n",
      "Number of sequences post-filter: 8573\n",
      "Number of sequence pre-filter: 8573\n",
      "Number of sequences post-filter: 8573\n",
      "Number of sequence pre-filter: 13299\n",
      "Number of sequences post-filter: 13299\n",
      "Number of sequence pre-filter: 13299\n",
      "Number of sequences post-filter: 13299\n",
      "Number of sequence pre-filter: 10817\n",
      "Number of sequences post-filter: 10817\n",
      "Number of sequence pre-filter: 10817\n",
      "Number of sequences post-filter: 10817\n",
      "Number of sequence pre-filter: 31181\n",
      "Number of sequences post-filter: 31181\n",
      "Number of sequence pre-filter: 31181\n",
      "Number of sequences post-filter: 31181\n",
      "Number of sequence pre-filter: 10698\n",
      "Number of sequences post-filter: 10698\n",
      "Number of sequence pre-filter: 10698\n",
      "Number of sequences post-filter: 10698\n",
      "Number of sequence pre-filter: 13494\n",
      "Number of sequences post-filter: 13494\n",
      "Number of sequence pre-filter: 13494\n",
      "Number of sequences post-filter: 13494\n",
      "Number of sequence pre-filter: 6180\n",
      "Number of sequences post-filter: 6180\n",
      "Number of sequence pre-filter: 6180\n",
      "Number of sequences post-filter: 6180\n",
      "Number of sequence pre-filter: 24\n",
      "Number of sequences post-filter: 24\n",
      "Number of sequence pre-filter: 24\n",
      "Number of sequences post-filter: 24\n",
      "Number of sequence pre-filter: 285\n",
      "Number of sequences post-filter: 285\n",
      "Number of sequence pre-filter: 285\n",
      "Number of sequences post-filter: 285\n",
      "Number of sequence pre-filter: 533\n",
      "Number of sequences post-filter: 533\n",
      "Number of sequence pre-filter: 533\n",
      "Number of sequences post-filter: 533\n",
      "Number of sequence pre-filter: 243\n",
      "Number of sequences post-filter: 243\n",
      "Number of sequence pre-filter: 243\n",
      "Number of sequences post-filter: 243\n",
      "Number of sequence pre-filter: 1061\n",
      "Number of sequences post-filter: 1061\n",
      "Number of sequence pre-filter: 1061\n",
      "Number of sequences post-filter: 1061\n",
      "Number of sequence pre-filter: 190\n",
      "Number of sequences post-filter: 190\n",
      "Number of sequence pre-filter: 190\n",
      "Number of sequences post-filter: 190\n",
      "Number of sequence pre-filter: 114\n",
      "Number of sequences post-filter: 114\n",
      "Number of sequence pre-filter: 114\n",
      "Number of sequences post-filter: 114\n",
      "Number of sequence pre-filter: 358\n",
      "Number of sequences post-filter: 358\n",
      "Number of sequence pre-filter: 358\n",
      "Number of sequences post-filter: 358\n",
      "Number of sequence pre-filter: 26930\n",
      "Number of sequences post-filter: 26930\n",
      "Number of sequence pre-filter: 26930\n",
      "Number of sequences post-filter: 26930\n",
      "Number of sequence pre-filter: 1067\n",
      "Number of sequences post-filter: 1067\n",
      "Number of sequence pre-filter: 1067\n",
      "Number of sequences post-filter: 1067\n",
      "Number of sequence pre-filter: 39646\n",
      "Number of sequences post-filter: 39646\n",
      "Number of sequence pre-filter: 39646\n",
      "Number of sequences post-filter: 39646\n",
      "Number of sequence pre-filter: 18386\n",
      "Number of sequences post-filter: 18386\n",
      "Number of sequence pre-filter: 18386\n",
      "Number of sequences post-filter: 18386\n",
      "Number of sequence pre-filter: 25203\n",
      "Number of sequences post-filter: 25203\n",
      "Number of sequence pre-filter: 25203\n",
      "Number of sequences post-filter: 25203\n",
      "Number of sequence pre-filter: 20923\n",
      "Number of sequences post-filter: 20923\n",
      "Number of sequence pre-filter: 20923\n",
      "Number of sequences post-filter: 20923\n",
      "Number of sequence pre-filter: 11048\n",
      "Number of sequences post-filter: 11048\n",
      "Number of sequence pre-filter: 11048\n",
      "Number of sequences post-filter: 11048\n",
      "Number of sequence pre-filter: 30474\n",
      "Number of sequences post-filter: 30474\n",
      "Number of sequence pre-filter: 30474\n",
      "Number of sequences post-filter: 30474\n",
      "Number of sequence pre-filter: 26105\n",
      "Number of sequences post-filter: 26105\n",
      "Number of sequence pre-filter: 26105\n",
      "Number of sequences post-filter: 26105\n",
      "Number of sequence pre-filter: 19901\n",
      "Number of sequences post-filter: 19901\n",
      "Number of sequence pre-filter: 19901\n",
      "Number of sequences post-filter: 19901\n",
      "Number of sequence pre-filter: 40998\n",
      "Number of sequences post-filter: 40998\n",
      "Number of sequence pre-filter: 40998\n",
      "Number of sequences post-filter: 40998\n",
      "Number of sequence pre-filter: 16452\n",
      "Number of sequences post-filter: 16452\n",
      "Number of sequence pre-filter: 16452\n",
      "Number of sequences post-filter: 16452\n",
      "Number of sequence pre-filter: 18224\n",
      "Number of sequences post-filter: 18224\n",
      "Number of sequence pre-filter: 18224\n",
      "Number of sequences post-filter: 18224\n",
      "Number of sequence pre-filter: 13703\n",
      "Number of sequences post-filter: 13703\n",
      "Number of sequence pre-filter: 13703\n",
      "Number of sequences post-filter: 13703\n",
      "Number of sequence pre-filter: 14619\n",
      "Number of sequences post-filter: 14619\n",
      "Number of sequence pre-filter: 14619\n",
      "Number of sequences post-filter: 14619\n",
      "Number of sequence pre-filter: 22538\n",
      "Number of sequences post-filter: 22538\n",
      "Number of sequence pre-filter: 22538\n",
      "Number of sequences post-filter: 22538\n",
      "Number of sequence pre-filter: 12958\n",
      "Number of sequences post-filter: 12958\n",
      "Number of sequence pre-filter: 12958\n",
      "Number of sequences post-filter: 12958\n",
      "Number of sequence pre-filter: 22827\n",
      "Number of sequences post-filter: 22827\n",
      "Number of sequence pre-filter: 22827\n",
      "Number of sequences post-filter: 22827\n",
      "Number of sequence pre-filter: 37182\n",
      "Number of sequences post-filter: 37182\n",
      "Number of sequence pre-filter: 37182\n",
      "Number of sequences post-filter: 37182\n",
      "Number of sequence pre-filter: 12847\n",
      "Number of sequences post-filter: 12847\n",
      "Number of sequence pre-filter: 12847\n",
      "Number of sequences post-filter: 12847\n",
      "Number of sequence pre-filter: 25681\n",
      "Number of sequences post-filter: 25681\n",
      "Number of sequence pre-filter: 25681\n",
      "Number of sequences post-filter: 25681\n",
      "Number of sequence pre-filter: 11183\n",
      "Number of sequences post-filter: 11183\n",
      "Number of sequence pre-filter: 11183\n",
      "Number of sequences post-filter: 11183\n",
      "Number of sequence pre-filter: 12615\n",
      "Number of sequences post-filter: 12615\n",
      "Number of sequence pre-filter: 12615\n",
      "Number of sequences post-filter: 12615\n",
      "Number of sequence pre-filter: 3482\n",
      "Number of sequences post-filter: 3482\n",
      "Number of sequence pre-filter: 3482\n",
      "Number of sequences post-filter: 3482\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "# Using \"bash magic\" - to run bash code in iPython notebook\n",
    "\n",
    "Seqs=../data/Seq_data/MergedSeqs/*maxee1*\n",
    "Seqs=($Seqs)\n",
    "# Setting array of merged and error-filtered files\n",
    "\n",
    "# For loop to run pear merge fwd and rev reads on all datasets.\n",
    "# Most of this code just pulls out the sample's name for each pair of seqeunce files.\n",
    "for ((i = 0; i < ${#Seqs[@]}; ++i)); do\n",
    "    filepath=${Seqs[i]}\n",
    "    # Take the full filepath for each sample (using read 1 arbitrarily)\n",
    "    splitfilepath=(${filepath//\\// })\n",
    "    # Split it up by /s\n",
    "    filename=${splitfilepath[${#splitfilepath[@]} - 1]}\n",
    "    # Take the last element from that filepath (i.e., the filename for read 1)\n",
    "    splitfilename=(${filename//_/ })\n",
    "    # Split the filename up using the underscore\n",
    "    outputname=${splitfilename[$1]}\n",
    "    # Take the first element of the split filename - i.e., everything before the first underscore - i.e., the sampleID\n",
    "    bioawk -c fastx '{if ($seq !~ /N/){print \">\" $name \" \" $4 \"\\n\" $seq}}' ${Seqs[i]} > ../data/Seq_data/MergedSeqs/$outputname.noN.fasta\n",
    "    # Copying only the sequences that contain no N characters\n",
    "    printf \"Number of sequence pre-filter: \"\n",
    "    grep -c \">\" ${Seqs[i]}\n",
    "    printf \"Number of sequences post-filter: \"\n",
    "    grep -c \">\" ../data/Seq_data/MergedSeqs/$outputname.noN.fasta\n",
    "    # Evaluate and report the number of sequences (indicated by >) in each filtered fasta file\n",
    "done\n",
    "#  Basically there weren't really many / any Ns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir ../data/Seq_data/QCSeqs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat ../data/Seq_data/MergedSeqs/*.assembled.fastq.maxee1.noN.fasta > ../data/Seq_data/QCSeqs/QCseqs.fasta\n",
    "# make one giant file with all our sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "6550268\r\n"
     ]
    }
   ],
   "source": [
    "! grep -c \">\" ../data/Seq_data/QCSeqs/QCseqs.fasta\n",
    "# Count total QC'd sequences so far (6.5M)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Using mothur for more filter and alignmnent based QC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!sed -e 's/ /_/g' ../data/Seq_data/QCSeqs/QCseqs.fasta > ../data/Seq_data/QCSeqs/QCseqs.sed.fasta\n",
    "# Keeping the whole samplename without spaces"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">HWI-M04026:146:000000000-B58PG:1:1101:13055:1749_1:N:0:TGCTCGTAGATCGTGT\r\n",
      "TACGTAGGGGGCAAGCGTTGTCCGGAATCATTGGGCGTAAAGAGCGTGTAGGCGGCCCGATAAGTCCGCTGTGAAAGTCCAGGGCTCAACTCTGGAATGCCGGTGGATACTGTCGGGCTCGAATCCGGAAGAGGTGAGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCAATGGCGAAGGCAGCTCACTGGGACGGTATTGACGCTGAGACGCGAAAGCGTGGGGAGCAAACAGG\r\n",
      ">HWI-M04026:146:000000000-B58PG:1:1101:17688:1769_1:N:0:TGCTCGTAGATCGTGT\r\n",
      "TACGTAGGGTGCGAGCGTTGTCCGGAATTACTGGGCGTAAAGAGCTCGTAGGTGGTTTGTCGCGTTGTTCGTGAAAACCGGAGGCTCAACCTCTGGCGTGCGGGCGATACGGGCAGACTTGAGTACTGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGGAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCAGTAACTGACGCTGAGGAGCGAAAGCGTGGGGAGCGAACAGG\r\n",
      ">HWI-M04026:146:000000000-B58PG:1:1101:19384:1834_1:N:0:TGCTCGTAGATCGTGT\r\n",
      "TACGTAGGGCGCAAGCGTTATCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTTTGTCGCGTCTGCCGTGAAAGTCCGGGGCTCAACTCCGGATCTGCGGTGGGTACGGGCAGACTAGAGTGATGTAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGGAGCGAACAGG\r\n",
      ">HWI-M04026:146:000000000-B58PG:1:1101:19367:1847_1:N:0:TGCTCGTAGATCGTGT\r\n",
      "TACGTAGGGCGCAAGCGTTATCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTTTGTCGCGTCTGCCGTGAAAGTCCGGGGCTCAACTCCGGATCTGCGGTGGGTACGGGCAGACTAGAGTGATGTAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGGAGCGAACAGG\r\n",
      ">HWI-M04026:146:000000000-B58PG:1:1101:15056:1900_1:N:0:TGCTCGTAGATCGTGT\r\n",
      "TACGTAGGGTCCGAGCGTTGTCCGGAATTACTGGGCGTAAAGAGCTCGTAGGTGGTTTGTCGCGTTGTTCGTGAAAACTCACAGCTCAACTGTGGGCGTGCGGGCGATACGGGCAGACTGGAGTACTGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGGAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCAGTAACTGACGCTGAGGAGCGAAAGCGTGGGGAGCGAACAGG\r\n"
     ]
    }
   ],
   "source": [
    "!head ../data/Seq_data/QCSeqs/QCseqs.sed.fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > help()\n",
      "For more information about a specific command type 'commandName(help)' i.e. 'cluster(help)'\n",
      "\n",
      "For further assistance please refer to the Mothur manual on our wiki at http://www.mothur.org/wiki, or contact Pat Schloss at mothur.bugs@gmail.com.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "help()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "May need to run this outside of notebook...\n",
    "\n",
    "echo $PATH\n",
    "\n",
    "export PATH=$PATH:/opt/virt_env/bin/mothur"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Just check out the sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > summary.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.fasta)\n",
      "\n",
      "Using 1 processors.\n",
      "\n",
      "Start\tEnd\tNBases\tAmbigs\tPolymer\tNumSeqs\n",
      "Minimum:\t1\t54\t54\t0\t3\t1\n",
      "2.5%-tile:\t1\t253\t253\t0\t4\t163757\n",
      "25%-tile:\t1\t253\t253\t0\t4\t1637568\n",
      "Median: \t1\t253\t253\t0\t5\t3275135\n",
      "75%-tile:\t1\t253\t253\t0\t5\t4912702\n",
      "97.5%-tile:\t1\t254\t254\t0\t6\t6386512\n",
      "Maximum:\t1\t427\t427\t0\t21\t6550268\n",
      "Mean:\t1\t252.848\t252.848\t0\t4.67561\n",
      "# of Seqs:\t6550268\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.summary\n",
      "\n",
      "It took 168 secs to summarize 6550268 sequences.\n",
      "[WARNING]: your sequence names contained ':'.  I changed them to '_' to avoid problems in your downstream analysis.\n",
      "\n",
      "mothur > quit()\n",
      "\n",
      "\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "Detected 1 [WARNING] messages, please review.\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "summary.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.fasta)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Getting only the unique sequences to work with next, for efficiency"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > unique.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.fasta)\n",
      "6550268\t1838561\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.names\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.fasta\n",
      "\n",
      "[WARNING]: your sequence names contained ':'.  I changed them to '_' to avoid problems in your downstream analysis.\n",
      "\n",
      "mothur > quit()\n",
      "\n",
      "\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "Detected 1 [WARNING] messages, please review.\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "unique.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.fasta)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "6.55M total sequences; 1.8M unique sequences.\n",
    "\n",
    "Next, aligning sequences based on filtered silva database (db created by trimming with our primers)\n",
    "\n",
    "Note: the alignment step takes a while (10s of minutes on my laptop)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Creating the database for alignment"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > pcr.seqs(fasta=silva.total.filter.fasta, oligos=oligos.txt, processors=4)\n",
      "\n",
      "Using 4 processors.\n",
      "\n",
      "Output File Names:\n",
      "silva.total.filter.pcr.fasta\n",
      "silva.total.filter.bad.accnos\n",
      "silva.total.filter.scrap.pcr.fasta\n",
      "\n",
      "\n",
      "It took 24 secs to screen 18491 sequences.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "pcr.seqs(fasta=silva.total.filter.fasta, oligos=oligos.txt, processors=4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > summary.seqs(fasta=silva.total.filter.pcr.fasta)\n",
      "\n",
      "Using 1 processors.\n",
      "\n",
      "Start\tEnd\tNBases\tAmbigs\tPolymer\tNumSeqs\n",
      "Minimum:\t4186\t7013\t230\t0\t3\t1\n",
      "2.5%-tile:\t4187\t7013\t252\t0\t4\t393\n",
      "25%-tile:\t4187\t7013\t253\t0\t4\t3922\n",
      "Median: \t4187\t7013\t253\t0\t4\t7844\n",
      "75%-tile:\t4187\t7013\t253\t0\t5\t11766\n",
      "97.5%-tile:\t4187\t7013\t254\t1\t6\t15295\n",
      "Maximum:\t4193\t7060\t1017\t5\t9\t15687\n",
      "Mean:\t4187\t7013\t253.095\t0.047619\t4.59788\n",
      "# of Seqs:\t15687\n",
      "\n",
      "Output File Names:\n",
      "silva.total.filter.pcr.summary\n",
      "\n",
      "It took 8 secs to summarize 15687 sequences.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "summary.seqs(fasta=silva.total.filter.pcr.fasta)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Looks pretty good - we got some very long sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > screen.seqs(fasta=silva.total.filter.pcr.fasta, start=4186, end=7013, processors=4)\n",
      "\n",
      "Using 4 processors.\n",
      "\n",
      "Output File Names:\n",
      "silva.total.filter.pcr.good.fasta\n",
      "silva.total.filter.pcr.bad.accnos\n",
      "\n",
      "\n",
      "It took 4 secs to screen 15687 sequences.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "screen.seqs(fasta=silva.total.filter.pcr.fasta, start=4186, end=7013, processors=4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > summary.seqs(fasta=silva.total.filter.pcr.good.fasta)\n",
      "\n",
      "Using 1 processors.\n",
      "\n",
      "Start\tEnd\tNBases\tAmbigs\tPolymer\tNumSeqs\n",
      "Minimum:\t4186\t7013\t253\t0\t4\t1\n",
      "2.5%-tile:\t4186\t7013\t253\t0\t4\t1\n",
      "25%-tile:\t4186\t7013\t253\t0\t4\t1\n",
      "Median: \t4186\t7013\t254\t0\t4\t2\n",
      "75%-tile:\t4186\t7013\t255\t0\t5\t3\n",
      "97.5%-tile:\t4186\t7013\t253\t0\t4\t3\n",
      "Maximum:\t4186\t7013\t255\t0\t5\t3\n",
      "Mean:\t4186\t7013\t254\t0\t4.33333\n",
      "# of Seqs:\t3\n",
      "\n",
      "Output File Names:\n",
      "silva.total.filter.pcr.good.summary\n",
      "\n",
      "It took 0 secs to summarize 3 sequences.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "summary.seqs(fasta=silva.total.filter.pcr.good.fasta)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!mv silva.total.filter.pcr.good.fasta silva.db.fasta\n",
    "#Renaming the database file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!head -10000 ../data/Seq_data/QCSeqs/QCseqs.sed.unique.fasta > test.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Testing seconds to align 5000 seqs with diff ksize\n",
    "ksize=5 6,5,6,7\n",
    "ksize=6 5,4,4,4\n",
    "ksize=7 4,4,4,4\n",
    "ksize=8 5,4,4,5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > align.seqs(candidate=../data/Seq_data/QCSeqs/QCseqs.sed.unique.fasta, template=silva.db.fasta, processors=4, flip=T, ksize=7)\n",
      "\n",
      "Using 4 processors.\n",
      "\n",
      "Reading in the silva.db.fasta template sequences...\tDONE.\n",
      "It took 0 to read  3 sequences.\n",
      "Aligning sequences from ../data/Seq_data/QCSeqs/QCseqs.sed.unique.fasta ...\n",
      "[WARNING]: Some of your sequences generated alignments that eliminated too many bases, a list is provided in ../data/Seq_data/QCSeqs/QCseqs.sed.unique.flip.accnos. If the reverse compliment proved to be better it was reported.\n",
      "It took 2459 secs to align 1838561 sequences.\n",
      "\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.align\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.align.report\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.flip.accnos\n",
      "\n",
      "\n",
      "mothur > quit()\n",
      "\n",
      "\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "Detected 1 [WARNING] messages, please review.\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n",
      "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<^>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "align.seqs(candidate=../data/Seq_data/QCSeqs/QCseqs.sed.unique.fasta, template=silva.db.fasta, processors=4, flip=T, ksize=7)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "260546\n",
      "945\n"
     ]
    }
   ],
   "source": [
    "!grep -c \">\" ../data/Seq_data/QCSeqs/QCseqs.sed.unique.align\n",
    "!grep -c \"HWI\" ../data/Seq_data/QCSeqs/QCseqs.sed.unique.flip.accnos\n",
    "# Seems like we lost a lot of unique sequences.\n",
    "# I.e., 1.8M to 0.2M... Yes, could be lots of sequence errors...\n",
    "# But might be worth re-running alignment step with re-filtered database.\n",
    "# For now..."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">HWI-M04026:146:000000000-B58PG:1:1105:16908:4966_1:N:0:TGCTCGTAGATCGTGT\r\n",
      "TACATGAAAGACTAGTGTTATTCATCTTTAATAGGTTTAAAGGGTACCTAGACAGTAAGTATAGCCTAAAAAAGGAACATATTTACTAGAGTTTTATTAGAGAAGGTGGAATTTTTGGAGTAAAGATATAATTTGCTAAAACCAAAGAGACAGACAACGGTGAAGACAACCTTCTATCTATAAACTGACGTTGAGGGACGAAGGCTTGGGTAGCGATAAGG\r\n"
     ]
    }
   ],
   "source": [
    "# We lost quite a few seqs.. blast a few sequences from the ../data/Seq_data/QCSeqs/QCseqs.sed.unique.flip.accnos file\n",
    "# !grep -A \"1107:21149:28053_1:N:0:TGCTCGTAGATCGTGT\" ../data/Seq_data/QCSeqs/QCseqs.sed.fasta\n",
    "    # Mitochondrial\n",
    "!grep -A 1 \"1105:16908:4966_1:N:0:TGCTCGTAGATCGTGT\" ../data/Seq_data/QCSeqs/QCseqs.sed.fasta\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > summary.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.align, name=../data/Seq_data/QCSeqs/QCseqs.sed.names, processors=4)\n",
      "\n",
      "Using 4 processors.\n",
      "\n",
      "Start\tEnd\tNBases\tAmbigs\tPolymer\tNumSeqs\n",
      "Minimum:\t629\t3455\t4\t0\t1\t1\n",
      "2.5%-tile:\t4187\t7013\t253\t0\t4\t82194\n",
      "25%-tile:\t4187\t7013\t253\t0\t4\t821937\n",
      "Median: \t4187\t7013\t253\t0\t4\t1643873\n",
      "75%-tile:\t4187\t7013\t253\t0\t5\t2465809\n",
      "97.5%-tile:\t4187\t7013\t254\t0\t6\t3205551\n",
      "Maximum:\t7008\t7013\t266\t0\t11\t3287744\n",
      "Mean:\t4189.84\t7008.13\t252.244\t0\t4.64514\n",
      "# of unique seqs:\t260546\n",
      "total # of seqs:\t3287744\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.summary\n",
      "\n",
      "It took 48 secs to summarize 3287744 sequences.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "summary.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.align, name=../data/Seq_data/QCSeqs/QCseqs.sed.names, processors=4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > screen.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.align, name=../data/Seq_data/QCSeqs/QCseqs.sed.names, minlength=250, maxlength=266, maxhomop=8, processors=4)\n",
      "\n",
      "Using 4 processors.\n",
      "Your namefile does not include the sequence HWI-M04026_146_000000000-B58PG_1_1106_20951_21276_1_N_0_TAGCAGCTHWI-M04026_146_000000000-B58PG_1_1108_23344_22291_1_N_0_ACTACGACTACGAGAC please correct.\n",
      "Your namefile does not include the sequence HWI-M04026_146_000000000-B58PG_1_1108 please correct.\n",
      "Your namefile does not include the sequence HWI-M04026_146_000000000-B58PG_1_HWI-M04026_146_000000000-B58PG_1_1113_15555_24380_1_N_0_TCTCTATGTAGTGTAG please correct.\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.align\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.bad.accnos\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.good.names\n",
      "\n",
      "\n",
      "It took 163 secs to screen 260546 sequences.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "screen.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.align, name=../data/Seq_data/QCSeqs/QCseqs.sed.names, minlength=250, maxlength=266, maxhomop=8, processors=4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "112378\r\n"
     ]
    }
   ],
   "source": [
    "!grep -c \">\" ../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.align"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Then, filter the sequences so there are no gaps in the alignment"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > filter.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.align, vertical=T, processors=4)\n",
      "\n",
      "Using 4 processors.\n",
      "Creating Filter...\n",
      "[ERROR]: Sequences are not all the same length, please correct.\n",
      "[ERROR]: Sequences are not all the same length, please correct.\n",
      "[ERROR]: Sequences are not all the same length, please correct.\n",
      "\n",
      "\n",
      "Running Filter...\n",
      "\n",
      "\n",
      "\n",
      "Length of filtered alignment: 11616\n",
      "Number of columns removed: 0\n",
      "Length of the original alignment: 11616\n",
      "Number of sequences used to construct filter: 112378\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.filter\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.fasta\n",
      "\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "filter.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.align, vertical=T, processors=4)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "And summarize them:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > summary.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.fasta, name=../data/Seq_data/QCSeqs/QCseqs.sed.good.names, processors=4)\n",
      "\n",
      "Using 4 processors.\n",
      "\n",
      "Start\tEnd\tNBases\tAmbigs\tPolymer\tNumSeqs\n",
      "Minimum:\t984\t3810\t250\t0\t3\t1\n",
      "2.5%-tile:\t4187\t7013\t253\t0\t4\t51245\n",
      "25%-tile:\t4187\t7013\t253\t0\t4\t512442\n",
      "Median: \t4187\t7013\t253\t0\t4\t1024883\n",
      "75%-tile:\t4187\t7013\t253\t0\t5\t1537324\n",
      "97.5%-tile:\t4187\t7013\t254\t0\t6\t1998521\n",
      "Maximum:\t4194\t7013\t260\t0\t8\t2049765\n",
      "Mean:\t4187\t7013\t253.034\t0\t4.63232\n",
      "# of unique seqs:\t29442\n",
      "total # of seqs:\t2049765\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.summary\n",
      "\n",
      "It took 15 secs to summarize 2049765 sequences.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "summary.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.fasta, name=../data/Seq_data/QCSeqs/QCseqs.sed.names, processors=4)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In this step, we will remove any sequences containing homopolymers with a length greater than 8, and any sequences that do not lie in our amplicon region.\n",
    "\n",
    "In the previous step, the median 'start' and 'end' positions will be used to trim off any sequences that do not lie in our amplicon region. Every time you run this step, you must adjust the values in the following code to match these for your specific dataset."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > screen.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.fasta, name=../data/Seq_data/QCSeqs/QCseqs.sed.good.names, end=7000, processors=4)\n",
      "\n",
      "Using 4 processors.\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.fasta\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.bad.accnos\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.good.good.names\n",
      "\n",
      "\n",
      "It took 32 secs to screen 29442 sequences.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "screen.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.fasta, name=../data/Seq_data/QCSeqs/QCseqs.sed.good.names, end=7000, processors=4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > summary.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.fasta, name=../data/Seq_data/QCSeqs/QCseqs.sed.good.good.names, processors=4)\n",
      "\n",
      "Using 4 processors.\n",
      "\n",
      "Start\tEnd\tNBases\tAmbigs\tPolymer\tNumSeqs\n",
      "Minimum:\t4187\t7009\t250\t0\t3\t1\n",
      "2.5%-tile:\t4187\t7013\t253\t0\t4\t51245\n",
      "25%-tile:\t4187\t7013\t253\t0\t4\t512441\n",
      "Median: \t4187\t7013\t253\t0\t4\t1024881\n",
      "75%-tile:\t4187\t7013\t253\t0\t5\t1537321\n",
      "97.5%-tile:\t4187\t7013\t254\t0\t6\t1998517\n",
      "Maximum:\t4194\t7013\t260\t0\t8\t2049761\n",
      "Mean:\t4187\t7013\t253.034\t0\t4.63232\n",
      "# of unique seqs:\t29440\n",
      "total # of seqs:\t2049761\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.summary\n",
      "\n",
      "It took 15 secs to summarize 2049761 sequences.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "summary.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.fasta, name=../data/Seq_data/QCSeqs/QCseqs.sed.names, processors=4)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Erring for now on the side of less trimming"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of sequences post-filter: 29440\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "printf \"Number of sequences post-filter: \"\n",
    "grep -c \">\" ../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Filter the sequences again to remove any new gaps"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > filter.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.fasta, vertical=T, trump=., processors=4)\n",
      "\n",
      "Using 4 processors.\n",
      "Creating Filter...\n",
      "[ERROR]: Sequences are not all the same length, please correct.\n",
      "[ERROR]: Sequences are not all the same length, please correct.\n",
      "\n",
      "\n",
      "Running Filter...\n",
      "\n",
      "\n",
      "\n",
      "Length of filtered alignment: 2816\n",
      "Number of columns removed: 8800\n",
      "Length of the original alignment: 11616\n",
      "Number of sequences used to construct filter: 29440\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.filter\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.filter.fasta\n",
      "\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "filter.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.fasta, vertical=T, trump=., processors=4)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Summarize them one more time"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > summary.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.filter.fasta, name=../data/Seq_data/QCSeqs/QCseqs.sed.good.good.names, processors=4)\n",
      "\n",
      "Using 4 processors.\n",
      "\n",
      "Start\tEnd\tNBases\tAmbigs\tPolymer\tNumSeqs\n",
      "Minimum:\t1\t2816\t246\t0\t3\t1\n",
      "2.5%-tile:\t1\t2816\t249\t0\t4\t51245\n",
      "25%-tile:\t1\t2816\t249\t0\t4\t512441\n",
      "Median: \t1\t2816\t249\t0\t4\t1024881\n",
      "75%-tile:\t1\t2816\t249\t0\t5\t1537321\n",
      "97.5%-tile:\t1\t2816\t250\t0\t6\t1998517\n",
      "Maximum:\t1\t2816\t256\t0\t8\t2049761\n",
      "Mean:\t1\t2816\t249.034\t0\t4.63229\n",
      "# of unique seqs:\t29440\n",
      "total # of seqs:\t2049761\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.filter.summary\n",
      "\n",
      "It took 14 secs to summarize 2049761 sequences.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "summary.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.filter.fasta, name=../data/Seq_data/QCSeqs/QCseqs.sed.names, processors=4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > deunique.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.filter.fasta, name=../data/Seq_data/QCSeqs/QCseqs.sed.names)\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.filter.redundant.fasta\n",
      "\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "deunique.seqs(fasta=../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.filter.fasta, name=../data/Seq_data/QCSeqs/QCseqs.sed.names)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "And we just have to take out all the alignment characters:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "nprocs=4\n",
    "!sed '/>/! s/-//g;/>/! s/\\.//g' ../data/Seq_data/QCSeqs/QCseqs.sed.unique.good.filter.good.filter.redundant.fasta > ../data/Seq_data/QCSeqs/16S_QC.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "One last summary:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mothur > summary.seqs(fasta=../data/Seq_data/QCSeqs/16S_QC.fasta, processors=4)\n",
      "\n",
      "Using 4 processors.\n",
      "\n",
      "Start\tEnd\tNBases\tAmbigs\tPolymer\tNumSeqs\n",
      "Minimum:\t1\t116\t116\t0\t3\t1\n",
      "2.5%-tile:\t1\t249\t249\t0\t4\t34497\n",
      "25%-tile:\t1\t249\t249\t0\t4\t344967\n",
      "Median: \t1\t249\t249\t0\t4\t689934\n",
      "75%-tile:\t1\t249\t249\t0\t5\t1034901\n",
      "97.5%-tile:\t1\t250\t250\t0\t6\t1345371\n",
      "Maximum:\t1\t251\t251\t0\t8\t1379867\n",
      "Mean:\t1\t249.023\t249.023\t0\t4.63081\n",
      "# of Seqs:\t1379867\n",
      "\n",
      "Output File Names:\n",
      "../data/Seq_data/QCSeqs/16S_QC.summary\n",
      "\n",
      "It took 15 secs to summarize 1379867 sequences.\n",
      "\n",
      "mothur > quit()\n"
     ]
    }
   ],
   "source": [
    "%%mothur\n",
    "summary.seqs(fasta=../data/Seq_data/QCSeqs/16S_QC.fasta, processors=4)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "These reductions are similar to those we saw for the mineral project - 8.x M seqs >> 1.x M seqs\n",
    "\n",
    "So, seems in line with previous labs' sequencing. Less than PyOM project, though - ended up with 8M seqs, but that was 2x300PE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
